<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>11 Regulation of Gene Expression | Text For Biology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Biology" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="11 Regulation of Gene Expression | Text For Biology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="biology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Biology" />
  <meta name="github-repo" content="nikolaussuchert/biology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="11 Regulation of Gene Expression | Text For Biology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Biology" />
  <meta name="twitter:image" content="biology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="molecular-biology-of-the-gene.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>



<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Biology</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="biology.html"><a href="biology.html"><i class="fa fa-check"></i><b>1</b> Biology</a><ul>
<li class="chapter" data-level="1.1" data-path="biology.html"><a href="biology.html#foundations-of-modern-biology"><i class="fa fa-check"></i><b>1.1</b> Foundations of modern biology</a><ul>
<li class="chapter" data-level="1.1.1" data-path="biology.html"><a href="biology.html#cell-theory"><i class="fa fa-check"></i><b>1.1.1</b> Cell theory</a></li>
<li class="chapter" data-level="1.1.2" data-path="biology.html"><a href="biology.html#evolution"><i class="fa fa-check"></i><b>1.1.2</b> Evolution</a></li>
<li class="chapter" data-level="1.1.3" data-path="biology.html"><a href="biology.html#genetics"><i class="fa fa-check"></i><b>1.1.3</b> Genetics</a></li>
<li class="chapter" data-level="1.1.4" data-path="biology.html"><a href="biology.html#homeostasis"><i class="fa fa-check"></i><b>1.1.4</b> Homeostasis</a></li>
<li class="chapter" data-level="1.1.5" data-path="biology.html"><a href="biology.html#energy"><i class="fa fa-check"></i><b>1.1.5</b> Energy</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="biology.html"><a href="biology.html#areas-and-levels-of-study-and-research-in-biology"><i class="fa fa-check"></i><b>1.2</b> Areas And Levels of Study And Research in Biology</a><ul>
<li class="chapter" data-level="1.2.1" data-path="biology.html"><a href="biology.html#structural"><i class="fa fa-check"></i><b>1.2.1</b> Structural</a></li>
<li class="chapter" data-level="1.2.2" data-path="biology.html"><a href="biology.html#physiological"><i class="fa fa-check"></i><b>1.2.2</b> Physiological</a></li>
<li class="chapter" data-level="1.2.3" data-path="biology.html"><a href="biology.html#evolutionary"><i class="fa fa-check"></i><b>1.2.3</b> Evolutionary</a></li>
<li class="chapter" data-level="1.2.4" data-path="biology.html"><a href="biology.html#systematic"><i class="fa fa-check"></i><b>1.2.4</b> Systematic</a></li>
<li class="chapter" data-level="1.2.5" data-path="biology.html"><a href="biology.html#ecological-and-environmental"><i class="fa fa-check"></i><b>1.2.5</b> Ecological And Environmental</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html"><i class="fa fa-check"></i><b>2</b> Chemistry For Biology</a><ul>
<li class="chapter" data-level="2.1" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#matter"><i class="fa fa-check"></i><b>2.1</b> Matter</a><ul>
<li class="chapter" data-level="2.1.1" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#atom"><i class="fa fa-check"></i><b>2.1.1</b> Atom</a></li>
<li class="chapter" data-level="2.1.2" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#element"><i class="fa fa-check"></i><b>2.1.2</b> Element</a></li>
<li class="chapter" data-level="2.1.3" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#compound"><i class="fa fa-check"></i><b>2.1.3</b> Compound</a></li>
<li class="chapter" data-level="2.1.4" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#molecule"><i class="fa fa-check"></i><b>2.1.4</b> Molecule</a></li>
<li class="chapter" data-level="2.1.5" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#chemical-bonds"><i class="fa fa-check"></i><b>2.1.5</b> Chemical Bonds</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="biology.html"><a href="biology.html#energy"><i class="fa fa-check"></i><b>2.2</b> Energy</a></li>
<li class="chapter" data-level="2.3" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#chemical-reactions"><i class="fa fa-check"></i><b>2.3</b> Chemical Reactions</a></li>
<li class="chapter" data-level="2.4" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#ions-and-salts"><i class="fa fa-check"></i><b>2.4</b> Ions And Salts</a></li>
<li class="chapter" data-level="2.5" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#acidity-and-basicity"><i class="fa fa-check"></i><b>2.5</b> Acidity and basicity</a></li>
<li class="chapter" data-level="2.6" data-path="chemistry-for-biology.html"><a href="chemistry-for-biology.html#redox"><i class="fa fa-check"></i><b>2.6</b> Redox</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="organic-chemistry-for-biology.html"><a href="organic-chemistry-for-biology.html"><i class="fa fa-check"></i><b>3</b> Organic Chemistry For Biology</a><ul>
<li class="chapter" data-level="3.1" data-path="organic-chemistry-for-biology.html"><a href="organic-chemistry-for-biology.html#biolmolecules"><i class="fa fa-check"></i><b>3.1</b> Biolmolecules</a></li>
<li class="chapter" data-level="3.2" data-path="organic-chemistry-for-biology.html"><a href="organic-chemistry-for-biology.html#protein"><i class="fa fa-check"></i><b>3.2</b> Protein</a><ul>
<li class="chapter" data-level="3.2.1" data-path="organic-chemistry-for-biology.html"><a href="organic-chemistry-for-biology.html#structure"><i class="fa fa-check"></i><b>3.2.1</b> Structure</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="organic-chemistry-for-biology.html"><a href="organic-chemistry-for-biology.html#carbohydrate"><i class="fa fa-check"></i><b>3.3</b> Carbohydrate</a></li>
<li class="chapter" data-level="3.4" data-path="organic-chemistry-for-biology.html"><a href="organic-chemistry-for-biology.html#lipid"><i class="fa fa-check"></i><b>3.4</b> Lipid</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="the-cell.html"><a href="the-cell.html"><i class="fa fa-check"></i><b>4</b> The Cell</a><ul>
<li class="chapter" data-level="4.1" data-path="the-cell.html"><a href="the-cell.html#prokaryotic-cells"><i class="fa fa-check"></i><b>4.1</b> Prokaryotic cells</a></li>
<li class="chapter" data-level="4.2" data-path="the-cell.html"><a href="the-cell.html#eukaryotic-cells"><i class="fa fa-check"></i><b>4.2</b> Eukaryotic cells</a><ul>
<li class="chapter" data-level="4.2.1" data-path="the-cell.html"><a href="the-cell.html#subcellular-components"><i class="fa fa-check"></i><b>4.2.1</b> Subcellular components</a></li>
<li class="chapter" data-level="4.2.2" data-path="the-cell.html"><a href="the-cell.html#membrane"><i class="fa fa-check"></i><b>4.2.2</b> Membrane</a></li>
<li class="chapter" data-level="4.2.3" data-path="the-cell.html"><a href="the-cell.html#cytoskeleton"><i class="fa fa-check"></i><b>4.2.3</b> Cytoskeleton</a></li>
<li class="chapter" data-level="4.2.4" data-path="the-cell.html"><a href="the-cell.html#genetic-material"><i class="fa fa-check"></i><b>4.2.4</b> Genetic material</a></li>
<li class="chapter" data-level="4.2.5" data-path="the-cell.html"><a href="the-cell.html#organelles"><i class="fa fa-check"></i><b>4.2.5</b> Organelles</a></li>
<li class="chapter" data-level="4.2.6" data-path="the-cell.html"><a href="the-cell.html#eukaryotic"><i class="fa fa-check"></i><b>4.2.6</b> Eukaryotic</a></li>
<li class="chapter" data-level="4.2.7" data-path="the-cell.html"><a href="the-cell.html#eukaryotic-and-prokaryotic"><i class="fa fa-check"></i><b>4.2.7</b> Eukaryotic and prokaryotic</a></li>
<li class="chapter" data-level="4.2.8" data-path="the-cell.html"><a href="the-cell.html#structures-outside-the-cell-membrane"><i class="fa fa-check"></i><b>4.2.8</b> Structures outside the cell membrane</a></li>
<li class="chapter" data-level="4.2.9" data-path="the-cell.html"><a href="the-cell.html#cell-wall"><i class="fa fa-check"></i><b>4.2.9</b> Cell wall</a></li>
<li class="chapter" data-level="4.2.10" data-path="the-cell.html"><a href="the-cell.html#prokaryotic"><i class="fa fa-check"></i><b>4.2.10</b> Prokaryotic</a></li>
<li class="chapter" data-level="4.2.11" data-path="the-cell.html"><a href="the-cell.html#capsule"><i class="fa fa-check"></i><b>4.2.11</b> Capsule</a></li>
<li class="chapter" data-level="4.2.12" data-path="the-cell.html"><a href="the-cell.html#flagella"><i class="fa fa-check"></i><b>4.2.12</b> Flagella</a></li>
<li class="chapter" data-level="4.2.13" data-path="the-cell.html"><a href="the-cell.html#fimbriae"><i class="fa fa-check"></i><b>4.2.13</b> Fimbriae</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="the-cell.html"><a href="the-cell.html#cellular-processes"><i class="fa fa-check"></i><b>4.3</b> Cellular processes</a><ul>
<li class="chapter" data-level="4.3.1" data-path="the-cell.html"><a href="the-cell.html#replication"><i class="fa fa-check"></i><b>4.3.1</b> Replication</a></li>
<li class="chapter" data-level="4.3.2" data-path="the-cell.html"><a href="the-cell.html#dna-repair"><i class="fa fa-check"></i><b>4.3.2</b> DNA repair</a></li>
<li class="chapter" data-level="4.3.3" data-path="the-cell.html"><a href="the-cell.html#growth-and-metabolism"><i class="fa fa-check"></i><b>4.3.3</b> Growth and metabolism</a></li>
<li class="chapter" data-level="4.3.4" data-path="the-cell.html"><a href="the-cell.html#protein-synthesis"><i class="fa fa-check"></i><b>4.3.4</b> Protein synthesis</a></li>
<li class="chapter" data-level="4.3.5" data-path="the-cell.html"><a href="the-cell.html#motility"><i class="fa fa-check"></i><b>4.3.5</b> Motility</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="the-cell.html"><a href="the-cell.html#origin-of-the-first-cell"><i class="fa fa-check"></i><b>4.4</b> Origin of the first cell</a></li>
<li class="chapter" data-level="4.5" data-path="the-cell.html"><a href="the-cell.html#origin-of-eukaryotic-cells"><i class="fa fa-check"></i><b>4.5</b> Origin of eukaryotic cells</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html"><i class="fa fa-check"></i><b>5</b> The Cell Membrane</a><ul>
<li class="chapter" data-level="5.1" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#function"><i class="fa fa-check"></i><b>5.1</b> Function</a></li>
<li class="chapter" data-level="5.2" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#fluid-mosaic-model"><i class="fa fa-check"></i><b>5.2</b> Fluid mosaic model</a><ul>
<li class="chapter" data-level="5.2.1" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#lipid-bilayer"><i class="fa fa-check"></i><b>5.2.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="5.2.2" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#membrane-polarity"><i class="fa fa-check"></i><b>5.2.2</b> Membrane polarity</a></li>
<li class="chapter" data-level="5.2.3" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#membrane-structures"><i class="fa fa-check"></i><b>5.2.3</b> Membrane structures</a></li>
<li class="chapter" data-level="5.2.4" data-path="the-cell.html"><a href="the-cell.html#cytoskeleton"><i class="fa fa-check"></i><b>5.2.4</b> Cytoskeleton</a></li>
<li class="chapter" data-level="5.2.5" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#intracellular-membranes"><i class="fa fa-check"></i><b>5.2.5</b> Intracellular membranes</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#permeability"><i class="fa fa-check"></i><b>5.3</b> Permeability</a></li>
<li class="chapter" data-level="5.4" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#diffusion"><i class="fa fa-check"></i><b>5.4</b> Diffusion</a></li>
<li class="chapter" data-level="5.5" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#facilitated-diffusion"><i class="fa fa-check"></i><b>5.5</b> Facilitated Diffusion</a></li>
<li class="chapter" data-level="5.6" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#osmosis"><i class="fa fa-check"></i><b>5.6</b> Osmosis</a></li>
<li class="chapter" data-level="5.7" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#active-transport"><i class="fa fa-check"></i><b>5.7</b> Active transport</a></li>
<li class="chapter" data-level="5.8" data-path="the-cell-membrane.html"><a href="the-cell-membrane.html#bulk-transport"><i class="fa fa-check"></i><b>5.8</b> Bulk transport</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="metabolism.html"><a href="metabolism.html"><i class="fa fa-check"></i><b>6</b> Metabolism</a></li>
<li class="chapter" data-level="7" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html"><i class="fa fa-check"></i><b>7</b> Photosynthesis And Cellular Respiration</a><ul>
<li class="chapter" data-level="7.1" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#photosynthesis"><i class="fa fa-check"></i><b>7.1</b> Photosynthesis</a></li>
<li class="chapter" data-level="7.2" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#cellular-respiration"><i class="fa fa-check"></i><b>7.2</b> Cellular respiration</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="the-cell-cycle.html"><a href="the-cell-cycle.html"><i class="fa fa-check"></i><b>8</b> The Cell Cycle</a></li>
<li class="chapter" data-level="9" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html"><i class="fa fa-check"></i><b>9</b> Mendelian Inheritance</a><ul>
<li class="chapter" data-level="9.1" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendels-genetic-discoveries"><i class="fa fa-check"></i><b>9.1</b> Mendel’s genetic discoveries</a></li>
<li class="chapter" data-level="9.2" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-dominance-and-uniformity"><i class="fa fa-check"></i><b>9.2</b> Law of Dominance and Uniformity</a></li>
<li class="chapter" data-level="9.3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-segregation-of-genes"><i class="fa fa-check"></i><b>9.3</b> Law of Segregation of genes</a></li>
<li class="chapter" data-level="9.4" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-independent-assortment"><i class="fa fa-check"></i><b>9.4</b> Law of Independent Assortment</a></li>
<li class="chapter" data-level="9.5" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-trait"><i class="fa fa-check"></i><b>9.5</b> Mendelian trait</a></li>
<li class="chapter" data-level="9.6" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#zygosity"><i class="fa fa-check"></i><b>9.6</b> Zygosity</a></li>
<li class="chapter" data-level="9.7" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#monohybrid-cross"><i class="fa fa-check"></i><b>9.7</b> Monohybrid cross</a></li>
<li class="chapter" data-level="9.8" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#punnett-square"><i class="fa fa-check"></i><b>9.8</b> Punnett square</a></li>
<li class="chapter" data-level="9.9" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#dihybrid-cross"><i class="fa fa-check"></i><b>9.9</b> Dihybrid cross</a></li>
<li class="chapter" data-level="9.10" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#forked-line-method"><i class="fa fa-check"></i><b>9.10</b> Forked-line method</a></li>
<li class="chapter" data-level="9.11" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-traits-in-humans"><i class="fa fa-check"></i><b>9.11</b> Mendelian traits in humans</a></li>
<li class="chapter" data-level="9.12" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#omim---online-mendelian-inheritance-in-man"><i class="fa fa-check"></i><b>9.12</b> OMIM - Online Mendelian Inheritance in Man</a></li>
<li class="chapter" data-level="9.13" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#non-mendelian-inheritance"><i class="fa fa-check"></i><b>9.13</b> Non-Mendelian inheritance</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html"><i class="fa fa-check"></i><b>10</b> Molecular Biology Of The Gene</a><ul>
<li class="chapter" data-level="10.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna"><i class="fa fa-check"></i><b>10.1</b> DNA</a><ul>
<li class="chapter" data-level="10.1.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#structure-and-composition-of-dna"><i class="fa fa-check"></i><b>10.1.1</b> Structure and composition of DNA</a></li>
<li class="chapter" data-level="10.1.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#base-modifications-and-dna-packaging"><i class="fa fa-check"></i><b>10.1.2</b> Base modifications and DNA packaging</a></li>
<li class="chapter" data-level="10.1.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-damage"><i class="fa fa-check"></i><b>10.1.3</b> DNA damage</a></li>
<li class="chapter" data-level="10.1.4" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#biological-functions-of-dna"><i class="fa fa-check"></i><b>10.1.4</b> Biological functions of DNA</a></li>
<li class="chapter" data-level="10.1.5" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-replication"><i class="fa fa-check"></i><b>10.1.5</b> DNA replication</a></li>
<li class="chapter" data-level="10.1.6" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#interactions-of-dna-with-proteins"><i class="fa fa-check"></i><b>10.1.6</b> Interactions of DNA with proteins</a></li>
<li class="chapter" data-level="10.1.7" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>10.1.7</b> DNA-modifying enzymes</a></li>
<li class="chapter" data-level="10.1.8" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-recombination"><i class="fa fa-check"></i><b>10.1.8</b> DNA recombination</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="biology.html"><a href="biology.html#evolution"><i class="fa fa-check"></i><b>10.2</b> Evolution</a></li>
<li class="chapter" data-level="10.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#genetic-engineering"><i class="fa fa-check"></i><b>10.3</b> Genetic engineering</a></li>
<li class="chapter" data-level="10.4" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-profiling"><i class="fa fa-check"></i><b>10.4</b> DNA profiling</a></li>
<li class="chapter" data-level="10.5" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#rna"><i class="fa fa-check"></i><b>10.5</b> RNA</a><ul>
<li class="chapter" data-level="10.5.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#comparison-with-dna"><i class="fa fa-check"></i><b>10.5.1</b> Comparison with DNA</a></li>
<li class="chapter" data-level="10.5.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#structure-of-rna"><i class="fa fa-check"></i><b>10.5.2</b> Structure of RNA</a></li>
<li class="chapter" data-level="10.5.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#synthesis-of-rna"><i class="fa fa-check"></i><b>10.5.3</b> Synthesis of RNA</a></li>
<li class="chapter" data-level="10.5.4" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#coding-and-non-coding-rna"><i class="fa fa-check"></i><b>10.5.4</b> Coding and non-coding RNA</a></li>
<li class="chapter" data-level="10.5.5" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#regulatory-rna"><i class="fa fa-check"></i><b>10.5.5</b> Regulatory RNA</a></li>
<li class="chapter" data-level="10.5.6" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>10.5.6</b> RNA interference by miRNAs</a></li>
</ul></li>
<li class="chapter" data-level="10.6" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#gene-expression"><i class="fa fa-check"></i><b>10.6</b> Gene Expression</a></li>
<li class="chapter" data-level="10.7" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#transcription"><i class="fa fa-check"></i><b>10.7</b> Transcription</a></li>
<li class="chapter" data-level="10.8" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#mrna-processing"><i class="fa fa-check"></i><b>10.8</b> mRNA processing</a></li>
<li class="chapter" data-level="10.9" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#rna-export"><i class="fa fa-check"></i><b>10.9</b> RNA export</a></li>
<li class="chapter" data-level="10.10" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#translation"><i class="fa fa-check"></i><b>10.10</b> Translation</a></li>
<li class="chapter" data-level="10.11" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#protein-folding"><i class="fa fa-check"></i><b>10.11</b> Protein Folding</a></li>
<li class="chapter" data-level="10.12" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#protein-translocation"><i class="fa fa-check"></i><b>10.12</b> Protein Translocation</a></li>
<li class="chapter" data-level="10.13" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#protein-transport"><i class="fa fa-check"></i><b>10.13</b> Protein Transport</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>11</b> Regulation of Gene Expression</a><ul>
<li class="chapter" data-level="11.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-lac-operon"><i class="fa fa-check"></i><b>11.1</b> The <em>lac</em> operon</a></li>
<li class="chapter" data-level="11.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-trp-operon"><i class="fa fa-check"></i><b>11.2</b> The <em>trp</em> operon</a></li>
<li class="chapter" data-level="11.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulated-stages-of-gene-expression"><i class="fa fa-check"></i><b>11.3</b> Regulated stages of gene expression</a><ul>
<li class="chapter" data-level="11.3.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-packing"><i class="fa fa-check"></i><b>11.3.1</b> Regulation of transciption by DNA packing</a></li>
<li class="chapter" data-level="11.3.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-modification"><i class="fa fa-check"></i><b>11.3.2</b> Regulation of transciption by DNA modification</a></li>
<li class="chapter" data-level="11.3.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-rna-polymerase"><i class="fa fa-check"></i><b>11.3.3</b> Regulation of RNA polymerase</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#post-transcriptional-regulation"><i class="fa fa-check"></i><b>11.4</b> Post-transcriptional regulation</a><ul>
<li class="chapter" data-level="11.4.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#three-prime-untranslated-regions-and-micrornas"><i class="fa fa-check"></i><b>11.4.1</b> Three prime untranslated regions and microRNAs</a></li>
</ul></li>
<li class="chapter" data-level="11.5" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-translation"><i class="fa fa-check"></i><b>11.5</b> Regulation of translation</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For Biology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="regulation-of-gene-expression" class="section level1">
<h1><span class="header-section-number">11</span> Regulation of Gene Expression</h1>
<p><a href="https://en.wikipedia.org/wiki/Regulation_of_gene_expression">Regulation of gene expression</a>, or gene regulation, includes a wide range of mechanisms that are used by cells to increase or decrease the production of specific gene products (protein or RNA). Sophisticated programs of gene expression are widely observed in biology, for example to trigger developmental pathways, respond to environmental stimuli, or adapt to new food sources. Virtually any step of gene expression can be modulated, from transcriptional initiation, to RNA processing, and to the post-translational modification of a protein. Often, one gene regulator controls another, and so on, in a gene regulatory network.</p>
<p>Gene regulation is essential for viruses, prokaryotes and eukaryotes as it increases the versatility and adaptability of an organism by allowing the cell to express protein when needed.</p>
<p>In multicellular organisms, gene regulation drives cellular differentiation and morphogenesis in the embryo, leading to the creation of different cell types that possess different gene expression profiles from the same genome sequence. Although this does not explain how gene regulation originated, evolutionary biologists include it as a partial explanation of how evolution works at a molecular level, and it is central to the science of evolutionary developmental biology (“evo-devo”).</p>
<p>Although as early as 1951, <a href="https://en.wikipedia.org/wiki/Barbara_McClintock">Barbara McClintock</a> showed interaction between two genetic loci, Activator (Ac) and Dissociator (Ds), in the color formation of maize seeds, the first discovery of a gene regulation system is widely considered to be the identification in 1961 of the <em>lac</em> operon, discovered by <a href="https://en.wikipedia.org/wiki/Fran%C3%A7ois_Jacob">François Jacob</a> and <a href="https://en.wikipedia.org/wiki/Jacques_Monod">Jacques Monod</a>, in which some enzymes involved in lactose metabolism are expressed by <em>Escherichia coli</em> only in the presence of lactose and absence of glucose.</p>
<div id="the-lac-operon" class="section level2">
<h2><span class="header-section-number">11.1</span> The <em>lac</em> operon</h2>
<p>The <a href="https://en.wikipedia.org/wiki/Lac_operon"><em>lac</em></a> operon (lactose operon) is an operon required for the transport and metabolism of lactose in <em>Escherichia coli</em> and many other enteric bacteria.</p>
<p>An operon is a functioning unit of DNA containing a cluster of genes under the control of a single promoter. The genes are transcribed together into an mRNA strand and either translated together in the cytoplasm, or undergo splicing to create monocistronic mRNAs that are translated separately, i.e. several strands of mRNA that each encode a single gene product. The result of this is that the genes contained in the operon are either expressed together or not at all. Several genes must be co-transcribed to define an operon.</p>
<p>Although glucose is the preferred carbon source for most bacteria, the <em>lac</em> operon allows for the effective digestion of lactose when glucose is not available through the activity of beta-galactosidase. Gene regulation of the <em>lac</em> operon was the first genetic regulatory mechanism to be understood clearly, so it has become a foremost example of prokaryotic gene regulation. It is discussed here in detail for this reason. This lactose metabolism system was used by François Jacob and Jacques Monod to determine how a biological cell knows which enzyme to synthesize. Their work on the <em>lac</em> operon won them the Nobel Prize in Physiology in 1965.</p>
<p>Bacterial operons are polycistronic transcripts that are able to produce multiple proteins from one mRNA transcript. In this case, when lactose is required as a sugar source for the bacterium, the three genes of the <em>lac</em> operon can be expressed and their subsequent proteins translated: lacZ, lacY, and lacA. The gene product of lacZ is β-galactosidase which cleaves lactose, a disaccharide, into glucose and galactose. lacY encodes Beta-galactoside permease, a membrane protein which becomes embedded in the cytoplasmic membrane to enable the cellular transport of lactose into the cell. Finally, lacA encodes Galactoside acetyltransferase.</p>

<div class="figure" style="text-align: center"><span id="fig:lacoper"></span>
<img src="figures/regulation/Lac_operon1.png" alt="" width="70%" />
<p class="caption">
Figure 11.1: <a href="https://commons.wikimedia.org/wiki/File:Lac_operon1.png"></a>
</p>
</div>
<p>It would be wasteful to produce enzymes when no lactose were available or if a preferable energy source such as glucose were available. The <em>lac</em> operon uses a two-part control mechanism to ensure that the cell expends energy producing the enzymes encoded by the <em>lac</em> operon only when necessary. In the absence of lactose, the <em>lac</em> repressor, lacI, halts production of the enzymes encoded by the <em>lac</em> operon. The <em>lac</em> repressor is always expressed unless a co-inducer binds to it. In other words, it is transcribed only in the presence of small molecule co-inducer. In the presence of glucose, the catabolite activator protein (CAP), required for production of the enzymes, remains inactive, and EIIAGlc shuts down lactose permease to prevent transport of lactose into the cell. This dual control mechanism causes the sequential utilization of glucose and lactose in two distinct growth phases, known as diauxie.</p>
<p>Three-letter abbreviations are used to describe phenotypes in bacteria including <em>E. coli</em>.</p>
<p>Examples include:</p>
<ul>
<li>Lac (the ability to use lactose),</li>
<li>His (the ability to synthesize the amino acid histidine)</li>
<li>Mot (swimming motility)</li>
<li>SmR (resistance to the antibiotic streptomycin)</li>
</ul>
<p>In the case of Lac, wild type cells are Lac+ and are able to use lactose as a carbon and energy source, while Lac− mutant derivatives cannot use lactose. The same three letters are typically used (lower-case, italicized) to label the genes involved in a particular phenotype, where each different gene is additionally distinguished by an extra letter. The <em>lac</em> genes encoding enzymes are lacZ, lacY, and lacA. The fourth <em>lac</em> gene is lacI, encoding the lactose repressor—“I” stands for inducibility.</p>
<p>One may distinguish between structural genes encoding enzymes, and regulatory genes encoding proteins that affect gene expression. Current usage expands the phenotypic nomenclature to apply to proteins: thus, LacZ is the protein product of the lacZ gene, β-galactosidase. Various short sequences that are not genes also affect gene expression, including the <em>lac</em> promoter, <em>lac</em> p, and the <em>lac</em> operator, <em>lac</em> o. Although it is not strictly standard usage, mutations affecting <em>lac</em> o are referred to as <em>lac</em> oc, for historical reasons.</p>
<p>The <em>lac</em> operon consists of 3 structural genes, and a promoter, a terminator, regulator, and an operator. The three structural genes are: lacZ, lacY, and lacA.</p>
<ul>
<li>lacZ encodes β-galactosidase (LacZ), an intracellular enzyme that cleaves the disaccharide lactose into glucose and galactose.</li>
<li>lacY encodes Beta-galactoside permease (LacY), a transmembrane symporter that pumps β-galactosides including lactose into the cell using a proton gradient in the same direction. Permease increases the permeability of the cell to β-galactosides.</li>
<li>lacA encodes β-galactoside transacetylase (LacA), an enzyme that transfers an acetyl group from acetyl-CoA to β-galactosides.</li>
</ul>
<p>Only lacZ and lacY appear to be necessary for lactose catabolism.</p>
<p>Specific control of the <em>lac</em> genes depends on the availability of the substrate lactose to the bacterium. The proteins are not produced by the bacterium when lactose is unavailable as a carbon source. The <em>lac</em> genes are organized into an operon; that is, they are oriented in the same direction immediately adjacent on the chromosome and are co-transcribed into a single polycistronic mRNA molecule. Transcription of all genes starts with the binding of the enzyme RNA polymerase (RNAP), a DNA-binding protein, which binds to a specific DNA binding site, the promoter, immediately upstream of the genes. Binding of RNA polymerase to the promoter is aided by the cAMP-bound catabolite activator protein (CAP, also known as the cAMP receptor protein). However, the lacI gene (regulatory gene for <em>lac</em> operon) produces a protein that blocks RNAP from binding to the promoter of the operon. This protein can only be removed when allolactose binds to it, and inactivates it. The protein that is formed by the lacI gene is known as the <em>lac</em> repressor. The type of regulation that the <em>lac</em> operon undergoes is referred to as negative inducible, meaning that the gene is turned off by the regulatory factor (lac repressor) unless some molecule (lactose) is added. Because of the presence of the <em>lac</em> repressor protein, genetic engineers who replace the lacZ gene with another gene will have to grow the experimental bacteria on agar with lactose available on it. If they do not, the gene they are trying to express will not be expressed as the repressor protein is still blocking RNAP from binding to the promoter and transcribing the gene. Once the repressor is removed, RNAP then proceeds to transcribe all three genes (lacZYA) into mRNA. Each of the three genes on the mRNA strand has its own Shine-Dalgarno sequence, so the genes are independently translated. The DNA sequence of the <em>E. coli</em> <em>lac</em> operon, the lacZYA mRNA, and the lacI genes are available from GenBank (view).</p>
<p>The first control mechanism is the regulatory response to lactose, which uses an intracellular regulatory protein called the lactose repressor to hinder production of β-galactosidase in the absence of lactose. The lacI gene coding for the repressor lies nearby the <em>lac</em> operon and is always expressed (constitutive). If lactose is missing from the growth medium, the repressor binds very tightly to a short DNA sequence just downstream of the promoter near the beginning of lacZ called the <em>lac</em> operator. The repressor binding to the operator interferes with binding of RNAP to the promoter, and therefore mRNA encoding LacZ and LacY is only made at very low levels. When cells are grown in the presence of lactose, however, a lactose metabolite called allolactose, made from lactose by the product of the lacZ gene, binds to the repressor, causing an allosteric shift. Thus altered, the repressor is unable to bind to the operator, allowing RNAP to transcribe the <em>lac</em> genes and thereby leading to higher levels of the encoded proteins.</p>
<p>The second control mechanism is a response to glucose, which uses the catabolite activator protein (CAP) homodimer to greatly increase production of β-galactosidase in the absence of glucose. Cyclic adenosine monophosphate (cAMP) is a signal molecule whose prevalence is inversely proportional to that of glucose. It binds to the CAP, which in turn allows the CAP to bind to the CAP binding site (a 16 bp DNA sequence upstream of the promoter on the left in the diagram below, about 60 bp upstream of the transcription start site), which assists the RNAP in binding to the DNA. In the absence of glucose, the cAMP concentration is high and binding of CAP-cAMP to the DNA significantly increases the production of β-galactosidase, enabling the cell to hydrolyse lactose and release galactose and glucose.</p>
<p>More recently inducer exclusion was shown to block expression of the <em>lac</em> operon when glucose is present. Glucose is transported into the cell by the PEP-dependent phosphotransferase system. The phosphate group of phosphoenolpyruvate is transferred via a phosphorylation cascade consisting of the general PTS (phosphotransferase system) proteins HPr and EIA and the glucose-specific PTS proteins EIIAGlc and EIIBGlc, the cytoplasmic domain of the EII glucose transporter. Transport of glucose is accompanied by its phosphorylation by EIIBGlc, draining the phosphate group from the other PTS proteins, including EIIAGlc. The unphosphorylated form of EIIAGlc binds to the <em>lac</em> permease and prevents it from bringing lactose into the cell. Therefore, if both glucose and lactose are present, the transport of glucose blocks the transport of the inducer of the <em>lac</em> operon.</p>
<p>The <em>lac</em> repressor is a four-part protein, a tetramer, with identical subunits (Figure <a href="regulation-of-gene-expression.html#fig:lacmodel">11.2</a>). Each subunit contains a helix-turn-helix (HTH) motif capable of binding to DNA. The operator site where repressor binds is a DNA sequence with inverted repeat symmetry. The two DNA half-sites of the operator together bind to two of the subunits of the repressor. Although the other two subunits of repressor are not doing anything in this model, this property was not understood for many years.</p>
<p>Eventually it was discovered that two additional operators are involved in <em>lac</em> regulation. One (O3) lies about −90 bp upstream of O1 in the end of the lacI gene, and the other (O2) is about +410 bp downstream of O1 in the early part of lacZ. These two sites were not found in the early work because they have redundant functions and individual mutations do not affect repression very much. Single mutations to either O2 or O3 have only 2 to 3-fold effects. However, their importance is demonstrated by the fact that a double mutant defective in both O2 and O3 is dramatically de-repressed (by about 70-fold).</p>
<p>In the current model, <em>lac</em> repressor is bound simultaneously to both the main operator O1 and to either O2 or O3. The intervening DNA loops out from the complex. The redundant nature of the two minor operators suggests that it is not a specific looped complex that is important. One idea is that the system works through tethering; if bound repressor releases from O1 momentarily, binding to a minor operator keeps it in the vicinity, so that it may rebind quickly. This would increase the affinity of repressor for O1.</p>

<div class="figure" style="text-align: center"><span id="fig:lacmodel"></span>
<img src="figures/regulation/lac_repressor_model.png" alt="A simulated structural model of a complex between the lac repressor protein (LacI) and a 107-bp-long DNA segment. Two dimeric LacI functional subunits (green +blue and yellow + orange) each bind a DNA operator sequence (top). The mobility of the DNA-binding head groups coupled to the stable body of the body of LacI provide the force for the looping of the DNA (Ville et al.)." width="70%" />
<p class="caption">
Figure 11.2: A simulated structural <a href="https://www.modelarchive.org/doi/10.5452/ma-cwr9z">model</a> of a complex between the <em>lac</em> repressor protein (LacI) and a 107-bp-long DNA segment. Two dimeric LacI functional subunits (green +blue and yellow + orange) each bind a DNA operator sequence (top). The mobility of the DNA-binding head groups coupled to the stable body of the body of LacI provide the force for the looping of the DNA (<a href="https://www.pnas.org/content/102/19/6783">Ville et al.</a>).
</p>
</div>
<p>The repressor is an allosteric protein, i.e. it can assume either one of two slightly different shapes, which are in equilibrium with each other. In one form the repressor will bind to the operator DNA with high specificity, and in the other form it has lost its specificity. According to the classical model of induction, binding of the inducer, either allolactose or IPTG, to the repressor affects the distribution of repressor between the two shapes. Thus, repressor with inducer bound is stabilized in the non-DNA-binding conformation. However, this simple model cannot be the whole story, because repressor is bound quite stably to DNA, yet it is released rapidly by addition of inducer. Therefore, it seems clear that an inducer can also bind to the repressor when the repressor is already bound to DNA. It is still not entirely known what the exact mechanism of binding is.</p>
<p>Non-specific binding of the repressor to DNA plays a crucial role in the repression and induction of the Lac-operon. The specific binding site for the Lac-repressor protein is the operator. The non-specific interaction is mediated mainly by charge-charge interactions while binding to the operator is reinforced by hydrophobic interactions. Additionally, there is an abundance of non-specific DNA sequences to which the repressor can bind. Essentially, any sequence that is not the operator, is considered non-specific. Studies have shown, that without the presence of non-specific binding, induction (or unrepression) of the Lac-operon could not occur even with saturated levels of inducer. It had been demonstrated that, without non-specific binding, the basal level of induction is ten thousand times smaller than observed normally. This is because the non-specific DNA acts as sort of a “sink” for the repressor proteins, distracting them from the operator. The non-specific sequences decrease the amount of available repressor in the cell. This in turn reduces the amount of inducer required to unrepress the system.</p>
<p>A number of lactose derivatives or analogs have been described that are useful for work with the <em>lac</em> operon. These compounds are mainly substituted galactosides, where the glucose moiety of lactose is replaced by another chemical group. Isopropyl-β-D-thiogalactopyranoside (IPTG) is frequently used as an inducer of the <em>lac</em> operon for physiological work. IPTG binds to repressor and inactivates it, but is not a substrate for β-galactosidase. One advantage of IPTG for in vivo studies is that since it cannot be metabolized by <em>E. coli</em> its concentration remains constant and the rate of expression of <em>lac</em> p/o-controlled genes, is not a variable in the experiment. IPTG intake is dependent on the action of lactose permease in P. fluorescens, but not in <em>E. coli</em>.</p>

<div class="figure" style="text-align: center"><span id="fig:iptgstruc"></span>
<img src="figures/regulation/IPTG2.svg" alt="Structure of isopropyl β-D-thiogalactopyranoside (IPTG)" width="70%" />
<p class="caption">
Figure 11.3: <a href="https://commons.wikimedia.org/wiki/File:IPTG2.svg">Structure of isopropyl β-D-thiogalactopyranoside (IPTG)</a>
</p>
</div>
<p>The experimental microorganism used by François Jacob and Jacques Monod was the common laboratory bacterium, <em>E. coli</em>, but many of the basic regulatory concepts that were discovered by Jacob and Monod are fundamental to cellular regulation in all organisms. The key idea is that proteins are not synthesized when they are not needed—E. coli conserves cellular resources and energy by not making the three Lac proteins when there is no need to metabolize lactose, such as when other sugars like glucose are available. The following section discusses how <em>E. coli</em> controls certain genes in response to metabolic needs.</p>
<p>During World War II, Monod was testing the effects of combinations of sugars as nutrient sources for <em>E. coli</em> and B. subtilis. Monod was following up on similar studies that had been conducted by other scientists with bacteria and yeast. He found that bacteria grown with two different sugars often displayed two phases of growth. For example, if glucose and lactose were both provided, glucose was metabolized first (growth phase I, see Figure 2) and then lactose (growth phase II). Lactose was not metabolized during the first part of the diauxic growth curve because β-galactosidase was not made when both glucose and lactose were present in the medium. Monod named this phenomenon diauxie.</p>
<p>Monod then focused his attention on the induction of β-galactosidase formation that occurred when lactose was the sole sugar in the culture medium.</p>
<p>A conceptual breakthrough of Jacob and Monod was to recognize the distinction between regulatory substances and sites where they act to change gene expression. A former soldier, Jacob used the analogy of a bomber that would release its lethal cargo upon receipt of a special radio transmission or signal. A working system requires both a ground transmitter and a receiver in the airplane. Now, suppose that the usual transmitter is broken. This system can be made to work by introduction of a second, functional transmitter. In contrast, he said, consider a bomber with a defective receiver. The behavior of this bomber cannot be changed by introduction of a second, functional aeroplane.</p>
<p>To analyze regulatory mutants of the <em>lac</em> operon, Jacob developed a system by which a second copy of the <em>lac</em> genes (lacI with its promoter, and lacZYA with promoter and operator) could be introduced into a single cell. A culture of such bacteria, which are diploid for the <em>lac</em> genes but otherwise normal, is then tested for the regulatory phenotype. In particular, it is determined whether LacZ and LacY are made even in the absence of IPTG (due to the lactose repressor produced by the mutant gene being non-functional). This experiment, in which genes or gene clusters are tested pairwise, is called a complementation test.</p>

<div class="figure" style="text-align: center"><span id="fig:laccomp"></span>
<img src="figures/regulation/Lac_complementation.png" alt="Analysis of lac regulatory mutants by Lac complementation." width="70%" />
<p class="caption">
Figure 11.4: <a href="https://commons.wikimedia.org/wiki/File:Lac_complementation.png">Analysis of <em>lac</em> regulatory mutants by Lac complementation.</a>
</p>
</div>
<p>This test is illustrated in Figure <a href="regulation-of-gene-expression.html#fig:laccomp">11.4</a> (lacA is omitted for simplicity). First, certain haploid states are shown (i.e. the cell carries only a single copy of the <em>lac</em> genes). Panel (a) shows repression, (b) shows induction by IPTG, and (c) and (d) show the effect of a mutation to the lacI gene or to the operator, respectively. In panel (e) the complementation test for repressor is shown. If one copy of the <em>lac</em> genes carries a mutation in lacI, but the second copy is wild type for lacI, the resulting phenotype is normal—but lacZ is expressed when exposed to inducer IPTG. Mutations affecting repressor are said to be recessive to wild type (and that wild type is dominant), and this is explained by the fact that repressor is a small protein which can diffuse in the cell. The copy of the <em>lac</em> operon adjacent to the defective lacI gene is effectively shut off by protein produced from the second copy of lacI.</p>
<p>If the same experiment is carried out using an operator mutation, a different result is obtained (panel (f)). The phenotype of a cell carrying one mutant and one wild type operator site is that LacZ and LacY are produced even in the absence of the inducer IPTG; because the damaged operator site, does not permit binding of the repressor to inhibit transcription of the structural genes. The operator mutation is dominant. When the operator site where repressor must bind is damaged by mutation, the presence of a second functional site in the same cell makes no difference to expression of genes controlled by the mutant site.</p>
<p>A more sophisticated version of this experiment uses marked operons to distinguish between the two copies of the <em>lac</em> genes and show that the unregulated structural gene(s) is(are) the one(s) next to the mutant operator (panel (g). For example, suppose that one copy is marked by a mutation inactivating lacZ so that it can only produce the LacY protein, while the second copy carries a mutation affecting lacY and can only produce LacZ. In this version, only the copy of the <em>lac</em> operon that is adjacent to the mutant operator is expressed without IPTG. We say that the operator mutation is cis-dominant, it is dominant to wild type but affects only the copy of the operon which is immediately adjacent to it.</p>
<p>This explanation is misleading in an important sense, because it proceeds from a description of the experiment and then explains the results in terms of a model. But in fact, it is often true that the model comes first, and an experiment is fashioned specifically to test the model. Jacob and Monod first imagined that there must be a site in DNA with the properties of the operator, and then designed their complementation tests to show this.</p>
<p>The dominance of operator mutants also suggests a procedure to select them specifically. If regulatory mutants are selected from a culture of wild type using phenyl-Gal, as described above, operator mutations are rare compared to repressor mutants because the target-size is so small. But if instead we start with a strain which carries two copies of the whole <em>lac</em> region (that is diploid for lac), the repressor mutations (which still occur) are not recovered because complementation by the second, wild type lacI gene confers a wild type phenotype. In contrast, mutation of one copy of the operator confers a mutant phenotype because it is dominant to the second, wild type copy.</p>
<p>Explanation of diauxie depended on the characterization of additional mutations affecting the <em>lac</em> genes other than those explained by the classical model. Two other genes, cya and crp, subsequently were identified that mapped far from lac, and that, when mutated, result in a decreased level of expression in the presence of IPTG and even in strains of the bacterium lacking the repressor or operator. The discovery of cAMP in <em>E. coli</em> led to the demonstration that mutants defective the cya gene but not the crp gene could be restored to full activity by the addition of cAMP to the medium.</p>
<p>The cya gene encodes adenylate cyclase, which produces cAMP. In a cya mutant, the absence of cAMP makes the expression of the lacZYA genes more than ten times lower than normal. Addition of cAMP corrects the low Lac expression characteristic of cya mutants. The second gene, crp, encodes a protein called catabolite activator protein (CAP) or cAMP receptor protein (CRP).</p>
<p>However the lactose metabolism enzymes are made in small quantities in the presence of both glucose and lactose (sometimes called leaky expression) due to the fact that the LacI repressor rapidly associates/dissociates from the DNA rather than tightly binding to it, which can allow time for RNAP to bind and transcribe mRNAs of lacZYA. Leaky expression is necessary in order to allow for metabolism of some lactose after the glucose source is expended, but before <em>lac</em> expression is fully activated.</p>
<p>In summary:</p>
<ul>
<li>When lactose is absent then there is very little Lac enzyme production (the operator has Lac repressor bound to it).</li>
<li>When lactose is present but a preferred carbon source (like glucose) is also present then a small amount of enzyme is produced (Lac repressor is not bound to the operator).</li>
<li>When glucose is absent, CAP-cAMP binds to a specific DNA site upstream of the promoter and makes a direct protein-protein interaction with RNAP that facilitates the binding of RNAP to the promoter.</li>
</ul>
<p>The delay between growth phases reflects the time needed to produce sufficient quantities of lactose-metabolizing enzymes. First, the CAP regulatory protein has to assemble on the <em>lac</em> promoter, resulting in an increase in the production of <em>lac</em> mRNA. More available copies of the <em>lac</em> mRNA results in the production (see translation) of significantly more copies of LacZ (β-galactosidase, for lactose metabolism) and LacY (lactose permease to transport lactose into the cell). After a delay needed to increase the level of the lactose metabolizing enzymes, the bacteria enter into a new rapid phase of cell growth.</p>
<p>Two puzzles of catabolite repression relate to how cAMP levels are coupled to the presence of glucose, and secondly, why the cells should even bother. After lactose is cleaved it actually forms glucose and galactose (easily converted to glucose). In metabolic terms, lactose is just as good a carbon and energy source as glucose. The cAMP level is related not to intracellular glucose concentration but to the rate of glucose transport, which influences the activity of adenylate cyclase. (In addition, glucose transport also leads to direct inhibition of the lactose permease.) As to why <em>E. coli</em> works this way, one can only speculate. All enteric bacteria ferment glucose, which suggests they encounter it frequently. It is possible that a small difference in efficiency of transport or metabolism of glucose v. lactose makes it advantageous for cells to regulate the <em>lac</em> operon in this way.</p>
<p>The <em>lac</em> gene and its derivatives are amenable to use as a reporter gene in a number of bacterial-based selection techniques such as two hybrid analysis, in which the successful binding of a transcriptional activator to a specific promoter sequence must be determined. In LB plates containing X-gal, the colour change from white colonies to a shade of blue corresponds to about 20–100 β-galactosidase units, while tetrazolium lactose and MacConkey lactose media have a range of 100–1000 units, being most sensitive in the high and low parts of this range respectively. Since MacConkey lactose and tetrazolium lactose media both rely on the products of lactose breakdown, they require the presence of both lacZ and lacY genes. The many <em>lac</em> fusion techniques which include only the lacZ gene are thus suited to X-gal plates or <a href="https://en.wikipedia.org/wiki/Ortho-Nitrophenyl-%CE%B2-galactoside">ONPG</a> liquid broths. ONPG (ortho-Nitrophenyl-β-galactoside) is a colorimetric and spectrophotometric substrate for detection of β-galactosidase activity. This compound is normally colorless. However, if β-galactosidase is present, it hydrolyzes the ONPG molecule into galactose and ortho-nitrophenol. The latter compound has a yellow color that can be used to check for enzyme activity by means of a colorimetric assay (at 420 nm wavelength). β-Galactosidase is required for lactose utilization, so the intensity of the color produced can be used as a measure of the enzymatic rate. Though ONPG mimics lactose and is hydrolyzed by β-galactosidase, it is unable to act as an inducer for the <em>lac</em> operon. Without another lactose analog that can act as an inducer, such as isopropyl β-D-1-thiogalactopyranoside (IPTG), β-galactosidase will not be transcribed and ONPG will not be hydrolyzed.</p>
</div>
<div id="the-trp-operon" class="section level2">
<h2><span class="header-section-number">11.2</span> The <em>trp</em> operon</h2>
<p>Discovered in 1953 by Jacques Monod and colleagues, the <em>trp</em> operon in <em>E. coli</em> was the first repressible operon to be discovered. While the lac operon can be activated by a chemical (allolactose), the tryptophan (Trp) operon is inhibited by a chemical (tryptophan). This operon contains five structural genes: <em>trp</em> E, <em>trp</em> D, <em>trp</em> C, <em>trp</em> B, and <em>trp</em> A, which encodes tryptophan synthetase. It also contains a promoter which binds to RNA polymerase and an operator which blocks transcription when bound to the protein synthesized by the repressor gene (trp R) that binds to the operator. In the lac operon, lactose binds to the repressor protein and prevents it from repressing gene transcription, while in the <em>trp</em> operon, tryptophan binds to the repressor protein and enables it to repress gene transcription. Also unlike the lac operon, the <em>trp</em> operon contains a leader peptide and an attenuator sequence which allows for graded regulation. This is an example of the corepressible model.</p>
</div>
<div id="regulated-stages-of-gene-expression" class="section level2">
<h2><span class="header-section-number">11.3</span> Regulated stages of gene expression</h2>
<p>Any step of gene expression may be modulated, from the DNA-RNA transcription step to post-translational modification of a protein. The following is a list of stages where gene expression is regulated, the most extensively utilised point is Transcription Initiation:</p>
<ul>
<li>Chromatin domains</li>
<li>Transcription</li>
<li>Post-transcriptional modification</li>
<li>RNA transport</li>
<li>Translation</li>
<li>mRNA degradation</li>
<li>Modification of DNA</li>
</ul>
<p>In eukaryotes, the accessibility of large regions of DNA can depend on its chromatin structure, which can be altered as a result of histone modifications directed by DNA methylation, ncRNA, or DNA-binding protein. Hence these modifications may up or down regulate the expression of a gene. Some of these modifications that regulate gene expression are inheritable and are referred to as epigenetic regulation.</p>
<div id="regulation-of-transciption-by-dna-packing" class="section level3">
<h3><span class="header-section-number">11.3.1</span> Regulation of transciption by DNA packing</h3>
<p>Transcription of DNA is dictated by its structure. In general, the density of its packing is indicative of the frequency of transcription. Octameric protein complexes called nucleosomes are responsible for the amount of supercoiling of DNA, and these complexes can be temporarily modified by processes such as phosphorylation or more permanently modified by processes such as methylation. Such modifications are considered to be responsible for more or less permanent changes in gene expression levels.</p>
</div>
<div id="regulation-of-transciption-by-dna-modification" class="section level3">
<h3><span class="header-section-number">11.3.2</span> Regulation of transciption by DNA modification</h3>
<p>Methylation of DNA is a common method of gene silencing. DNA is typically methylated by methyltransferase enzymes on cytosine nucleotides in a CpG dinucleotide sequence (also called “CpG islands” when densely clustered). Analysis of the pattern of methylation in a given region of DNA (which can be a promoter) can be achieved through a method called bisulfite mapping. Methylated cytosine residues are unchanged by the treatment, whereas unmethylated ones are changed to uracil. The differences are analyzed by DNA sequencing or by methods developed to quantify SNPs, such as Pyrosequencing (Biotage) or MassArray (Sequenom), measuring the relative amounts of C/T at the CG dinucleotide. Abnormal methylation patterns are thought to be involved in oncogenesis.</p>
<p>Histone acetylation is also an important process in transcription. Histone acetyltransferase enzymes (HATs) such as CREB-binding protein also dissociate the DNA from the histone complex, allowing transcription to proceed. Often, DNA methylation and histone deacetylation work together in gene silencing. The combination of the two seems to be a signal for DNA to be packed more densely, lowering gene expression.[citation needed]</p>
</div>
<div id="regulation-of-rna-polymerase" class="section level3">
<h3><span class="header-section-number">11.3.3</span> Regulation of RNA polymerase</h3>
<p>Regulation of transcription thus controls when transcription occurs and how much RNA is created. Transcription of a gene by RNA polymerase can be regulated by several mechanisms. Specificity factors alter the specificity of RNA polymerase for a given promoter or set of promoters, making it more or less likely to bind to them (i.e., sigma factors used in prokaryotic transcription). Repressors bind to the Operator, coding sequences on the DNA strand that are close to or overlapping the promoter region, impeding RNA polymerase’s progress along the strand, thus impeding the expression of the gene. The image to the right demonstrates regulation by a repressor in the <em>lac</em> operon. General transcription factors position RNA polymerase at the start of a protein-coding sequence and then release the polymerase to transcribe the mRNA. Activators enhance the interaction between RNA polymerase and a particular promoter, encouraging the expression of the gene. Activators do this by increasing the attraction of RNA polymerase for the promoter, through interactions with subunits of the RNA polymerase or indirectly by changing the structure of the DNA. Enhancers are sites on the DNA helix that are bound by activators in order to loop the DNA bringing a specific promoter to the initiation complex. Enhancers are much more common in eukaryotes than prokaryotes, where only a few examples exist (to date). Silencers are regions of DNA sequences that, when bound by particular transcription factors, can silence expression of the gene.</p>
</div>
</div>
<div id="post-transcriptional-regulation" class="section level2">
<h2><span class="header-section-number">11.4</span> Post-transcriptional regulation</h2>
<p>After the DNA is transcribed and mRNA is formed, there must be some sort of regulation on how much the mRNA is translated into proteins. Cells do this by modulating the capping, splicing, addition of a Poly(A) Tail, the sequence-specific nuclear export rates, and, in several contexts, sequestration of the RNA transcript. These processes occur in eukaryotes but not in prokaryotes. This modulation is a result of a protein or transcript that, in turn, is regulated and may have an affinity for certain sequences.</p>
<div id="three-prime-untranslated-regions-and-micrornas" class="section level3">
<h3><span class="header-section-number">11.4.1</span> Three prime untranslated regions and microRNAs</h3>
<p>Three prime untranslated regions (3’-UTRs) of messenger RNAs (mRNAs) often contain regulatory sequences that post-transcriptionally influence gene expression. Such 3’-UTRs often contain both binding sites for microRNAs (miRNAs) as well as for regulatory proteins. By binding to specific sites within the 3’-UTR, miRNAs can decrease gene expression of various mRNAs by either inhibiting translation or directly causing degradation of the transcript. The 3’-UTR also may have silencer regions that bind repressor proteins that inhibit the expression of a mRNA.</p>
<p>The 3’-UTR often contains miRNA response elements (MREs). MREs are sequences to which miRNAs bind. These are prevalent motifs within 3’-UTRs. Among all regulatory motifs within the 3’-UTRs (e.g. including silencer regions), MREs make up about half of the motifs.</p>
<p>As of 2014, the miRBase web site, an archive of miRNA sequences and annotations, listed 28,645 entries in 233 biologic species. Of these, 1,881 miRNAs were in annotated human miRNA loci. miRNAs were predicted to have an average of about four hundred target mRNAs (affecting expression of several hundred genes). Freidman et al. estimate that &gt;45,000 miRNA target sites within human mRNA 3’-UTRs are conserved above background levels, and &gt;60% of human protein-coding genes have been under selective pressure to maintain pairing to miRNAs.</p>
<p>Direct experiments show that a single miRNA can reduce the stability of hundreds of unique mRNAs. Other experiments show that a single miRNA may repress the production of hundreds of proteins, but that this repression often is relatively mild (less than 2-fold).</p>
<p>The effects of miRNA dysregulation of gene expression seem to be important in cancer. For instance, in gastrointestinal cancers, a 2015 paper identified nine miRNAs as epigenetically altered and effective in down-regulating DNA repair enzymes.</p>
<p>The effects of miRNA dysregulation of gene expression also seem to be important in neuropsychiatric disorders, such as schizophrenia, bipolar disorder, major depressive disorder, Parkinson’s disease, Alzheimer’s disease and autism spectrum disorders.</p>
</div>
</div>
<div id="regulation-of-translation" class="section level2">
<h2><span class="header-section-number">11.5</span> Regulation of translation</h2>
<p>The translation of mRNA can also be controlled by a number of mechanisms, mostly at the level of initiation. Recruitment of the small ribosomal subunit can indeed be modulated by mRNA secondary structure, antisense RNA binding, or protein binding. In both prokaryotes and eukaryotes, a large number of RNA binding proteins exist, which often are directed to their target sequence by the secondary structure of the transcript, which may change depending on certain conditions, such as temperature or presence of a ligand (aptamer). Some transcripts act as ribozymes and self-regulate their expression.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="molecular-biology-of-the-gene.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/biology-text/edit/master/gene_regulation.rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
