<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>25 The Nervous System | Text For Biology at Roxbury Community College</title>
  <meta name="description" content="This is a free online Text for Biology" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="25 The Nervous System | Text For Biology at Roxbury Community College" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="biology_text_cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Biology" />
  <meta name="github-repo" content="nikolaussuchert/biology-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="25 The Nervous System | Text For Biology at Roxbury Community College" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Biology" />
  <meta name="twitter:image" content="biology_text_cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="the-excretory-system.html"/>
<link rel="next" href="animal-locomotion.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>



<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Biology</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html"><i class="fa fa-check"></i><b>1</b> Biology: The Science of Life</a><ul>
<li class="chapter" data-level="1.1" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#characteristics-of-life"><i class="fa fa-check"></i><b>1.1</b> Characteristics of Life</a></li>
<li class="chapter" data-level="1.2" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#foundations-of-modern-biology"><i class="fa fa-check"></i><b>1.2</b> Foundations of Modern Biology</a></li>
<li class="chapter" data-level="1.3" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#cell-theory"><i class="fa fa-check"></i><b>1.3</b> Cell theory</a></li>
<li class="chapter" data-level="1.4" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#evolution"><i class="fa fa-check"></i><b>1.4</b> Evolution</a></li>
<li class="chapter" data-level="1.5" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#genetics"><i class="fa fa-check"></i><b>1.5</b> Genetics</a></li>
<li class="chapter" data-level="1.6" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#homeostasis"><i class="fa fa-check"></i><b>1.6</b> Homeostasis</a></li>
<li class="chapter" data-level="1.7" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#energy"><i class="fa fa-check"></i><b>1.7</b> Energy</a></li>
<li class="chapter" data-level="1.8" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#areas-and-levels-of-study-and-research-in-biology"><i class="fa fa-check"></i><b>1.8</b> Areas And Levels of Study And Research in Biology</a><ul>
<li class="chapter" data-level="1.8.1" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#structural"><i class="fa fa-check"></i><b>1.8.1</b> Structural</a></li>
<li class="chapter" data-level="1.8.2" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#physiological"><i class="fa fa-check"></i><b>1.8.2</b> Physiological</a></li>
<li class="chapter" data-level="1.8.3" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#evolutionary"><i class="fa fa-check"></i><b>1.8.3</b> Evolutionary</a></li>
<li class="chapter" data-level="1.8.4" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#systematic"><i class="fa fa-check"></i><b>1.8.4</b> Systematic</a></li>
<li class="chapter" data-level="1.8.5" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#ecological-and-environmental"><i class="fa fa-check"></i><b>1.8.5</b> Ecological And Environmental</a></li>
</ul></li>
<li class="chapter" data-level="1.9" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#science"><i class="fa fa-check"></i><b>1.9</b> Science</a></li>
<li class="chapter" data-level="1.10" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#the-scientific-method"><i class="fa fa-check"></i><b>1.10</b> The Scientific Method</a></li>
<li class="chapter" data-level="1.11" data-path="biology-the-science-of-life.html"><a href="biology-the-science-of-life.html#philosophy-of-science"><i class="fa fa-check"></i><b>1.11</b> Philosophy of Science</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html"><i class="fa fa-check"></i><b>2</b> Basic General Chemistry For Biology</a><ul>
<li class="chapter" data-level="2.1" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#matter"><i class="fa fa-check"></i><b>2.1</b> Matter</a><ul>
<li class="chapter" data-level="2.1.1" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#atom"><i class="fa fa-check"></i><b>2.1.1</b> Atom</a></li>
<li class="chapter" data-level="2.1.2" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#element"><i class="fa fa-check"></i><b>2.1.2</b> Element</a></li>
<li class="chapter" data-level="2.1.3" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#compound"><i class="fa fa-check"></i><b>2.1.3</b> Compound</a></li>
<li class="chapter" data-level="2.1.4" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#molecule"><i class="fa fa-check"></i><b>2.1.4</b> Molecule</a></li>
<li class="chapter" data-level="2.1.5" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#chemical-bonds"><i class="fa fa-check"></i><b>2.1.5</b> Chemical Bonds</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#energy-1"><i class="fa fa-check"></i><b>2.2</b> Energy</a></li>
<li class="chapter" data-level="2.3" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#chemical-reactions"><i class="fa fa-check"></i><b>2.3</b> Chemical Reactions</a></li>
<li class="chapter" data-level="2.4" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#water"><i class="fa fa-check"></i><b>2.4</b> Water</a></li>
<li class="chapter" data-level="2.5" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#ions-and-salts"><i class="fa fa-check"></i><b>2.5</b> Ions And Salts</a></li>
<li class="chapter" data-level="2.6" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#acids-and-bases"><i class="fa fa-check"></i><b>2.6</b> Acids And Bases</a></li>
<li class="chapter" data-level="2.7" data-path="basic-general-chemistry-for-biology.html"><a href="basic-general-chemistry-for-biology.html#redox"><i class="fa fa-check"></i><b>2.7</b> Redox</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="basic-organic-chemistry-for-biology.html"><a href="basic-organic-chemistry-for-biology.html"><i class="fa fa-check"></i><b>3</b> Basic Organic Chemistry For Biology</a><ul>
<li class="chapter" data-level="3.1" data-path="basic-organic-chemistry-for-biology.html"><a href="basic-organic-chemistry-for-biology.html#biomolecules"><i class="fa fa-check"></i><b>3.1</b> Biomolecules</a></li>
<li class="chapter" data-level="3.2" data-path="basic-organic-chemistry-for-biology.html"><a href="basic-organic-chemistry-for-biology.html#proteins"><i class="fa fa-check"></i><b>3.2</b> Proteins</a><ul>
<li class="chapter" data-level="3.2.1" data-path="basic-organic-chemistry-for-biology.html"><a href="basic-organic-chemistry-for-biology.html#structure"><i class="fa fa-check"></i><b>3.2.1</b> Structure</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="basic-organic-chemistry-for-biology.html"><a href="basic-organic-chemistry-for-biology.html#carbohydrates"><i class="fa fa-check"></i><b>3.3</b> Carbohydrates</a></li>
<li class="chapter" data-level="3.4" data-path="basic-organic-chemistry-for-biology.html"><a href="basic-organic-chemistry-for-biology.html#lipids"><i class="fa fa-check"></i><b>3.4</b> Lipids</a></li>
<li class="chapter" data-level="3.5" data-path="basic-organic-chemistry-for-biology.html"><a href="basic-organic-chemistry-for-biology.html#nucleic-acids"><i class="fa fa-check"></i><b>3.5</b> Nucleic Acids</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="the-cell.html"><a href="the-cell.html"><i class="fa fa-check"></i><b>4</b> The Cell</a><ul>
<li class="chapter" data-level="4.1" data-path="the-cell.html"><a href="the-cell.html#prokaryotic-cells"><i class="fa fa-check"></i><b>4.1</b> Prokaryotic Cells</a></li>
<li class="chapter" data-level="4.2" data-path="the-cell.html"><a href="the-cell.html#eukaryotic-cells"><i class="fa fa-check"></i><b>4.2</b> Eukaryotic Cells</a></li>
<li class="chapter" data-level="4.3" data-path="the-cell.html"><a href="the-cell.html#subcellular-components"><i class="fa fa-check"></i><b>4.3</b> Subcellular Components</a><ul>
<li class="chapter" data-level="4.3.1" data-path="the-cell.html"><a href="the-cell.html#the-cell-membrane"><i class="fa fa-check"></i><b>4.3.1</b> The Cell Membrane</a></li>
<li class="chapter" data-level="4.3.2" data-path="the-cell.html"><a href="the-cell.html#the-cytoskeleton"><i class="fa fa-check"></i><b>4.3.2</b> The Cytoskeleton</a></li>
<li class="chapter" data-level="4.3.3" data-path="the-cell.html"><a href="the-cell.html#the-genetic-material"><i class="fa fa-check"></i><b>4.3.3</b> The Genetic Material</a></li>
<li class="chapter" data-level="4.3.4" data-path="the-cell.html"><a href="the-cell.html#organelles-of-eukaryotic-cells"><i class="fa fa-check"></i><b>4.3.4</b> Organelles of Eukaryotic Cells</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="the-cell.html"><a href="the-cell.html#the-cell-nucleus"><i class="fa fa-check"></i><b>4.4</b> The Cell Nucleus</a></li>
<li class="chapter" data-level="4.5" data-path="the-cell.html"><a href="the-cell.html#the-mitochondria"><i class="fa fa-check"></i><b>4.5</b> The Mitochondria</a></li>
<li class="chapter" data-level="4.6" data-path="the-cell.html"><a href="the-cell.html#the-chloroplasts"><i class="fa fa-check"></i><b>4.6</b> The Chloroplasts</a></li>
<li class="chapter" data-level="4.7" data-path="the-cell.html"><a href="the-cell.html#the-endoplasmic-reticulum"><i class="fa fa-check"></i><b>4.7</b> The Endoplasmic Reticulum</a></li>
<li class="chapter" data-level="4.8" data-path="the-cell.html"><a href="the-cell.html#the-golgi-apparatus"><i class="fa fa-check"></i><b>4.8</b> The Golgi Apparatus</a></li>
<li class="chapter" data-level="4.9" data-path="the-cell.html"><a href="the-cell.html#the-ribosomes"><i class="fa fa-check"></i><b>4.9</b> The Ribosomes</a><ul>
<li class="chapter" data-level="4.9.1" data-path="the-cell.html"><a href="the-cell.html#structures-outside-the-cell-membrane"><i class="fa fa-check"></i><b>4.9.1</b> Structures Outside The Cell Membrane</a></li>
<li class="chapter" data-level="4.9.2" data-path="the-cell.html"><a href="the-cell.html#cell-wall"><i class="fa fa-check"></i><b>4.9.2</b> Cell wall</a></li>
<li class="chapter" data-level="4.9.3" data-path="the-cell.html"><a href="the-cell.html#prokaryotic"><i class="fa fa-check"></i><b>4.9.3</b> Prokaryotic</a></li>
<li class="chapter" data-level="4.9.4" data-path="the-cell.html"><a href="the-cell.html#capsule"><i class="fa fa-check"></i><b>4.9.4</b> Capsule</a></li>
<li class="chapter" data-level="4.9.5" data-path="the-cell.html"><a href="the-cell.html#flagella"><i class="fa fa-check"></i><b>4.9.5</b> Flagella</a></li>
<li class="chapter" data-level="4.9.6" data-path="the-cell.html"><a href="the-cell.html#fimbriae"><i class="fa fa-check"></i><b>4.9.6</b> Fimbriae</a></li>
</ul></li>
<li class="chapter" data-level="4.10" data-path="the-cell.html"><a href="the-cell.html#cellular-replication"><i class="fa fa-check"></i><b>4.10</b> Cellular Replication</a></li>
<li class="chapter" data-level="4.11" data-path="the-cell.html"><a href="the-cell.html#dna-repair"><i class="fa fa-check"></i><b>4.11</b> DNA repair</a></li>
<li class="chapter" data-level="4.12" data-path="the-cell.html"><a href="the-cell.html#cellular-growth-and-metabolism"><i class="fa fa-check"></i><b>4.12</b> Cellular Growth And Metabolism</a></li>
<li class="chapter" data-level="4.13" data-path="the-cell.html"><a href="the-cell.html#protein-synthesis"><i class="fa fa-check"></i><b>4.13</b> Protein Synthesis</a></li>
<li class="chapter" data-level="4.14" data-path="the-cell.html"><a href="the-cell.html#cell-motility"><i class="fa fa-check"></i><b>4.14</b> Cell Motility</a></li>
<li class="chapter" data-level="4.15" data-path="the-cell.html"><a href="the-cell.html#origin-of-the-first-cell"><i class="fa fa-check"></i><b>4.15</b> Origin of The First Cell</a></li>
<li class="chapter" data-level="4.16" data-path="the-cell.html"><a href="the-cell.html#origin-of-eukaryotic-cells"><i class="fa fa-check"></i><b>4.16</b> Origin of Eukaryotic Cells</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html"><i class="fa fa-check"></i><b>5</b> The Cell Membrane</a><ul>
<li class="chapter" data-level="5.1" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#function"><i class="fa fa-check"></i><b>5.1</b> Function</a></li>
<li class="chapter" data-level="5.2" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#the-fluid-mosaic-model-of-the-cell-membrane"><i class="fa fa-check"></i><b>5.2</b> The Fluid Mosaic Model of The Cell Membrane</a><ul>
<li class="chapter" data-level="5.2.1" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#lipid-bilayer"><i class="fa fa-check"></i><b>5.2.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="5.2.2" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#membrane-polarity"><i class="fa fa-check"></i><b>5.2.2</b> Membrane Polarity</a></li>
<li class="chapter" data-level="5.2.3" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#membrane-structures"><i class="fa fa-check"></i><b>5.2.3</b> Membrane Structures</a></li>
<li class="chapter" data-level="5.2.4" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#cytoskeleton"><i class="fa fa-check"></i><b>5.2.4</b> Cytoskeleton</a></li>
<li class="chapter" data-level="5.2.5" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#intracellular-membranes-in-eukaryotic-cells"><i class="fa fa-check"></i><b>5.2.5</b> Intracellular Membranes In Eukaryotic Cells</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#membrane-permeability"><i class="fa fa-check"></i><b>5.3</b> Membrane Permeability</a></li>
<li class="chapter" data-level="5.4" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#diffusion"><i class="fa fa-check"></i><b>5.4</b> Diffusion</a></li>
<li class="chapter" data-level="5.5" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#facilitated-diffusion"><i class="fa fa-check"></i><b>5.5</b> Facilitated Diffusion</a></li>
<li class="chapter" data-level="5.6" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#osmosis"><i class="fa fa-check"></i><b>5.6</b> Osmosis</a></li>
<li class="chapter" data-level="5.7" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#active-transport"><i class="fa fa-check"></i><b>5.7</b> Active Transport</a></li>
<li class="chapter" data-level="5.8" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#bulk-transport"><i class="fa fa-check"></i><b>5.8</b> Bulk transport</a></li>
<li class="chapter" data-level="5.9" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#the-extracellular-matrix"><i class="fa fa-check"></i><b>5.9</b> The Extracellular Matrix</a></li>
<li class="chapter" data-level="5.10" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#cell-junctions"><i class="fa fa-check"></i><b>5.10</b> Cell Junctions</a><ul>
<li class="chapter" data-level="5.10.1" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#anchoring-junctions"><i class="fa fa-check"></i><b>5.10.1</b> Anchoring Junctions</a></li>
<li class="chapter" data-level="5.10.2" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#desmosomes"><i class="fa fa-check"></i><b>5.10.2</b> Desmosomes</a></li>
<li class="chapter" data-level="5.10.3" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#hemidesmosomes"><i class="fa fa-check"></i><b>5.10.3</b> Hemidesmosomes</a></li>
<li class="chapter" data-level="5.10.4" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#adherens-junctions"><i class="fa fa-check"></i><b>5.10.4</b> Adherens Junctions</a></li>
<li class="chapter" data-level="5.10.5" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#communicating-gap-junctions"><i class="fa fa-check"></i><b>5.10.5</b> Communicating (gap) Junctions</a></li>
<li class="chapter" data-level="5.10.6" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#tight-junctions"><i class="fa fa-check"></i><b>5.10.6</b> Tight Junctions</a></li>
</ul></li>
<li class="chapter" data-level="5.11" data-path="the-cell-membrane-1.html"><a href="the-cell-membrane-1.html#cell-junction-molecules"><i class="fa fa-check"></i><b>5.11</b> Cell Junction Molecules</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="bioenergetics.html"><a href="bioenergetics.html"><i class="fa fa-check"></i><b>6</b> Bioenergetics</a><ul>
<li class="chapter" data-level="6.1" data-path="bioenergetics.html"><a href="bioenergetics.html#thermodynamics-of-living-organisms"><i class="fa fa-check"></i><b>6.1</b> Thermodynamics of Living Organisms</a></li>
<li class="chapter" data-level="6.2" data-path="bioenergetics.html"><a href="bioenergetics.html#metabolism"><i class="fa fa-check"></i><b>6.2</b> Metabolism</a></li>
<li class="chapter" data-level="6.3" data-path="bioenergetics.html"><a href="bioenergetics.html#catabolism"><i class="fa fa-check"></i><b>6.3</b> Catabolism</a></li>
<li class="chapter" data-level="6.4" data-path="bioenergetics.html"><a href="bioenergetics.html#digestion"><i class="fa fa-check"></i><b>6.4</b> Digestion</a></li>
<li class="chapter" data-level="6.5" data-path="bioenergetics.html"><a href="bioenergetics.html#energy-from-organic-compounds"><i class="fa fa-check"></i><b>6.5</b> Energy From Organic Compounds</a></li>
<li class="chapter" data-level="6.6" data-path="bioenergetics.html"><a href="bioenergetics.html#energy-transformations"><i class="fa fa-check"></i><b>6.6</b> Energy Transformations</a></li>
<li class="chapter" data-level="6.7" data-path="bioenergetics.html"><a href="bioenergetics.html#oxidative-phosphorylation"><i class="fa fa-check"></i><b>6.7</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="6.8" data-path="bioenergetics.html"><a href="bioenergetics.html#energy-from-inorganic-compounds"><i class="fa fa-check"></i><b>6.8</b> Energy From Inorganic Compounds</a></li>
<li class="chapter" data-level="6.9" data-path="bioenergetics.html"><a href="bioenergetics.html#energy-from-light"><i class="fa fa-check"></i><b>6.9</b> Energy From Light</a></li>
<li class="chapter" data-level="6.10" data-path="bioenergetics.html"><a href="bioenergetics.html#anabolism"><i class="fa fa-check"></i><b>6.10</b> Anabolism</a></li>
<li class="chapter" data-level="6.11" data-path="bioenergetics.html"><a href="bioenergetics.html#carbon-fixation"><i class="fa fa-check"></i><b>6.11</b> Carbon Fixation</a><ul>
<li class="chapter" data-level="6.11.1" data-path="bioenergetics.html"><a href="bioenergetics.html#carbohydrates-and-glycans"><i class="fa fa-check"></i><b>6.11.1</b> Carbohydrates And Glycans</a></li>
<li class="chapter" data-level="6.11.2" data-path="bioenergetics.html"><a href="bioenergetics.html#fatty-acids-isoprenoids-and-sterol"><i class="fa fa-check"></i><b>6.11.2</b> Fatty Acids, Isoprenoids and Sterol</a></li>
<li class="chapter" data-level="6.11.3" data-path="bioenergetics.html"><a href="bioenergetics.html#proteins-1"><i class="fa fa-check"></i><b>6.11.3</b> Proteins</a></li>
<li class="chapter" data-level="6.11.4" data-path="bioenergetics.html"><a href="bioenergetics.html#nucleotide-synthesis-and-salvage"><i class="fa fa-check"></i><b>6.11.4</b> Nucleotide Synthesis And Salvage</a></li>
</ul></li>
<li class="chapter" data-level="6.12" data-path="bioenergetics.html"><a href="bioenergetics.html#enzymes"><i class="fa fa-check"></i><b>6.12</b> Enzymes</a></li>
<li class="chapter" data-level="6.13" data-path="bioenergetics.html"><a href="bioenergetics.html#allosteric-modulation"><i class="fa fa-check"></i><b>6.13</b> Allosteric Modulation</a></li>
<li class="chapter" data-level="6.14" data-path="bioenergetics.html"><a href="bioenergetics.html#cofactors-of-enzymes-and-coenzymes"><i class="fa fa-check"></i><b>6.14</b> Cofactors of Enzymes and Coenzymes</a></li>
<li class="chapter" data-level="6.15" data-path="bioenergetics.html"><a href="bioenergetics.html#inhibition-of-enzymes"><i class="fa fa-check"></i><b>6.15</b> Inhibition of Enzymes</a></li>
<li class="chapter" data-level="6.16" data-path="bioenergetics.html"><a href="bioenergetics.html#factors-affecting-enzyme-activity"><i class="fa fa-check"></i><b>6.16</b> Factors Affecting Enzyme Activity</a></li>
<li class="chapter" data-level="6.17" data-path="bioenergetics.html"><a href="bioenergetics.html#control-of-activity"><i class="fa fa-check"></i><b>6.17</b> Control of activity</a><ul>
<li class="chapter" data-level="6.17.1" data-path="bioenergetics.html"><a href="bioenergetics.html#post-translational-modification"><i class="fa fa-check"></i><b>6.17.1</b> Post-translational Modification</a></li>
<li class="chapter" data-level="6.17.2" data-path="bioenergetics.html"><a href="bioenergetics.html#quantity"><i class="fa fa-check"></i><b>6.17.2</b> Quantity</a></li>
<li class="chapter" data-level="6.17.3" data-path="bioenergetics.html"><a href="bioenergetics.html#subcellular-distribution"><i class="fa fa-check"></i><b>6.17.3</b> Subcellular Distribution</a></li>
<li class="chapter" data-level="6.17.4" data-path="bioenergetics.html"><a href="bioenergetics.html#organ-specialization"><i class="fa fa-check"></i><b>6.17.4</b> Organ Specialization</a></li>
<li class="chapter" data-level="6.17.5" data-path="bioenergetics.html"><a href="bioenergetics.html#coenzymes"><i class="fa fa-check"></i><b>6.17.5</b> Coenzymes</a></li>
<li class="chapter" data-level="6.17.6" data-path="bioenergetics.html"><a href="bioenergetics.html#mineral-cofactors"><i class="fa fa-check"></i><b>6.17.6</b> Mineral Cofactors</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html"><i class="fa fa-check"></i><b>7</b> Photosynthesis And Cellular Respiration</a><ul>
<li class="chapter" data-level="7.1" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#the-electron-transfer-chain-and-chemiosmosis"><i class="fa fa-check"></i><b>7.1</b> The Electron Transfer Chain And Chemiosmosis</a></li>
<li class="chapter" data-level="7.2" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#photosynthesis"><i class="fa fa-check"></i><b>7.2</b> Photosynthesis</a></li>
<li class="chapter" data-level="7.3" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#photosynthetic-membranes-and-organelles"><i class="fa fa-check"></i><b>7.3</b> Photosynthetic Membranes And Organelles</a></li>
<li class="chapter" data-level="7.4" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#light-dependent-reactions"><i class="fa fa-check"></i><b>7.4</b> Light-dependent Reactions</a></li>
<li class="chapter" data-level="7.5" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#z-scheme"><i class="fa fa-check"></i><b>7.5</b> Z Scheme</a></li>
<li class="chapter" data-level="7.6" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#water-photolysis"><i class="fa fa-check"></i><b>7.6</b> Water Photolysis</a></li>
<li class="chapter" data-level="7.7" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#light-independent-reactions"><i class="fa fa-check"></i><b>7.7</b> Light-independent Reactions</a></li>
<li class="chapter" data-level="7.8" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#calvin-cycle"><i class="fa fa-check"></i><b>7.8</b> Calvin Cycle</a></li>
<li class="chapter" data-level="7.9" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#carbon-concentrating-mechanisms"><i class="fa fa-check"></i><b>7.9</b> Carbon Concentrating Mechanisms</a></li>
<li class="chapter" data-level="7.10" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#overview-of-c4-carbon-fixation"><i class="fa fa-check"></i><b>7.10</b> Overview of C4 Carbon Fixation</a></li>
<li class="chapter" data-level="7.11" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#cam-photosynthesis"><i class="fa fa-check"></i><b>7.11</b> CAM Photosynthesis</a></li>
<li class="chapter" data-level="7.12" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#symbiosis-and-the-origin-of-chloroplasts"><i class="fa fa-check"></i><b>7.12</b> Symbiosis And The Origin of Chloroplasts</a></li>
<li class="chapter" data-level="7.13" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#cyanobacteria-and-the-evolution-of-photosynthesis"><i class="fa fa-check"></i><b>7.13</b> Cyanobacteria And The Evolution of Photosynthesis</a></li>
<li class="chapter" data-level="7.14" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#cellular-respiration"><i class="fa fa-check"></i><b>7.14</b> Cellular Respiration</a></li>
<li class="chapter" data-level="7.15" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#aerobic-respiration"><i class="fa fa-check"></i><b>7.15</b> Aerobic Respiration</a></li>
<li class="chapter" data-level="7.16" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#glycolysis"><i class="fa fa-check"></i><b>7.16</b> Glycolysis</a></li>
<li class="chapter" data-level="7.17" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#oxidative-decarboxylation-of-pyruvate"><i class="fa fa-check"></i><b>7.17</b> Oxidative Decarboxylation of Pyruvate</a></li>
<li class="chapter" data-level="7.18" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#the-citric-acid-cycle"><i class="fa fa-check"></i><b>7.18</b> The Citric Acid Cycle</a></li>
<li class="chapter" data-level="7.19" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#oxidative-phosphorylation-1"><i class="fa fa-check"></i><b>7.19</b> Oxidative Phosphorylation</a></li>
<li class="chapter" data-level="7.20" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#fermentation"><i class="fa fa-check"></i><b>7.20</b> Fermentation</a></li>
<li class="chapter" data-level="7.21" data-path="photosynthesis-and-cellular-respiration.html"><a href="photosynthesis-and-cellular-respiration.html#anaerobic-respiration"><i class="fa fa-check"></i><b>7.21</b> Anaerobic respiration</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html"><i class="fa fa-check"></i><b>8</b> The Cell Cycle And Cell Division</a><ul>
<li class="chapter" data-level="8.1" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#the-prokaryotic-cell-cycle"><i class="fa fa-check"></i><b>8.1</b> The Prokaryotic Cell Cycle</a></li>
<li class="chapter" data-level="8.2" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#binary-fission"><i class="fa fa-check"></i><b>8.2</b> Binary Fission</a></li>
<li class="chapter" data-level="8.3" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#the-eukaryotic-cell-cycle"><i class="fa fa-check"></i><b>8.3</b> The Eukaryotic Cell Cycle</a></li>
<li class="chapter" data-level="8.4" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#g0-phase-quiescence"><i class="fa fa-check"></i><b>8.4</b> G0 phase (quiescence)</a></li>
<li class="chapter" data-level="8.5" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#interphase"><i class="fa fa-check"></i><b>8.5</b> Interphase</a></li>
<li class="chapter" data-level="8.6" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#g1-phase-first-growth-phase-or-post-mitotic-gap-phase"><i class="fa fa-check"></i><b>8.6</b> G1 phase (First growth phase or Post mitotic gap phase)</a></li>
<li class="chapter" data-level="8.7" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#s-phase-dna-replication"><i class="fa fa-check"></i><b>8.7</b> S phase (DNA replication)</a></li>
<li class="chapter" data-level="8.8" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#g2-phase-growth"><i class="fa fa-check"></i><b>8.8</b> G2 phase (growth)</a></li>
<li class="chapter" data-level="8.9" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#mitotic-phase-chromosome-separation"><i class="fa fa-check"></i><b>8.9</b> Mitotic Phase (Chromosome Separation)</a></li>
<li class="chapter" data-level="8.10" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#chromosomes-the-vectors-of-heredity"><i class="fa fa-check"></i><b>8.10</b> Chromosomes: The Vectors of Heredity</a><ul>
<li class="chapter" data-level="8.10.1" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#interphase-chromatin"><i class="fa fa-check"></i><b>8.10.1</b> Interphase chromatin</a></li>
<li class="chapter" data-level="8.10.2" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#metaphase-chromatin-and-division"><i class="fa fa-check"></i><b>8.10.2</b> Metaphase chromatin and division</a></li>
</ul></li>
<li class="chapter" data-level="8.11" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#human-chromosomes"><i class="fa fa-check"></i><b>8.11</b> Human chromosomes</a></li>
<li class="chapter" data-level="8.12" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#number-of-chromosomes-in-various-organisms"><i class="fa fa-check"></i><b>8.12</b> Number of chromosomes in various organisms</a><ul>
<li class="chapter" data-level="8.12.1" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#in-prokaryotes"><i class="fa fa-check"></i><b>8.12.1</b> In prokaryotes</a></li>
</ul></li>
<li class="chapter" data-level="8.13" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#mitosis"><i class="fa fa-check"></i><b>8.13</b> Mitosis</a><ul>
<li class="chapter" data-level="8.13.1" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#interphase-1"><i class="fa fa-check"></i><b>8.13.1</b> Interphase</a></li>
<li class="chapter" data-level="8.13.2" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#preprophase-plant-cells"><i class="fa fa-check"></i><b>8.13.2</b> Preprophase (plant cells)</a></li>
<li class="chapter" data-level="8.13.3" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#prophase"><i class="fa fa-check"></i><b>8.13.3</b> Prophase</a></li>
<li class="chapter" data-level="8.13.4" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#prometaphase"><i class="fa fa-check"></i><b>8.13.4</b> Prometaphase</a></li>
<li class="chapter" data-level="8.13.5" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#metaphase"><i class="fa fa-check"></i><b>8.13.5</b> Metaphase</a></li>
<li class="chapter" data-level="8.13.6" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#anaphase"><i class="fa fa-check"></i><b>8.13.6</b> Anaphase</a></li>
<li class="chapter" data-level="8.13.7" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#telophase"><i class="fa fa-check"></i><b>8.13.7</b> Telophase</a></li>
<li class="chapter" data-level="8.13.8" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#cytokinesis"><i class="fa fa-check"></i><b>8.13.8</b> Cytokinesis</a></li>
<li class="chapter" data-level="8.13.9" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#cytokinesis-phase-separation-of-all-cell-components"><i class="fa fa-check"></i><b>8.13.9</b> Cytokinesis phase (separation of all cell components)</a></li>
</ul></li>
<li class="chapter" data-level="8.14" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#regulation-of-eukaryotic-cell-cycle"><i class="fa fa-check"></i><b>8.14</b> Regulation of eukaryotic cell cycle</a><ul>
<li class="chapter" data-level="8.14.1" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#role-of-cyclins-and-cdks"><i class="fa fa-check"></i><b>8.14.1</b> Role of cyclins and CDKs</a></li>
<li class="chapter" data-level="8.14.2" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#cell-cycle-checkpoints"><i class="fa fa-check"></i><b>8.14.2</b> Cell Cycle Checkpoints</a></li>
<li class="chapter" data-level="8.14.3" data-path="the-cell-cycle-and-cell-division.html"><a href="the-cell-cycle-and-cell-division.html#role-in-tumor-formation"><i class="fa fa-check"></i><b>8.14.3</b> Role in tumor formation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="9" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html"><i class="fa fa-check"></i><b>9</b> Sexual Reproductin And Meiosis</a><ul>
<li class="chapter" data-level="9.1" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#reproduction"><i class="fa fa-check"></i><b>9.1</b> Reproduction</a><ul>
<li class="chapter" data-level="9.1.1" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#sexual-reproduction"><i class="fa fa-check"></i><b>9.1.1</b> Sexual reproduction</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#meiosis"><i class="fa fa-check"></i><b>9.2</b> Meiosis</a></li>
<li class="chapter" data-level="9.3" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#meiosis-i"><i class="fa fa-check"></i><b>9.3</b> Meiosis I</a></li>
<li class="chapter" data-level="9.4" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#prophase-i"><i class="fa fa-check"></i><b>9.4</b> Prophase I</a></li>
<li class="chapter" data-level="9.5" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#metaphase-i"><i class="fa fa-check"></i><b>9.5</b> Metaphase I</a></li>
<li class="chapter" data-level="9.6" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#anaphase-i"><i class="fa fa-check"></i><b>9.6</b> Anaphase I</a></li>
<li class="chapter" data-level="9.7" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#telophase-i"><i class="fa fa-check"></i><b>9.7</b> Telophase I</a></li>
<li class="chapter" data-level="9.8" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#meiosis-ii"><i class="fa fa-check"></i><b>9.8</b> Meiosis II</a></li>
<li class="chapter" data-level="9.9" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#nondisjunction"><i class="fa fa-check"></i><b>9.9</b> Nondisjunction</a></li>
<li class="chapter" data-level="9.10" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#chromosomal-disorders"><i class="fa fa-check"></i><b>9.10</b> Chromosomal Disorders</a></li>
<li class="chapter" data-level="9.11" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#aberrations"><i class="fa fa-check"></i><b>9.11</b> Aberrations</a></li>
<li class="chapter" data-level="9.12" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#numerical-abnormalities"><i class="fa fa-check"></i><b>9.12</b> Numerical abnormalities</a></li>
<li class="chapter" data-level="9.13" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#mechanisms"><i class="fa fa-check"></i><b>9.13</b> Mechanisms</a></li>
<li class="chapter" data-level="9.14" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#diagnosis"><i class="fa fa-check"></i><b>9.14</b> Diagnosis</a></li>
<li class="chapter" data-level="9.15" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#structural-abnormalities"><i class="fa fa-check"></i><b>9.15</b> Structural abnormalities</a></li>
<li class="chapter" data-level="9.16" data-path="sexual-reproductin-and-meiosis.html"><a href="sexual-reproductin-and-meiosis.html#acquired-chromosomal-abnormalities"><i class="fa fa-check"></i><b>9.16</b> Acquired chromosomal abnormalities</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html"><i class="fa fa-check"></i><b>10</b> Mendelian Inheritance</a><ul>
<li class="chapter" data-level="10.1" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendels-genetic-discoveries"><i class="fa fa-check"></i><b>10.1</b> Mendel’s genetic discoveries</a></li>
<li class="chapter" data-level="10.2" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-dominance-and-uniformity"><i class="fa fa-check"></i><b>10.2</b> Law of Dominance and Uniformity</a></li>
<li class="chapter" data-level="10.3" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-segregation-of-genes"><i class="fa fa-check"></i><b>10.3</b> Law of Segregation of genes</a></li>
<li class="chapter" data-level="10.4" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#law-of-independent-assortment"><i class="fa fa-check"></i><b>10.4</b> Law of Independent Assortment</a></li>
<li class="chapter" data-level="10.5" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-trait"><i class="fa fa-check"></i><b>10.5</b> Mendelian trait</a></li>
<li class="chapter" data-level="10.6" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#zygosity"><i class="fa fa-check"></i><b>10.6</b> Zygosity</a></li>
<li class="chapter" data-level="10.7" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#monohybrid-cross"><i class="fa fa-check"></i><b>10.7</b> Monohybrid cross</a></li>
<li class="chapter" data-level="10.8" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#punnett-square"><i class="fa fa-check"></i><b>10.8</b> Punnett square</a></li>
<li class="chapter" data-level="10.9" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#dihybrid-cross"><i class="fa fa-check"></i><b>10.9</b> Dihybrid cross</a></li>
<li class="chapter" data-level="10.10" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#forked-line-method"><i class="fa fa-check"></i><b>10.10</b> Forked-line method</a></li>
<li class="chapter" data-level="10.11" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#mendelian-traits-in-humans"><i class="fa fa-check"></i><b>10.11</b> Mendelian traits in humans</a></li>
<li class="chapter" data-level="10.12" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#omim---online-mendelian-inheritance-in-man"><i class="fa fa-check"></i><b>10.12</b> OMIM - Online Mendelian Inheritance in Man</a></li>
<li class="chapter" data-level="10.13" data-path="mendelian-inheritance.html"><a href="mendelian-inheritance.html#non-mendelian-inheritance"><i class="fa fa-check"></i><b>10.13</b> Non-Mendelian inheritance</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html"><i class="fa fa-check"></i><b>11</b> Molecular Biology Of The Gene</a><ul>
<li class="chapter" data-level="11.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna"><i class="fa fa-check"></i><b>11.1</b> DNA</a><ul>
<li class="chapter" data-level="11.1.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#structure-and-composition-of-dna"><i class="fa fa-check"></i><b>11.1.1</b> Structure and composition of DNA</a></li>
<li class="chapter" data-level="11.1.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#base-modifications-and-dna-packaging"><i class="fa fa-check"></i><b>11.1.2</b> Base modifications and DNA packaging</a></li>
<li class="chapter" data-level="11.1.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-damage"><i class="fa fa-check"></i><b>11.1.3</b> DNA damage</a></li>
<li class="chapter" data-level="11.1.4" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#biological-functions-of-dna"><i class="fa fa-check"></i><b>11.1.4</b> Biological functions of DNA</a></li>
<li class="chapter" data-level="11.1.5" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-replication"><i class="fa fa-check"></i><b>11.1.5</b> DNA replication</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#eukaryotic-dna-replication"><i class="fa fa-check"></i><b>11.2</b> Eukaryotic DNA replication</a><ul>
<li class="chapter" data-level="11.2.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#initiation"><i class="fa fa-check"></i><b>11.2.1</b> Initiation</a></li>
<li class="chapter" data-level="11.2.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#pre-replication-complex"><i class="fa fa-check"></i><b>11.2.2</b> Pre-replication complex</a></li>
<li class="chapter" data-level="11.2.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#preinitiation-complex"><i class="fa fa-check"></i><b>11.2.3</b> Preinitiation complex</a></li>
<li class="chapter" data-level="11.2.4" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#elongation"><i class="fa fa-check"></i><b>11.2.4</b> Elongation</a></li>
<li class="chapter" data-level="11.2.5" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#replication-fork"><i class="fa fa-check"></i><b>11.2.5</b> Replication fork</a></li>
<li class="chapter" data-level="11.2.6" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#leading-strand"><i class="fa fa-check"></i><b>11.2.6</b> Leading strand</a></li>
<li class="chapter" data-level="11.2.7" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#lagging-strand"><i class="fa fa-check"></i><b>11.2.7</b> Lagging strand</a></li>
</ul></li>
<li class="chapter" data-level="11.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-replication-proteins"><i class="fa fa-check"></i><b>11.3</b> DNA replication proteins</a><ul>
<li class="chapter" data-level="11.3.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#termination"><i class="fa fa-check"></i><b>11.3.1</b> Termination</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#regulation-of-replication"><i class="fa fa-check"></i><b>11.4</b> Regulation of replication</a><ul>
<li class="chapter" data-level="11.4.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-modifying-enzymes"><i class="fa fa-check"></i><b>11.4.1</b> DNA-modifying enzymes</a></li>
<li class="chapter" data-level="11.4.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-recombination"><i class="fa fa-check"></i><b>11.4.2</b> DNA recombination</a></li>
</ul></li>
<li class="chapter" data-level="11.5" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#evolution-1"><i class="fa fa-check"></i><b>11.5</b> Evolution</a></li>
<li class="chapter" data-level="11.6" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#genetic-engineering"><i class="fa fa-check"></i><b>11.6</b> Genetic engineering</a></li>
<li class="chapter" data-level="11.7" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#dna-profiling"><i class="fa fa-check"></i><b>11.7</b> DNA profiling</a></li>
<li class="chapter" data-level="11.8" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#rna"><i class="fa fa-check"></i><b>11.8</b> RNA</a><ul>
<li class="chapter" data-level="11.8.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#comparison-with-dna"><i class="fa fa-check"></i><b>11.8.1</b> Comparison with DNA</a></li>
<li class="chapter" data-level="11.8.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#structure-of-rna"><i class="fa fa-check"></i><b>11.8.2</b> Structure of RNA</a></li>
<li class="chapter" data-level="11.8.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#synthesis-of-rna"><i class="fa fa-check"></i><b>11.8.3</b> Synthesis of RNA</a></li>
<li class="chapter" data-level="11.8.4" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#coding-and-non-coding-rna"><i class="fa fa-check"></i><b>11.8.4</b> Coding and non-coding RNA</a></li>
<li class="chapter" data-level="11.8.5" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#regulatory-rna"><i class="fa fa-check"></i><b>11.8.5</b> Regulatory RNA</a></li>
<li class="chapter" data-level="11.8.6" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#rna-interference-by-mirnas"><i class="fa fa-check"></i><b>11.8.6</b> RNA interference by miRNAs</a></li>
</ul></li>
<li class="chapter" data-level="11.9" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#gene-expression"><i class="fa fa-check"></i><b>11.9</b> Gene Expression</a></li>
<li class="chapter" data-level="11.10" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#transcription-and-rna-processing"><i class="fa fa-check"></i><b>11.10</b> Transcription And RNA Processing</a></li>
<li class="chapter" data-level="11.11" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#initiation-1"><i class="fa fa-check"></i><b>11.11</b> Initiation</a></li>
<li class="chapter" data-level="11.12" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#elongation-1"><i class="fa fa-check"></i><b>11.12</b> Elongation</a></li>
<li class="chapter" data-level="11.13" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#termination-1"><i class="fa fa-check"></i><b>11.13</b> Termination</a></li>
<li class="chapter" data-level="11.14" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#inhibitors-of-transcription"><i class="fa fa-check"></i><b>11.14</b> Inhibitors of transcription</a></li>
<li class="chapter" data-level="11.15" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#transcription-factors"><i class="fa fa-check"></i><b>11.15</b> Transcription factors</a></li>
<li class="chapter" data-level="11.16" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#rna-processing"><i class="fa fa-check"></i><b>11.16</b> RNA Processing</a></li>
<li class="chapter" data-level="11.17" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#mrna-processing"><i class="fa fa-check"></i><b>11.17</b> mRNA processing</a><ul>
<li class="chapter" data-level="11.17.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#processing"><i class="fa fa-check"></i><b>11.17.1</b> 5’ processing</a></li>
<li class="chapter" data-level="11.17.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#processing-1"><i class="fa fa-check"></i><b>11.17.2</b> 3’ processing</a></li>
<li class="chapter" data-level="11.17.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#intron-splicing"><i class="fa fa-check"></i><b>11.17.3</b> Intron splicing</a></li>
</ul></li>
<li class="chapter" data-level="11.18" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#reverse-transcription"><i class="fa fa-check"></i><b>11.18</b> Reverse transcription</a></li>
<li class="chapter" data-level="11.19" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#rna-export"><i class="fa fa-check"></i><b>11.19</b> RNA export</a></li>
<li class="chapter" data-level="11.20" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#translation-and-the-genetic-code"><i class="fa fa-check"></i><b>11.20</b> Translation And The Genetic Code</a></li>
<li class="chapter" data-level="11.21" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#ribosomes"><i class="fa fa-check"></i><b>11.21</b> Ribosomes</a></li>
<li class="chapter" data-level="11.22" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#bacterial-ribosomes"><i class="fa fa-check"></i><b>11.22</b> Bacterial ribosomes</a></li>
<li class="chapter" data-level="11.23" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#eukaryotic-ribosomes"><i class="fa fa-check"></i><b>11.23</b> Eukaryotic ribosomes</a></li>
<li class="chapter" data-level="11.24" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#translation"><i class="fa fa-check"></i><b>11.24</b> Translation</a></li>
<li class="chapter" data-level="11.25" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#eukaryotic-translation"><i class="fa fa-check"></i><b>11.25</b> Eukaryotic translation</a><ul>
<li class="chapter" data-level="11.25.1" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#initiation-2"><i class="fa fa-check"></i><b>11.25.1</b> Initiation</a></li>
<li class="chapter" data-level="11.25.2" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#elongation-2"><i class="fa fa-check"></i><b>11.25.2</b> Elongation</a></li>
<li class="chapter" data-level="11.25.3" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#termination-2"><i class="fa fa-check"></i><b>11.25.3</b> Termination</a></li>
</ul></li>
<li class="chapter" data-level="11.26" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#genetic-code"><i class="fa fa-check"></i><b>11.26</b> Genetic code</a></li>
<li class="chapter" data-level="11.27" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#protein-folding"><i class="fa fa-check"></i><b>11.27</b> Protein Folding</a></li>
<li class="chapter" data-level="11.28" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#protein-translocation"><i class="fa fa-check"></i><b>11.28</b> Protein Translocation</a></li>
<li class="chapter" data-level="11.29" data-path="molecular-biology-of-the-gene.html"><a href="molecular-biology-of-the-gene.html#protein-transport"><i class="fa fa-check"></i><b>11.29</b> Protein Transport</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html"><i class="fa fa-check"></i><b>12</b> Regulation of Gene Expression</a><ul>
<li class="chapter" data-level="12.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-lac-operon"><i class="fa fa-check"></i><b>12.1</b> The <em>lac</em> operon</a></li>
<li class="chapter" data-level="12.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#the-trp-operon"><i class="fa fa-check"></i><b>12.2</b> The <em>trp</em> operon</a></li>
<li class="chapter" data-level="12.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulated-stages-of-gene-expression"><i class="fa fa-check"></i><b>12.3</b> Regulated stages of gene expression</a><ul>
<li class="chapter" data-level="12.3.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-packing"><i class="fa fa-check"></i><b>12.3.1</b> Regulation of transciption by DNA packing</a></li>
<li class="chapter" data-level="12.3.2" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-transciption-by-dna-modification"><i class="fa fa-check"></i><b>12.3.2</b> Regulation of transciption by DNA modification</a></li>
<li class="chapter" data-level="12.3.3" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-rna-polymerase"><i class="fa fa-check"></i><b>12.3.3</b> Regulation of RNA polymerase</a></li>
</ul></li>
<li class="chapter" data-level="12.4" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#post-transcriptional-regulation"><i class="fa fa-check"></i><b>12.4</b> Post-transcriptional regulation</a><ul>
<li class="chapter" data-level="12.4.1" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#three-prime-untranslated-regions-and-micrornas"><i class="fa fa-check"></i><b>12.4.1</b> Three prime untranslated regions and microRNAs</a></li>
</ul></li>
<li class="chapter" data-level="12.5" data-path="regulation-of-gene-expression.html"><a href="regulation-of-gene-expression.html#regulation-of-translation"><i class="fa fa-check"></i><b>12.5</b> Regulation of translation</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html"><i class="fa fa-check"></i><b>13</b> Bacteria, Archaea And Viruses</a><ul>
<li class="chapter" data-level="13.1" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#bacteria"><i class="fa fa-check"></i><b>13.1</b> Bacteria</a></li>
<li class="chapter" data-level="13.2" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#cellular-structure"><i class="fa fa-check"></i><b>13.2</b> Cellular structure</a><ul>
<li class="chapter" data-level="13.2.1" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#intracellular-structures"><i class="fa fa-check"></i><b>13.2.1</b> Intracellular structures</a></li>
<li class="chapter" data-level="13.2.2" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#extracellular-structures"><i class="fa fa-check"></i><b>13.2.2</b> Extracellular structures</a></li>
<li class="chapter" data-level="13.2.3" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#endospores"><i class="fa fa-check"></i><b>13.2.3</b> Endospores</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#metabolism-1"><i class="fa fa-check"></i><b>13.3</b> Metabolism</a></li>
<li class="chapter" data-level="13.4" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#growth-and-reproduction"><i class="fa fa-check"></i><b>13.4</b> Growth and reproduction</a></li>
<li class="chapter" data-level="13.5" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#genetics-1"><i class="fa fa-check"></i><b>13.5</b> Genetics</a></li>
<li class="chapter" data-level="13.6" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#movement"><i class="fa fa-check"></i><b>13.6</b> Movement</a></li>
<li class="chapter" data-level="13.7" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#communication"><i class="fa fa-check"></i><b>13.7</b> Communication</a></li>
<li class="chapter" data-level="13.8" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#classification-and-identification"><i class="fa fa-check"></i><b>13.8</b> Classification and identification</a></li>
<li class="chapter" data-level="13.9" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#interactions-with-other-organisms"><i class="fa fa-check"></i><b>13.9</b> Interactions with other organisms</a></li>
<li class="chapter" data-level="13.10" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#pathogens"><i class="fa fa-check"></i><b>13.10</b> Pathogens</a></li>
<li class="chapter" data-level="13.11" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#significance-in-technology-and-industry"><i class="fa fa-check"></i><b>13.11</b> Significance in technology and industry</a></li>
<li class="chapter" data-level="13.12" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#archaea"><i class="fa fa-check"></i><b>13.12</b> Archaea</a></li>
<li class="chapter" data-level="13.13" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#origin-and-evolution"><i class="fa fa-check"></i><b>13.13</b> Origin and evolution</a><ul>
<li class="chapter" data-level="13.13.1" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#relationship-to-bacteria"><i class="fa fa-check"></i><b>13.13.1</b> Relationship to bacteria</a></li>
<li class="chapter" data-level="13.13.2" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#relation-to-eukaryotes"><i class="fa fa-check"></i><b>13.13.2</b> Relation to eukaryotes</a></li>
</ul></li>
<li class="chapter" data-level="13.14" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#morphology"><i class="fa fa-check"></i><b>13.14</b> Morphology</a><ul>
<li class="chapter" data-level="13.14.1" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#structure-composition-development-and-operation"><i class="fa fa-check"></i><b>13.14.1</b> Structure, composition development, and operation</a></li>
<li class="chapter" data-level="13.14.2" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#cell-wall-and-flagella"><i class="fa fa-check"></i><b>13.14.2</b> Cell wall and flagella</a></li>
<li class="chapter" data-level="13.14.3" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#membranes"><i class="fa fa-check"></i><b>13.14.3</b> Membranes</a></li>
<li class="chapter" data-level="13.14.4" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#metabolism-2"><i class="fa fa-check"></i><b>13.14.4</b> Metabolism</a></li>
</ul></li>
<li class="chapter" data-level="13.15" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#viruses"><i class="fa fa-check"></i><b>13.15</b> Viruses</a><ul>
<li class="chapter" data-level="13.15.1" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#role-in-human-disease"><i class="fa fa-check"></i><b>13.15.1</b> Role in human disease</a></li>
<li class="chapter" data-level="13.15.2" data-path="bacteria-archaea-and-viruses.html"><a href="bacteria-archaea-and-viruses.html#bacterial-viruses"><i class="fa fa-check"></i><b>13.15.2</b> Bacterial viruses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14" data-path="protists.html"><a href="protists.html"><i class="fa fa-check"></i><b>14</b> Protists</a><ul>
<li class="chapter" data-level="14.1" data-path="protists.html"><a href="protists.html#reproduction-1"><i class="fa fa-check"></i><b>14.1</b> Reproduction</a></li>
<li class="chapter" data-level="14.2" data-path="protists.html"><a href="protists.html#ecology"><i class="fa fa-check"></i><b>14.2</b> Ecology</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="evolution-2.html"><a href="evolution-2.html"><i class="fa fa-check"></i><b>15</b> Evolution</a><ul>
<li class="chapter" data-level="15.1" data-path="evolution-2.html"><a href="evolution-2.html#history-of-evolutionary-thought"><i class="fa fa-check"></i><b>15.1</b> History of evolutionary thought</a></li>
<li class="chapter" data-level="15.2" data-path="evolution-2.html"><a href="evolution-2.html#pangenesis-and-heredity"><i class="fa fa-check"></i><b>15.2</b> Pangenesis And Heredity</a></li>
<li class="chapter" data-level="15.3" data-path="evolution-2.html"><a href="evolution-2.html#the-modern-synthesis"><i class="fa fa-check"></i><b>15.3</b> The Modern Synthesis</a></li>
<li class="chapter" data-level="15.4" data-path="evolution-2.html"><a href="evolution-2.html#heredity"><i class="fa fa-check"></i><b>15.4</b> Heredity</a></li>
<li class="chapter" data-level="15.5" data-path="evolution-2.html"><a href="evolution-2.html#variation"><i class="fa fa-check"></i><b>15.5</b> Variation</a></li>
<li class="chapter" data-level="15.6" data-path="evolution-2.html"><a href="evolution-2.html#mutation"><i class="fa fa-check"></i><b>15.6</b> Mutation</a></li>
<li class="chapter" data-level="15.7" data-path="evolution-2.html"><a href="evolution-2.html#sex-and-recombination"><i class="fa fa-check"></i><b>15.7</b> Sex And Recombination</a></li>
<li class="chapter" data-level="15.8" data-path="evolution-2.html"><a href="evolution-2.html#gene-flow"><i class="fa fa-check"></i><b>15.8</b> Gene Flow</a></li>
<li class="chapter" data-level="15.9" data-path="evolution-2.html"><a href="evolution-2.html#mechanisms-1"><i class="fa fa-check"></i><b>15.9</b> Mechanisms</a></li>
<li class="chapter" data-level="15.10" data-path="evolution-2.html"><a href="evolution-2.html#natural-selection"><i class="fa fa-check"></i><b>15.10</b> Natural selection</a></li>
<li class="chapter" data-level="15.11" data-path="evolution-2.html"><a href="evolution-2.html#genetic-hitchhiking"><i class="fa fa-check"></i><b>15.11</b> Genetic Hitchhiking</a></li>
<li class="chapter" data-level="15.12" data-path="evolution-2.html"><a href="evolution-2.html#genetic-drift"><i class="fa fa-check"></i><b>15.12</b> Genetic Drift</a></li>
<li class="chapter" data-level="15.13" data-path="evolution-2.html"><a href="evolution-2.html#gene-flow-1"><i class="fa fa-check"></i><b>15.13</b> Gene Flow</a></li>
<li class="chapter" data-level="15.14" data-path="evolution-2.html"><a href="evolution-2.html#mutation-bias"><i class="fa fa-check"></i><b>15.14</b> Mutation Bias</a></li>
<li class="chapter" data-level="15.15" data-path="evolution-2.html"><a href="evolution-2.html#outcomes"><i class="fa fa-check"></i><b>15.15</b> Outcomes</a></li>
<li class="chapter" data-level="15.16" data-path="evolution-2.html"><a href="evolution-2.html#adaptation"><i class="fa fa-check"></i><b>15.16</b> Adaptation</a></li>
<li class="chapter" data-level="15.17" data-path="evolution-2.html"><a href="evolution-2.html#coevolution"><i class="fa fa-check"></i><b>15.17</b> Coevolution</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="biological-taxonomy-and-systematics.html"><a href="biological-taxonomy-and-systematics.html"><i class="fa fa-check"></i><b>16</b> Biological Taxonomy And Systematics</a><ul>
<li class="chapter" data-level="16.1" data-path="biological-taxonomy-and-systematics.html"><a href="biological-taxonomy-and-systematics.html#cladistics"><i class="fa fa-check"></i><b>16.1</b> Cladistics</a></li>
</ul></li>
<li class="chapter" data-level="17" data-path="fungi.html"><a href="fungi.html"><i class="fa fa-check"></i><b>17</b> Fungi</a><ul>
<li class="chapter" data-level="17.1" data-path="fungi.html"><a href="fungi.html#mycology"><i class="fa fa-check"></i><b>17.1</b> Mycology</a></li>
<li class="chapter" data-level="17.2" data-path="fungi.html"><a href="fungi.html#morphology-1"><i class="fa fa-check"></i><b>17.2</b> Morphology</a></li>
<li class="chapter" data-level="17.3" data-path="fungi.html"><a href="fungi.html#growth-and-physiology"><i class="fa fa-check"></i><b>17.3</b> Growth and physiology</a></li>
<li class="chapter" data-level="17.4" data-path="fungi.html"><a href="fungi.html#reproduction-2"><i class="fa fa-check"></i><b>17.4</b> Reproduction</a><ul>
<li class="chapter" data-level="17.4.1" data-path="fungi.html"><a href="fungi.html#asexual-reproduction"><i class="fa fa-check"></i><b>17.4.1</b> Asexual reproduction</a></li>
<li class="chapter" data-level="17.4.2" data-path="fungi.html"><a href="fungi.html#sexual-reproduction-1"><i class="fa fa-check"></i><b>17.4.2</b> Sexual reproduction</a></li>
<li class="chapter" data-level="17.4.3" data-path="fungi.html"><a href="fungi.html#spore-dispersal"><i class="fa fa-check"></i><b>17.4.3</b> Spore dispersal</a></li>
<li class="chapter" data-level="17.4.4" data-path="fungi.html"><a href="fungi.html#homothallism"><i class="fa fa-check"></i><b>17.4.4</b> Homothallism</a></li>
<li class="chapter" data-level="17.4.5" data-path="fungi.html"><a href="fungi.html#other-sexual-processes"><i class="fa fa-check"></i><b>17.4.5</b> Other sexual processes</a></li>
</ul></li>
<li class="chapter" data-level="17.5" data-path="fungi.html"><a href="fungi.html#evolution-3"><i class="fa fa-check"></i><b>17.5</b> Evolution</a></li>
<li class="chapter" data-level="17.6" data-path="fungi.html"><a href="fungi.html#fungus-like-organisms"><i class="fa fa-check"></i><b>17.6</b> Fungus-like organisms</a></li>
<li class="chapter" data-level="17.7" data-path="fungi.html"><a href="fungi.html#ecology-1"><i class="fa fa-check"></i><b>17.7</b> Ecology</a><ul>
<li class="chapter" data-level="17.7.1" data-path="fungi.html"><a href="fungi.html#symbiosis"><i class="fa fa-check"></i><b>17.7.1</b> Symbiosis</a></li>
</ul></li>
<li class="chapter" data-level="17.8" data-path="fungi.html"><a href="fungi.html#fungi-as-pathogens-and-parasites"><i class="fa fa-check"></i><b>17.8</b> Fungi As pathogens and parasites</a></li>
<li class="chapter" data-level="17.9" data-path="fungi.html"><a href="fungi.html#mycotoxins"><i class="fa fa-check"></i><b>17.9</b> Mycotoxins</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="plants.html"><a href="plants.html"><i class="fa fa-check"></i><b>18</b> Plants</a><ul>
<li class="chapter" data-level="18.1" data-path="plants.html"><a href="plants.html#plant-tissues"><i class="fa fa-check"></i><b>18.1</b> Plant Tissues</a><ul>
<li class="chapter" data-level="18.1.1" data-path="plants.html"><a href="plants.html#meristematic-tissues"><i class="fa fa-check"></i><b>18.1.1</b> Meristematic tissues</a></li>
<li class="chapter" data-level="18.1.2" data-path="plants.html"><a href="plants.html#permanent-tissues"><i class="fa fa-check"></i><b>18.1.2</b> Permanent tissues</a></li>
<li class="chapter" data-level="18.1.3" data-path="plants.html"><a href="plants.html#parenchyma"><i class="fa fa-check"></i><b>18.1.3</b> Parenchyma</a></li>
<li class="chapter" data-level="18.1.4" data-path="plants.html"><a href="plants.html#collenchyma"><i class="fa fa-check"></i><b>18.1.4</b> Collenchyma</a></li>
<li class="chapter" data-level="18.1.5" data-path="plants.html"><a href="plants.html#sclerenchyma"><i class="fa fa-check"></i><b>18.1.5</b> Sclerenchyma</a></li>
<li class="chapter" data-level="18.1.6" data-path="plants.html"><a href="plants.html#epidermis"><i class="fa fa-check"></i><b>18.1.6</b> Epidermis</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="plants.html"><a href="plants.html#complex-permanent-tissue"><i class="fa fa-check"></i><b>18.2</b> Complex permanent tissue</a></li>
<li class="chapter" data-level="18.3" data-path="plants.html"><a href="plants.html#phloem"><i class="fa fa-check"></i><b>18.3</b> Phloem</a></li>
<li class="chapter" data-level="18.4" data-path="plants.html"><a href="plants.html#embryophytes"><i class="fa fa-check"></i><b>18.4</b> Embryophytes</a></li>
<li class="chapter" data-level="18.5" data-path="plants.html"><a href="plants.html#non-vascular-plants"><i class="fa fa-check"></i><b>18.5</b> Non-vascular Plants</a><ul>
<li class="chapter" data-level="18.5.1" data-path="plants.html"><a href="plants.html#life-cycle"><i class="fa fa-check"></i><b>18.5.1</b> Life cycle</a></li>
</ul></li>
<li class="chapter" data-level="18.6" data-path="plants.html"><a href="plants.html#vascular-plants"><i class="fa fa-check"></i><b>18.6</b> Vascular Plants</a></li>
<li class="chapter" data-level="18.7" data-path="plants.html"><a href="plants.html#ferns"><i class="fa fa-check"></i><b>18.7</b> Ferns</a><ul>
<li class="chapter" data-level="18.7.1" data-path="plants.html"><a href="plants.html#life-cycle-1"><i class="fa fa-check"></i><b>18.7.1</b> Life cycle</a></li>
</ul></li>
<li class="chapter" data-level="18.8" data-path="plants.html"><a href="plants.html#seed-plants-spermatophytes"><i class="fa fa-check"></i><b>18.8</b> Seed Plants (Spermatophytes)</a></li>
<li class="chapter" data-level="18.9" data-path="plants.html"><a href="plants.html#gymnosperms"><i class="fa fa-check"></i><b>18.9</b> Gymnosperms</a></li>
<li class="chapter" data-level="18.10" data-path="plants.html"><a href="plants.html#angiosperms"><i class="fa fa-check"></i><b>18.10</b> Angiosperms</a><ul>
<li class="chapter" data-level="18.10.1" data-path="plants.html"><a href="plants.html#fertilization-and-embryogenesis"><i class="fa fa-check"></i><b>18.10.1</b> Fertilization and embryogenesis</a></li>
</ul></li>
<li class="chapter" data-level="18.11" data-path="plants.html"><a href="plants.html#plant-anatomy"><i class="fa fa-check"></i><b>18.11</b> Plant anatomy</a></li>
<li class="chapter" data-level="18.12" data-path="plants.html"><a href="plants.html#biochemistry-of-plants"><i class="fa fa-check"></i><b>18.12</b> Biochemistry of plants</a><ul>
<li class="chapter" data-level="18.12.1" data-path="plants.html"><a href="plants.html#pigments"><i class="fa fa-check"></i><b>18.12.1</b> Pigments</a></li>
<li class="chapter" data-level="18.12.2" data-path="plants.html"><a href="plants.html#plant-hormones"><i class="fa fa-check"></i><b>18.12.2</b> Plant Hormones</a></li>
<li class="chapter" data-level="18.12.3" data-path="plants.html"><a href="plants.html#photomorphogenesis"><i class="fa fa-check"></i><b>18.12.3</b> Photomorphogenesis</a></li>
<li class="chapter" data-level="18.12.4" data-path="plants.html"><a href="plants.html#photoperiodism"><i class="fa fa-check"></i><b>18.12.4</b> Photoperiodism</a></li>
</ul></li>
<li class="chapter" data-level="18.13" data-path="plants.html"><a href="plants.html#plant-secondary-metabolism"><i class="fa fa-check"></i><b>18.13</b> Plant secondary metabolism</a></li>
<li class="chapter" data-level="18.14" data-path="plants.html"><a href="plants.html#tropisms-and-nastic-movements"><i class="fa fa-check"></i><b>18.14</b> Tropisms and nastic movements</a><ul>
<li class="chapter" data-level="18.14.1" data-path="plants.html"><a href="plants.html#asexual-reproduction-1"><i class="fa fa-check"></i><b>18.14.1</b> Asexual reproduction</a></li>
<li class="chapter" data-level="18.14.2" data-path="plants.html"><a href="plants.html#sexual-reproduction-2"><i class="fa fa-check"></i><b>18.14.2</b> Sexual reproduction</a></li>
<li class="chapter" data-level="18.14.3" data-path="plants.html"><a href="plants.html#flowering-plants"><i class="fa fa-check"></i><b>18.14.3</b> Flowering plants</a></li>
<li class="chapter" data-level="18.14.4" data-path="plants.html"><a href="plants.html#pollination"><i class="fa fa-check"></i><b>18.14.4</b> Pollination</a></li>
<li class="chapter" data-level="18.14.5" data-path="plants.html"><a href="plants.html#ferns-1"><i class="fa fa-check"></i><b>18.14.5</b> Ferns</a></li>
<li class="chapter" data-level="18.14.6" data-path="plants.html"><a href="plants.html#bryophytes"><i class="fa fa-check"></i><b>18.14.6</b> Bryophytes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="19" data-path="animals.html"><a href="animals.html"><i class="fa fa-check"></i><b>19</b> Animals</a><ul>
<li class="chapter" data-level="19.1" data-path="animals.html"><a href="animals.html#non-bilaterian-animals"><i class="fa fa-check"></i><b>19.1</b> Non-bilaterian animals</a></li>
<li class="chapter" data-level="19.2" data-path="animals.html"><a href="animals.html#bilaterian-animals"><i class="fa fa-check"></i><b>19.2</b> Bilaterian animals</a></li>
<li class="chapter" data-level="19.3" data-path="animals.html"><a href="animals.html#protostomes-and-deuterostomes"><i class="fa fa-check"></i><b>19.3</b> Protostomes and deuterostomes</a></li>
<li class="chapter" data-level="19.4" data-path="animals.html"><a href="animals.html#ecdysozoa"><i class="fa fa-check"></i><b>19.4</b> Ecdysozoa</a></li>
<li class="chapter" data-level="19.5" data-path="animals.html"><a href="animals.html#spiralia"><i class="fa fa-check"></i><b>19.5</b> Spiralia</a></li>
<li class="chapter" data-level="19.6" data-path="animals.html"><a href="animals.html#tissues"><i class="fa fa-check"></i><b>19.6</b> Tissues</a></li>
<li class="chapter" data-level="19.7" data-path="animals.html"><a href="animals.html#epithelial-tissue"><i class="fa fa-check"></i><b>19.7</b> Epithelial tissue</a></li>
<li class="chapter" data-level="19.8" data-path="animals.html"><a href="animals.html#connective-tissue"><i class="fa fa-check"></i><b>19.8</b> Connective tissue</a></li>
<li class="chapter" data-level="19.9" data-path="animals.html"><a href="animals.html#muscular-tissue"><i class="fa fa-check"></i><b>19.9</b> Muscular tissue</a></li>
<li class="chapter" data-level="19.10" data-path="animals.html"><a href="animals.html#nervous-tissue"><i class="fa fa-check"></i><b>19.10</b> Nervous tissue</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="the-circulatory-and-lymphatic-systems.html"><a href="the-circulatory-and-lymphatic-systems.html"><i class="fa fa-check"></i><b>20</b> The Circulatory And Lymphatic Systems</a><ul>
<li class="chapter" data-level="20.1" data-path="the-circulatory-and-lymphatic-systems.html"><a href="the-circulatory-and-lymphatic-systems.html#history"><i class="fa fa-check"></i><b>20.1</b> History</a></li>
<li class="chapter" data-level="20.2" data-path="the-circulatory-and-lymphatic-systems.html"><a href="the-circulatory-and-lymphatic-systems.html#the-lymphatic-system"><i class="fa fa-check"></i><b>20.2</b> The Lymphatic System</a></li>
</ul></li>
<li class="chapter" data-level="21" data-path="the-immune-system.html"><a href="the-immune-system.html"><i class="fa fa-check"></i><b>21</b> The Immune System</a></li>
<li class="chapter" data-level="22" data-path="the-respiratory-system.html"><a href="the-respiratory-system.html"><i class="fa fa-check"></i><b>22</b> The Respiratory System</a></li>
<li class="chapter" data-level="23" data-path="the-digestive-system.html"><a href="the-digestive-system.html"><i class="fa fa-check"></i><b>23</b> The Digestive System</a></li>
<li class="chapter" data-level="24" data-path="the-excretory-system.html"><a href="the-excretory-system.html"><i class="fa fa-check"></i><b>24</b> The Excretory System</a><ul>
<li class="chapter" data-level="24.1" data-path="the-excretory-system.html"><a href="the-excretory-system.html#the-urinary-system"><i class="fa fa-check"></i><b>24.1</b> The Urinary System</a></li>
</ul></li>
<li class="chapter" data-level="25" data-path="the-nervous-system.html"><a href="the-nervous-system.html"><i class="fa fa-check"></i><b>25</b> The Nervous System</a><ul>
<li class="chapter" data-level="25.1" data-path="the-nervous-system.html"><a href="the-nervous-system.html#introduction"><i class="fa fa-check"></i><b>25.1</b> Introduction</a></li>
<li class="chapter" data-level="25.2" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-cells-of-the-nervous-system"><i class="fa fa-check"></i><b>25.2</b> The Cells Of The Nervous System</a></li>
<li class="chapter" data-level="25.3" data-path="the-nervous-system.html"><a href="the-nervous-system.html#comparative-anatomy-and-evolution-of-nervous-systems"><i class="fa fa-check"></i><b>25.3</b> Comparative Anatomy And Evolution Of Nervous Systems</a></li>
<li class="chapter" data-level="25.4" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-human-brain"><i class="fa fa-check"></i><b>25.4</b> The Human Brain</a></li>
<li class="chapter" data-level="25.5" data-path="the-nervous-system.html"><a href="the-nervous-system.html#development-of-the-nervous-system"><i class="fa fa-check"></i><b>25.5</b> Development Of The Nervous System</a></li>
<li class="chapter" data-level="25.6" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-function-of-the-nervous-system"><i class="fa fa-check"></i><b>25.6</b> The Function Of The Nervous System</a></li>
<li class="chapter" data-level="25.7" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-sensory-system"><i class="fa fa-check"></i><b>25.7</b> The Sensory System</a></li>
<li class="chapter" data-level="25.8" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-motor-system"><i class="fa fa-check"></i><b>25.8</b> The Motor System</a></li>
<li class="chapter" data-level="25.9" data-path="the-nervous-system.html"><a href="the-nervous-system.html#neuronal-signalling"><i class="fa fa-check"></i><b>25.9</b> Neuronal Signalling</a></li>
<li class="chapter" data-level="25.10" data-path="the-nervous-system.html"><a href="the-nervous-system.html#neurons-and-glial-cells"><i class="fa fa-check"></i><b>25.10</b> Neurons And Glial Cells</a><ul>
<li class="chapter" data-level="25.10.1" data-path="the-nervous-system.html"><a href="the-nervous-system.html#neurons"><i class="fa fa-check"></i><b>25.10.1</b> Neurons</a></li>
<li class="chapter" data-level="25.10.2" data-path="the-nervous-system.html"><a href="the-nervous-system.html#glia"><i class="fa fa-check"></i><b>25.10.2</b> Glia</a></li>
</ul></li>
<li class="chapter" data-level="25.11" data-path="the-nervous-system.html"><a href="the-nervous-system.html#electrical-basis-of-neuronal-function"><i class="fa fa-check"></i><b>25.11</b> Electrical Basis Of Neuronal Function</a><ul>
<li class="chapter" data-level="25.11.1" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-membrane-potential"><i class="fa fa-check"></i><b>25.11.1</b> The Membrane Potential</a></li>
</ul></li>
<li class="chapter" data-level="25.12" data-path="the-nervous-system.html"><a href="the-nervous-system.html#ions-and-the-forces-driving-their-motion"><i class="fa fa-check"></i><b>25.12</b> Ions And The Forces Driving Their Motion</a></li>
<li class="chapter" data-level="25.13" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-plasma-membrane"><i class="fa fa-check"></i><b>25.13</b> The Plasma Membrane</a></li>
<li class="chapter" data-level="25.14" data-path="the-nervous-system.html"><a href="the-nervous-system.html#ion-pumps"><i class="fa fa-check"></i><b>25.14</b> Ion Pumps</a></li>
<li class="chapter" data-level="25.15" data-path="the-nervous-system.html"><a href="the-nervous-system.html#ion-channels"><i class="fa fa-check"></i><b>25.15</b> Ion Channels</a></li>
<li class="chapter" data-level="25.16" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-reversal-potential"><i class="fa fa-check"></i><b>25.16</b> The Reversal Potential</a></li>
<li class="chapter" data-level="25.17" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-resting-potential"><i class="fa fa-check"></i><b>25.17</b> The Resting Potential</a></li>
<li class="chapter" data-level="25.18" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-action-potential"><i class="fa fa-check"></i><b>25.18</b> The Action Potential</a></li>
<li class="chapter" data-level="25.19" data-path="the-nervous-system.html"><a href="the-nervous-system.html#graded-potentials"><i class="fa fa-check"></i><b>25.19</b> Graded Potentials</a></li>
</ul></li>
<li class="chapter" data-level="26" data-path="animal-locomotion.html"><a href="animal-locomotion.html"><i class="fa fa-check"></i><b>26</b> Animal locomotion</a><ul>
<li class="chapter" data-level="26.1" data-path="animal-locomotion.html"><a href="animal-locomotion.html#the-muscular-system"><i class="fa fa-check"></i><b>26.1</b> The Muscular System</a></li>
<li class="chapter" data-level="26.2" data-path="animal-locomotion.html"><a href="animal-locomotion.html#muscle-contraction"><i class="fa fa-check"></i><b>26.2</b> Muscle Contraction</a></li>
<li class="chapter" data-level="26.3" data-path="animal-locomotion.html"><a href="animal-locomotion.html#smooth-muscle"><i class="fa fa-check"></i><b>26.3</b> Smooth Muscle</a></li>
<li class="chapter" data-level="26.4" data-path="animal-locomotion.html"><a href="animal-locomotion.html#human-musculoskeletal-system"><i class="fa fa-check"></i><b>26.4</b> Human Musculoskeletal System</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="the-endocrine-system.html"><a href="the-endocrine-system.html"><i class="fa fa-check"></i><b>27</b> The Endocrine System</a><ul>
<li class="chapter" data-level="27.1" data-path="the-endocrine-system.html"><a href="the-endocrine-system.html#hypothalamicpituitarythyroid-axis"><i class="fa fa-check"></i><b>27.1</b> Hypothalamic–pituitary–thyroid axis</a></li>
<li class="chapter" data-level="27.2" data-path="the-endocrine-system.html"><a href="the-endocrine-system.html#hypothalamicpituitarygonadal-axis"><i class="fa fa-check"></i><b>27.2</b> Hypothalamic–pituitary–gonadal axis</a></li>
<li class="chapter" data-level="27.3" data-path="the-endocrine-system.html"><a href="the-endocrine-system.html#hypothalamicpituitaryadrenal-axis"><i class="fa fa-check"></i><b>27.3</b> Hypothalamic–pituitary–adrenal axis</a></li>
</ul></li>
<li class="chapter" data-level="28" data-path="the-reproductive-system.html"><a href="the-reproductive-system.html"><i class="fa fa-check"></i><b>28</b> The Reproductive System</a><ul>
<li class="chapter" data-level="28.1" data-path="the-reproductive-system.html"><a href="the-reproductive-system.html#the-human-reproductive-system"><i class="fa fa-check"></i><b>28.1</b> The Human Reproductive System</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For Biology at Roxbury Community College</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="the-nervous-system" class="section level1">
<h1><span class="header-section-number">25</span> The Nervous System</h1>
<div id="introduction" class="section level2">
<h2><span class="header-section-number">25.1</span> Introduction</h2>
<p>The nervous system is a highly complex part of an animal that coordinates its actions and sensory information by transmitting signals to and from different parts of its body. The nervous system detects environmental changes that impact the body, then works in tandem with the endocrine system to respond to such events.</p>

<div class="figure" style="text-align: center"><span id="fig:nervoussystem"></span>
<img src="figures/nervoussystem/NervousSystem.jpg" alt="The human nervous system." width="70%" />
<p class="caption">
Figure 25.1: The human nervous system.
</p>
</div>
<p>The nervous system derives its name from nerves, which are cylindrical bundles of fibers (the axons of neurons), that emanate from the brain and spinal cord, and branch repeatedly to innervate every part of the body. Nerves are large enough to have been recognized by the ancient Egyptians, Greeks, and Romans, but their internal structure was not understood until it became possible to examine them using a microscope.</p>

<div class="figure" style="text-align: center"><span id="fig:nervesection"></span>
<img src="figures/nervoussystem/CajalNerve.jpg" alt="Transverse section of a nerve. a) a single nerve fibre (axon) surrounded by a thick layer of myelin. c) an interstitial cell. Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay." width="70%" />
<p class="caption">
Figure 25.2: Transverse section of a nerve. a) a single nerve fibre (axon) surrounded by a thick layer of myelin. c) an interstitial cell. <a href="https://wellcomelibrary.org/item/b2129592x#?c=0&amp;m=0&amp;s=0&amp;cv=14&amp;z=0%2C-3.48%2C1%2C8.6591">Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier</a> (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:brainbow"></span>
<img src="figures/nervoussystem/nihms37882f2.jpg" alt="A brainbow of mouse neurons." width="70%" />
<p class="caption">
Figure 25.3: <a href="https://commons.wikimedia.org/wiki/File:Brainbow_(Smith_2007).jpg">A brainbow of mouse neurons.</a>
</p>
</div>
<p>Neuroscience (or neurobiology) is a multidisciplinary branch of biology that combines physiology, anatomy, molecular biology, developmental biology, cytology, mathematical modeling, and psychology to understand the fundamental and emergent properties of neurons and neural circuits. The understanding of the biological basis of learning, memory, behavior, perception, and consciousness has been described as the “ultimate challenge” of the biological sciences. The human brain is often referred to as the most complicated structure in the universe. The scope of neuroscience has broadened over time to include different approaches used to study the nervous system at different scales and the techniques used by neuroscientists have expanded enormously, from molecular and cellular studies of individual neurons to imaging of sensory, motor and cognitive tasks in the brain. Malfunction of the nervous system can occur as a result of genetic defects, physical damage due to trauma or toxicity, infection or simply of ageing. The medical specialty of neurology studies disorders of the nervous system and looks for interventions that can prevent or treat them. Although mental illnesses are believed by many to be neurological disorders affecting the central nervous system, traditionally they are classified separately, and treated by psychiatrists.</p>
<p>Nervous systems are found in most multicellular animals, but vary greatly in complexity. The only multicellular animals that have no nervous system at all are sponges, placozoans, and mesozoans, which have very simple body plans. However, even sponges, unicellular animals, and even protists such as slime molds have cell-to-cell signalling mechanisms that are precursors to those of neurons. The nervous systems of the radially symmetric organisms ctenophores (comb jellies) and cnidarians (which include anemones, hydras, corals and jellyfish) consist of a diffuse nerve net. All other animal species, with the exception of a few types of worm, have a nervous system containing a brain, a central cord (or two cords running in parallel), and nerves radiating from the brain and central cord. The size of the nervous system ranges from a few hundred cells in the simplest worms, to around 300 billion cells in African elephants.</p>
<p>Nervous tissue first arose in wormlike organisms about 550 to 600 million years ago. In humans and other vertebrates it consists of two main parts, the central nervous system (CNS) and the peripheral nervous system (PNS). The CNS consists of the brain and spinal cord. The PNS consists mainly of nerves, which are enclosed bundles of the long fibers or axons, that connect the CNS to every other part of the body. Nerves that transmit signals from the brain are called motor or efferent nerves, while those nerves that transmit information from the body to the CNS are called sensory or afferent. Spinal nerves serve both functions and are called mixed nerves. The PNS is divided into three separate subsystems, the somatic, autonomic, and enteric nervous systems. Somatic nerves carry sensory information from the periphery to the CNS and signals for voluntary movement from the CNS to the muscles. The autonomic nervous system is further subdivided into the sympathetic and the parasympathetic nervous systems. The sympathetic nervous system is activated in cases of emergencies to mobilize energy, while the parasympathetic nervous system is activated when organisms are in a relaxed state. The enteric nervous system functions to control the gastrointestinal system. Both autonomic and enteric nervous systems function involuntarily. Nerves that exit from the cranium are called cranial nerves while those exiting from the spinal cord are called spinal nerves.</p>
</div>
<div id="the-cells-of-the-nervous-system" class="section level2">
<h2><span class="header-section-number">25.2</span> The Cells Of The Nervous System</h2>
<p>At the cellular level, the nervous system is defined by the presence of a special type of cell, called the neuron, also known as a nerve cell. Neurons have special structures that allow them to receive and send signals from and to other cells. They send these signals in the form of electrochemical waves traveling along thin fibers called axons, which cause chemicals called neurotransmitters to be released at junctions called synapses. Neurons usually receive signalls at tree-like processes called dendrites. A cell that receives a synaptic signal from another neuron may be excited, inhibited, or otherwise modulated. The connections between neurons can form neural pathways, neural circuits, and larger networks that generate an organism’s perception of the world and determine its behavior. Along with neurons, the nervous system contains other specialized cells called glial cells (or simply glia), which provide structural and metabolic support.</p>

<div class="figure" style="text-align: center"><span id="fig:pyramidalcell"></span>
<img src="figures/nervoussystem/RabbitPyramidalCellCajalMetodos.jpg" alt="A nerve cell from the cerebral cortex of a rabbit. Notice the extensive tree of dendrites at the top, the long axon at the bottom. Because of the pyramid-like shape of the cell body, this type of neuron is referred to as a pyramidal cell." width="70%" />
<p class="caption">
Figure 25.4: A nerve cell from the cerebral cortex of a rabbit. Notice the extensive tree of dendrites at the top, the long axon at the bottom. Because of the pyramid-like shape of the cell body, this type of neuron is referred to as a pyramidal cell.
</p>
</div>
<p>Even in the nervous system of a single species such as humans, hundreds of different types of neurons exist, with a wide variety of morphologies and functions. These include sensory neurons that convert physical stimuli such as light and sound into neural signals, and motor neurons that activate muscles or glands; however in many species the great majority of neurons participate in the formation of centralized structures (the brain and ganglia) and they receive all of their input from other neurons and send their output to other neurons.</p>
<p>Glial cells (named from the Greek for “glue”) are non-neuronal cells that provide support and nutrition, maintain homeostasis, form myelin, and participate in signal transmission in the nervous system. In the human brain, it is estimated that the total number of glia roughly equals the number of neurons, although the proportions vary in different brain areas. Among the most important functions of glial cells are to support neurons and hold them in place; to supply nutrients to neurons; to insulate neurons electrically; to destroy pathogens and remove dead neurons; and to provide guidance cues directing the axons of neurons to their targets. A very important type of glial cell (oligodendrocytes in the central nervous system, and Schwann cells in the peripheral nervous system) generates layers of a fatty substance called myelin that wraps around axons and provides electrical insulation which allows them to transmit action potentials much more rapidly and efficiently. Microglia serve as important resident immune cells within the central nervous system.</p>
</div>
<div id="comparative-anatomy-and-evolution-of-nervous-systems" class="section level2">
<h2><span class="header-section-number">25.3</span> Comparative Anatomy And Evolution Of Nervous Systems</h2>
<p>Porifera (sponges) have no cells connected to each other by synaptic junctions, that is, no neurons, and therefore no nervous system. They do, however, have homologs of many genes that play key roles in synaptic function. Recent studies have shown that sponge cells express a group of proteins that cluster together to form a structure resembling a postsynaptic density (the signal-receiving part of a synapse). However, the function of this structure is currently unclear. Although sponge cells do not show synaptic transmission, they do communicate with each other via calcium waves and other impulses, which mediate some simple actions such as whole-body contraction.</p>
<p>Radiata such as the cnidaria (jellyfish) and ctenophora (comb jellies) have diffuse nerve nets rather than a central nervous system. In most jellyfish the nerve net is spread more or less evenly across the body; in comb jellies it is concentrated near the mouth. The nerve nets consist of sensory neurons, which pick up chemical, tactile, and visual signals; motor neurons, which can activate contractions of the body wall; and intermediate neurons, which detect patterns of activity in the sensory neurons and, in response, send signals to groups of motor neurons. In some cases groups of intermediate neurons are clustered into discrete ganglia.</p>
<p>The development of the nervous system in radiata is relatively unstructured. Unlike bilaterians, radiata only have two primordial cell layers, endoderm and ectoderm. Neurons are generated from a special set of ectodermal precursor cells, which also serve as precursors for every other ectodermal cell type.</p>
<p>The vast majority of existing animals are bilaterians, meaning animals with left and right sides that are approximate mirror images of each other. All bilateria are thought to have descended from a common wormlike ancestor that appeared in the Ediacaran period, 550–600 million years ago. The fundamental bilaterian body form is a tube with a hollow gut cavity running from mouth to anus, and a nerve cord with an enlargement (a “ganglion”) for each body segment, with an especially large ganglion at the front, called the “brain”.</p>

<div class="figure" style="text-align: center"><span id="fig:invertebratenervoussystem"></span>
<img src="figures/nervoussystem/invertebrate_nervoussystems.jpg" alt="Comparison of nervous systems of invertebrates. Top left: A diffuse nerve net in Actinia (a genus of sea anemones in the family Actiniidae in the phylum Cnidaria); top right: The nervous system of Anadonta anatina, a freshwater mussel in the family Unionidae in the phylum Mollusca. c, foot; k, pedal ganglion; i, cerebro-pedal connective; g, cerebral ganglion; h, cerebral connective; a, anterior adductor muscle; r, q, anterior pallial nerves; d, liver; s, visceral nerve; l, cerebro-visceral connective; e, gill; f, edge of mantle; n, branchial nerves; m, visceral ganglion; o, posterior pallial nerves; b, posterior adductor muscle; p, lateral pallial nerves; bottom left: the nervous system of Alitta virens, a polychaete worm in the phylum Annelida. J, jaws; b, antennal nerves; c, palpal nerves; f, ganglia for the dorsal peristomial cirri; n1 , ganglion; n, nerves for the dissepimenta; m, parapodial nerves; i, parapodial branch; h, ventral chain of ganglia; C, cerebral ganglion; o, nerve passing through dissepiment to preceding segment; k, parapodial ganglion. Bottom right: the nervous system of an insect (Arthropoda). From Morphology of invertebrate types, by Alexander Petrunkevitch. New York, Macmillan company, 1916." width="70%" />
<p class="caption">
Figure 25.5: Comparison of nervous systems of invertebrates. Top left: A diffuse nerve net in <em>Actinia</em> (a genus of sea anemones in the family <em>Actiniidae</em> in the phylum <em>Cnidaria</em>); top right: The nervous system of <em>Anadonta anatina</em>, a freshwater mussel in the family <em>Unionidae</em> in the phylum <em>Mollusca</em>. c, foot; k, pedal ganglion; i, cerebro-pedal connective; g, cerebral ganglion; h, cerebral connective; a, anterior adductor muscle; r, q, anterior pallial nerves; d, liver; s, visceral nerve; l, cerebro-visceral connective; e, gill; f, edge of mantle; n, branchial nerves; m, visceral ganglion; o, posterior pallial nerves; b, posterior adductor muscle; p, lateral pallial nerves; bottom left: the nervous system of <em>Alitta virens</em>, a polychaete worm in the phylum <em>Annelida</em>. J, jaws; b, antennal nerves; c, palpal nerves; f, ganglia for the dorsal peristomial cirri; n<sup>1</sup> , ganglion; n, nerves for the dissepimenta; m, parapodial nerves; i, parapodial branch; h, ventral chain of ganglia; C, cerebral ganglion; o, nerve passing through dissepiment to preceding segment; k, parapodial ganglion. Bottom right: the nervous system of an insect (<em>Arthropoda</em>). From <a href="https://www.biodiversitylibrary.org/ia/morphologyofinve00petr#page/7/mode/1up">Morphology of invertebrate types, by Alexander Petrunkevitch. New York, Macmillan company, 1916.</a>
</p>
</div>
<p>Even mammals, including humans, show the segmented bilaterian body plan at the level of the nervous system. The spinal cord contains a series of segmental ganglia, each giving rise to motor and sensory nerves that innervate a portion of the body surface and underlying musculature. On the limbs, the layout of the innervation pattern is complex, but on the trunk it gives rise to a series of narrow bands. The top three segments belong to the brain, giving rise to the forebrain, midbrain, and hindbrain.</p>
<p>Bilaterians can be divided, based on events that occur very early in embryonic development, into two groups (superphyla) called protostomes and deuterostomes. Deuterostomes include vertebrates as well as echinoderms, hemichordates (mainly acorn worms), and Xenoturbellidans. Protostomes, the more diverse group, include arthropods, molluscs, and numerous types of worms. There is a basic difference between the two groups in the placement of the nervous system within the body: protostomes possess a nerve cord on the ventral (usually bottom) side of the body, whereas in deuterostomes the nerve cord is on the dorsal (usually top) side. In fact, numerous aspects of the body are inverted between the two groups, including the expression patterns of several genes that show dorsal-to-ventral gradients. Most anatomists now consider that the bodies of protostomes and deuterostomes are “flipped over” with respect to each other, a hypothesis that was first proposed by Geoffroy Saint-Hilaire for insects in comparison to vertebrates. Thus insects, for example, have nerve cords that run along the ventral midline of the body, while all vertebrates have spinal cords that run along the dorsal midline.</p>
<p>There are a few types of existing bilaterians that lack a recognizable brain, including echinoderms and tunicates. It has not been definitively established whether the existence of these brainless species indicates that the earliest bilaterians lacked a brain, or whether their ancestors evolved in a way that led to the disappearance of a previously existing brain structure.</p>
<p>The diversity of invertebrate body plans is matched by an equal diversity in brain structures. Two groups of invertebrates have notably complex brains: arthropods (insects, crustaceans, arachnids, and others), and cephalopods (octopuses, squids, and similar molluscs). The brains of arthropods and cephalopods arise from twin parallel nerve cords that extend through the body of the animal. Arthropods have a central brain, the supraesophageal ganglion, with three divisions and large optical lobes behind each eye for visual processing. Cephalopods such as the octopus and squid have the largest brains of any invertebrates.</p>
<p>There are several invertebrate species whose brains have been studied intensively because they have properties that make them convenient for experimental work:</p>
<ul>
<li>Fruit flies (<em>Drosophila melanogaster</em>), because of the large array of techniques available for studying their genetics, have been a natural subject for studying the role of genes in brain development. In spite of the large evolutionary distance between insects and mammals, many aspects of <em>Drosophila</em> neurogenetics have been shown to be relevant to humans. The first biological clock genes, for example, were identified by examining <em>Drosophila</em> mutants that showed disrupted daily activity cycles. A search in the genomes of vertebrates revealed a set of analogous genes, which were found to play similar roles in the mouse biological clock—and therefore almost certainly in the human biological clock as well. Studies done on Drosophila, also show that most neuropil regions of the brain are continuously reorganized throughout life in response to specific living conditions.</li>
<li>The nematode worm <em>Caenorhabditis elegans</em>, like Drosophila, has been studied largely because of its importance in genetics. In the early 1970s, <a href="https://en.wikipedia.org/wiki/Sydney_Brenner">Sydney Brenner</a> chose it as a model organism for studying the way that genes control development. One of the advantages of working with this worm is that the body plan is very stereotyped: the nervous system of the hermaphrodite contains exactly 302 neurons, always in the same places, making identical synaptic connections in every worm. Brenner’s team sliced worms into thousands of ultrathin sections and photographed each one under an electron microscope, then visually matched fibers from section to section, to map out every neuron and synapse in the entire body. The complete neuronal wiring diagram of <em>C. elegans</em> – its connectome was achieved. Nothing approaching this level of detail is available for any other organism, and the information gained has enabled a multitude of studies that would otherwise have not been possible.</li>
<li>The sea slug <em>Aplysia californica</em> was chosen by Nobel Prize-winning neurophysiologist <a href="https://en.wikipedia.org/wiki/Eric_Kandel">Eric Kandel</a> as a model for studying the cellular basis of learning and memory, because of the simplicity and accessibility of its nervous system, and it has been examined in hundreds of experiments.</li>
</ul>
<p>Worms are the simplest bilaterian animals, and reveal the basic structure of the bilaterian nervous system in the most straightforward way. As an example, earthworms have dual nerve cords running along the length of the body and merging at the tail and the mouth. These nerve cords are connected by transverse nerves like the rungs of a ladder. These transverse nerves help coordinate the two sides of the animal. Two ganglia at the head (the “nerve ring”) end function similar to a simple brain. Photoreceptors on the animal’s eyespots provide sensory information on light and dark.</p>
<p>Arthropods, such as insects and crustaceans, have a nervous system made up of a series of ganglia, connected by a ventral nerve cord made up of two parallel connectives running along the length of the belly. Typically, each body segment has one ganglion on each side, though some ganglia are fused to form the brain and other large ganglia. The head segment contains the brain, also known as the supraesophageal ganglion. In the insect nervous system, the brain is anatomically divided into the protocerebrum, deutocerebrum, and tritocerebrum. Immediately behind the brain is the subesophageal ganglion, which is composed of three pairs of fused ganglia. It controls the mouthparts, the salivary glands and certain muscles. Many arthropods have well-developed sensory organs, including compound eyes for vision and antennae for olfaction and pheromone sensation. The sensory information from these organs is processed by the brain.</p>
<p>In insects, many neurons have cell bodies that are positioned at the edge of the brain and are electrically passive—the cell bodies serve only to provide metabolic support and do not participate in signalling. A protoplasmic fiber runs from the cell body and branches profusely, with some parts transmitting signals and other parts receiving signals. Thus, most parts of the insect brain have passive cell bodies arranged around the periphery, while the neural signal processing takes place in a tangle of protoplasmic fibers called neuropil, in the interior.</p>
<p>Brains are most simply compared in terms of their size. The relationship between brain size, body size and other variables has been studied across a wide range of vertebrate species. As a rule, brain size increases with body size, but not in a simple linear proportion. In general, smaller animals tend to have larger brains, measured as a fraction of body size. For mammals, the relationship between brain volume and body mass essentially follows a power law with an exponent of about 0.75. This formula describes the central tendency, but every family of mammals departs from it to some degree, in a way that reflects in part the complexity of their behavior. For example, primates have brains 5 to 10 times larger than the formula predicts. Predators tend to have larger brains than their prey, relative to body size.</p>
<p>All vertebrate brains share a common underlying form, which appears most clearly during early stages of embryonic development. In its earliest form, the brain appears as three swellings at the front end of the neural tube; these swellings eventually become the forebrain, midbrain, and hindbrain (the prosencephalon, mesencephalon, and rhombencephalon, respectively). At the earliest stages of brain development, the three areas are roughly equal in size. In many classes of vertebrates, such as fish and amphibians, the three parts remain similar in size in the adult, but in mammals the forebrain becomes much larger than the other parts, and the midbrain becomes very small.</p>

<div class="figure" style="text-align: center"><span id="fig:vertebratecns"></span>
<img src="figures/nervoussystem/cns_comparison.svg" alt="Dorsal views of the central nervous systems of the teleosts (from left to right) Trigla hirundo (a) and Mola mola, the urodele Ambystoma tigrinum, the anuran Xenopus laevis, the tortoise Testudo hermanni, the tegu lizard Tupinambis teguixin, the pigeon, the cat and human. In a, c, d, e, f, g and j the full length of the spinalcord, including the filum terminale (where present) is shown; in Mola mola and the cat most of the filum terminale is cut. Vertical black bars correspond to 1 cm in length. Modified from Nieuwenhuys, R., ten Donkelaar, H. J., &amp; Nicholson, C. (1998). The Meaning of It All. The Central Nervous System of Vertebrates, 2135–2195" width="70%" />
<p class="caption">
Figure 25.6: Dorsal views of the central nervous systems of the teleosts (from left to right) <em>Trigla hirundo</em> (a) and <em>Mola mola</em>, the urodele <em>Ambystoma tigrinum</em>, the anuran <em>Xenopus laevis</em>, the tortoise <em>Testudo hermanni</em>, the tegu lizard <em>Tupinambis teguixin</em>, the pigeon, the cat and human. In a, c, d, e, f, g and j the full length of the spinalcord, including the filum terminale (where present) is shown; in <em>Mola mola</em> and the cat most of the filum terminale is cut. Vertical black bars correspond to 1 cm in length. Modified from <a href="https://doi.org/10.1007/978-3-642-18262-4_24">Nieuwenhuys, R., ten Donkelaar, H. J., &amp; Nicholson, C. (1998). The Meaning of It All. The Central Nervous System of Vertebrates, 2135–2195</a>
</p>
</div>
<p>The brains of vertebrates are made of very soft tissue. Living brain tissue is pinkish on the outside and mostly white on the inside, with subtle variations in color. Vertebrate brains are surrounded by a system of connective tissue membranes called meninges that separate the skull from the brain. Blood vessels enter the central nervous system through holes in the meningeal layers. The cells in the blood vessel walls are joined tightly to one another, forming the blood–brain barrier, which blocks the passage of many toxins and pathogens (though at the same time blocking antibodies and some drugs, thereby presenting special challenges in treatment of diseases of the brain).</p>
<p>Neuroanatomists usually divide the vertebrate brain into six main regions: the telencephalon (cerebral hemispheres), diencephalon (thalamus and hypothalamus), mesencephalon (midbrain), cerebellum, pons, and medulla oblongata. Each of these areas has a complex internal structure. Some parts, such as the cerebral cortex and the cerebellar cortex, consist of layers that are folded or convoluted to fit within the available space. Other parts, such as the thalamus and hypothalamus, consist of clusters of many small nuclei. Thousands of distinguishable areas can be identified within the vertebrate brain based on fine distinctions of neural structure, chemistry, and connectivity.</p>
<p>There is an anatomical convention that a cluster of neurons in the brain or spinal cord is called a nucleus, whereas a cluster of neurons in the periphery is called a ganglion. There are, however, a few exceptions to this rule, notably including the part of the forebrain called the basal ganglia.</p>
<p>Although the same basic components are present in all vertebrate brains, some branches of vertebrate evolution have led to substantial distortions of brain geometry, especially in the forebrain area. The brain of a shark shows the basic components in a straightforward way, but in teleost fishes (the great majority of existing fish species), the forebrain has become “everted”, like a sock turned inside out. In birds, there are also major changes in forebrain structure. These distortions can make it difficult to match brain components from one species with those of another species.</p>
<p>Here is a list of some of the most important vertebrate brain components, along with a brief description of their functions as currently understood:</p>
<ul>
<li>The medulla, along with the spinal cord, contains many small nuclei involved in a wide variety of sensory and involuntary motor functions such as vomiting, heart rate and digestive processes.</li>
<li>The pons lies in the brainstem directly above the medulla. Among other things, it contains nuclei that control often voluntary but simple acts such as sleep, respiration, swallowing, bladder function, equilibrium, eye movement, facial expressions, and posture.</li>
<li>The hypothalamus is a small region at the base of the forebrain, whose complexity and importance belies its size. It is composed of numerous small nuclei, each with distinct connections and neurochemistry. The hypothalamus is engaged in additional involuntary or partially voluntary acts such as sleep and wake cycles, eating and drinking, and the release of some hormones.</li>
<li>The thalamus is a collection of nuclei with diverse functions: some are involved in relaying information to and from the cerebral hemispheres, while others are involved in motivation. The subthalamic area (zona incerta) seems to contain action-generating systems for several types of “consummatory” behaviors such as eating, drinking, defecation, and copulation.</li>
<li>The cerebellum modulates the outputs of other brain systems, whether motor related or thought related, to make them certain and precise. Removal of the cerebellum does not prevent an animal from doing anything in particular, but it makes actions hesitant and clumsy. This precision is not built-in, but learned by trial and error. The muscle coordination learned while riding a bicycle is an example of a type of neural plasticity that may take place largely within the cerebellum. 10% of the brain’s total volume consists of the cerebellum and 50% of all neurons are held within its structure.</li>
<li>The optic tectum allows actions to be directed toward points in space, most commonly in response to visual input. In mammals it is usually referred to as the superior colliculus, and its best-studied function is to direct eye movements. It also directs reaching movements and other object-directed actions. It receives strong visual inputs, but also inputs from other senses that are useful in directing actions, such as auditory input in owls and input from the thermosensitive pit organs in snakes. In some primitive fishes, such as lampreys, this region is the largest part of the brain. The superior colliculus is part of the midbrain.</li>
<li>The pallium is a layer of gray matter that lies on the surface of the forebrain and is the most complex and most recent evolutionary development of the brain as an organ. In reptiles and mammals, it is called the cerebral cortex. Multiple functions involve the pallium, including smell and spatial memory. In mammals, where it becomes so large as to dominate the brain, it takes over functions from many other brain areas. In many mammals, the cerebral cortex consists of folded bulges called gyri that create deep furrows or fissures called sulci. The folds increase the surface area of the cortex and therefore increase the amount of gray matter and the amount of information that can be stored and processed.</li>
<li>The hippocampus, strictly speaking, is found only in mammals. However, the area it derives from, the medial pallium, has counterparts in all vertebrates. There is evidence that this part of the brain is involved in complex events such as spatial memory and navigation in fishes, birds, reptiles, and mammals.</li>
<li>The basal ganglia are a group of interconnected structures in the forebrain. The primary function of the basal ganglia appears to be action selection: they send inhibitory signals to all parts of the brain that can generate motor behaviors, and in the right circumstances can release the inhibition, so that the action-generating systems are able to execute their actions. Reward and punishment exert their most important neural effects by altering connections within the basal ganglia.</li>
<li>The olfactory bulb is a special structure that processes olfactory sensory signals and sends its output to the olfactory part of the pallium. It is a major brain component in many vertebrates, but is greatly reduced in humans and other primates (whose senses are dominated by information acquired by sight rather than smell).</li>
</ul>
<p>The most obvious difference between the brains of mammals and other vertebrates is in terms of size. On average, a mammal has a brain roughly twice as large as that of a bird of the same body size, and ten times as large as that of a reptile of the same body size.</p>
<p>Size, however, is not the only difference: there are also substantial differences in shape. The hindbrain and midbrain of mammals are generally similar to those of other vertebrates, but dramatic differences appear in the forebrain, which is greatly enlarged and also altered in structure. The cerebral cortex is the part of the brain that most strongly distinguishes mammals. In non-mammalian vertebrates, the surface of the cerebrum is lined with a comparatively simple three-layered structure called the pallium. In mammals, the pallium evolves into a complex six-layered structure called neocortex or isocortex. Several areas at the edge of the neocortex, including the hippocampus and amygdala, are also much more extensively developed in mammals than in other vertebrates.</p>
<p>The elaboration of the cerebral cortex carries with it changes to other brain areas. The superior colliculus, which plays a major role in visual control of behavior in most vertebrates, shrinks to a small size in mammals, and many of its functions are taken over by visual areas of the cerebral cortex. The cerebellum of mammals contains a large portion (the neocerebellum) dedicated to supporting the cerebral cortex, which has no counterpart in other vertebrates.</p>
<p>The brains of humans and other primates contain the same structures as the brains of other mammals, but are generally larger in proportion to body size. The encephalization quotient (EQ) is used to compare brain sizes across species. It takes into account the nonlinearity of the brain-to-body relationship. Humans have an average EQ in the 7-to-8 range, while most other primates have an EQ in the 2-to-3 range. Dolphins have values higher than those of primates other than humans, but nearly all other mammals have EQ values that are substantially lower.</p>

<div class="figure" style="text-align: center"><span id="fig:mammaliancns"></span>
<img src="figures/nervoussystem/fnana-05-00029-g007.jpg" alt="Variability of brain size and external topography. Photographs and weights of the brains of different species. Primates: human (Homo sapiens, 1.176 kg), chimpanzee (Pan troglodytes, 273 g), baboon (Papio cynocephalus, 151 g), mandrill (Mandrillus sphinx, 123 g), macaque (Macaca tonkeana, 110 g). Carnivores: bear (Ursus arctos, 289 g), lion (Panthera leo, 165 g), cheetah (Acinonyx jubatus, 119 g), dog (Canis familiaris, 95 g), cat (Felis catus, 32 g). Artiodactyls: giraffe (Giraffa camelopardalis, 700 g), kudu (Tragelaphus strepsiceros, 166 g), mouflon (Ovis musimon, 118 g), ibex (Capra pyrenaica, 115 g); peccary (Tayassu pecari, 41 g). Marsupials: wallaby (Protemnodon rufogrisea, 28 g). Lagomorphs: rabbit (Oryctolagus cuniculus, 5.2 g). Rodents: rat (Rattus rattus, 2.6 g), mouse (Mus musculus, 0.5 g). The chimpanzee brain was kindly supplied by Dr. Dean Falk. The rest of non-human brains were from material used in Ballesteros-Yánez et al., 2005). Scale bar: 5 cm. From DeFelipe J (2011) The evolution of the brain, the human nature of cortical circuits, and intellectual creativity. Front. Neuroanat. 5:29" width="70%" />
<p class="caption">
Figure 25.7: Variability of brain size and external topography. Photographs and weights of the brains of different species. Primates: human (Homo sapiens, 1.176 kg), chimpanzee (Pan troglodytes, 273 g), baboon (Papio cynocephalus, 151 g), mandrill (Mandrillus sphinx, 123 g), macaque (Macaca tonkeana, 110 g). Carnivores: bear (Ursus arctos, 289 g), lion (Panthera leo, 165 g), cheetah (Acinonyx jubatus, 119 g), dog (Canis familiaris, 95 g), cat (Felis catus, 32 g). Artiodactyls: giraffe (Giraffa camelopardalis, 700 g), kudu (Tragelaphus strepsiceros, 166 g), mouflon (Ovis musimon, 118 g), ibex (Capra pyrenaica, 115 g); peccary (Tayassu pecari, 41 g). Marsupials: wallaby (Protemnodon rufogrisea, 28 g). Lagomorphs: rabbit (Oryctolagus cuniculus, 5.2 g). Rodents: rat (Rattus rattus, 2.6 g), mouse (Mus musculus, 0.5 g). The chimpanzee brain was kindly supplied by Dr. Dean Falk. The rest of non-human brains were from material used in Ballesteros-Yánez et al., 2005). Scale bar: 5 cm. From <a href="https://www.frontiersin.org/article/10.3389/fnana.2011.00029">DeFelipe J (2011) The evolution of the brain, the human nature of cortical circuits, and intellectual creativity. Front. Neuroanat. 5:29</a>
</p>
</div>
<p>Most of the enlargement of the primate brain comes from a massive expansion of the cerebral cortex, especially the prefrontal cortex and the parts of the cortex involved in vision. The visual processing network of primates includes at least 30 distinguishable brain areas, with a complex web of interconnections. It has been estimated that visual processing areas occupy more than half of the total surface of the primate neocortex. The prefrontal cortex carries out functions that include planning, working memory, motivation, attention, and executive control. It takes up a much larger proportion of the brain for primates than for other species, and an especially large fraction of the human brain.</p>
<p>The peripheral nervous system (PNS) is a collective term for the nervous system structures that do not lie within the CNS. The large majority of the axon bundles called nerves are considered to belong to the PNS, even when the cell bodies of the neurons to which they belong reside within the brain or spinal cord. The PNS is divided into somatic and visceral parts. The somatic part consists of the nerves that innervate the skin, joints, and muscles. The cell bodies of somatic sensory neurons lie in dorsal root ganglia of the spinal cord. The visceral part, also known as the autonomic nervous system, contains neurons that innervate the internal organs, blood vessels, and glands. The autonomic nervous system itself consists of two parts: the sympathetic nervous system and the parasympathetic nervous system. Some authors also include sensory neurons whose cell bodies lie in the periphery (for senses such as hearing) as part of the PNS; others, however, omit them.</p>
<p>The vertebrate nervous system can also be divided into areas called gray matter and white matter. Gray matter (which is only gray in preserved tissue, and is better described as pink or light brown in living tissue) contains a high proportion of cell bodies of neurons. White matter is composed mainly of myelinated axons, and takes its color from the myelin. White matter includes all of the nerves, and much of the interior of the brain and spinal cord. Gray matter is found in clusters of neurons in the brain and spinal cord, and in cortical layers that line their surfaces.</p>
</div>
<div id="the-human-brain" class="section level2">
<h2><span class="header-section-number">25.4</span> The Human Brain</h2>
<p>The adult human brain weighs on average about 1.2–1.4 kg (2.6–3.1 lb) which is about 2% of the total body weight, with a volume of around 1260 cm<sup>3</sup> in men and 1130 cm<sup>3</sup> in women. There is substantial individual variation, with the standard reference range for men being 1,180–1,620 g (2.60–3.57 lb) and for women 1,030–1,400 g (2.27–3.09 lb).</p>
<p>The human brain is divided into nearly symmetrical left and right hemispheres by a deep groove, the longitudinal fissure. Each hemisphere is conventionally divided into four main lobes; the frontal lobe, parietal lobe, temporal lobe, and occipital lobe, named according to the skull bones that overlie them. The surface of the brain is folded into ridges (gyri) and grooves (sulci), many of which are named, usually according to their position, such as the frontal gyrus of the frontal lobe or the central sulcus separating the central regions of the hemispheres. There are many small variations in the secondary and tertiary folds.</p>

<div class="figure" style="text-align: center"><span id="fig:principallobes"></span>
<img src="figures/nervoussystem/GrayAnat1918p821.jpg" alt="Principal lobes and fissures of the cerebrum viewed laterally. From Gray Henry, Anatomy of the Human Body. 20th Edition, Lea &amp; Febiger, Philadelphia &amp; New York, 1918" width="70%" />
<p class="caption">
Figure 25.8: Principal lobes and fissures of the cerebrum viewed laterally. From <a href="https://archive.org/details/anatomyofhumanbo1918gray/page/n6/mode/2up">Gray Henry, Anatomy of the Human Body. 20<sup>th</sup> Edition, Lea &amp; Febiger, Philadelphia &amp; New York, 1918</a>
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:gyrilateral"></span>
<img src="figures/nervoussystem/GrayAnat1918p819.jpg" alt="Diagram showing a lateral view of the ridges (gyri) and grooves (sulci) of the left hemisphere of the brain. From Gray Henry, Anatomy of the Human Body. 20th Edition, Lea &amp; Febiger, Philadelphia &amp; New York, 1918" width="70%" />
<p class="caption">
Figure 25.9: Diagram showing a lateral view of the ridges (gyri) and grooves (sulci) of the left hemisphere of the brain. From <a href="https://archive.org/details/anatomyofhumanbo1918gray/page/n6/mode/2up">Gray Henry, Anatomy of the Human Body. 20<sup>th</sup> Edition, Lea &amp; Febiger, Philadelphia &amp; New York, 1918</a>
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:gyrimedial"></span>
<img src="figures/nervoussystem/GrayAnat1918p820.jpg" alt="Diagram showing a medial view of the ridges (gyri) and grooves (sulci) of the left hemisphere of the brain. From Gray Henry, Anatomy of the Human Body. 20th Edition, Lea &amp; Febiger, Philadelphia &amp; New York, 1918" width="70%" />
<p class="caption">
Figure 25.10: Diagram showing a medial view of the ridges (gyri) and grooves (sulci) of the left hemisphere of the brain. From <a href="https://archive.org/details/anatomyofhumanbo1918gray/page/n6/mode/2up">Gray Henry, Anatomy of the Human Body. 20<sup>th</sup> Edition, Lea &amp; Febiger, Philadelphia &amp; New York, 1918</a>
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:gyribottom"></span>
<img src="figures/nervoussystem/gyri_bot.jpg" alt="Diagram showing a view from the bottom of the ridges (gyri) and grooves (sulci) of the left hemisphere of the brain." width="70%" />
<p class="caption">
Figure 25.11: Diagram showing a view from the bottom of the ridges (gyri) and grooves (sulci) of the left hemisphere of the brain.
</p>
</div>
<p>Although the human brain represents only 2% of the body weight, it receives 15% of the cardiac output, 20% of total body oxygen consumption, and 25% of total body glucose utilization. The brain mostly uses glucose for energy, and deprivation of glucose, as can happen in hypoglycemia, can result in loss of consciousness. The energy consumption of the brain does not vary greatly over time, but active regions of the cortex consume somewhat more energy than inactive regions: this fact forms the basis for the functional brain imaging methods PET and fMRI. These functional imaging techniques provide a three-dimensional image of metabolic activity.</p>
<p>The simplest way to gain information about brain anatomy is by visual inspection, but many more sophisticated techniques have been developed. Brain tissue in its natural state is too soft to work with, but it can be hardened by immersion in alcohol or other fixatives, and then sliced apart for examination of the interior. Visually, the interior of the brain consists of areas of so-called grey matter, with a dark color, separated by areas of white matter, with a lighter color. Further information can be gained by staining slices of brain tissue with a variety of chemicals that bring out areas where specific types of molecules are present in high concentrations. It is also possible to examine the microstructure of brain tissue using a microscope, and to trace the pattern of connections from one brain area to another.</p>
</div>
<div id="development-of-the-nervous-system" class="section level2">
<h2><span class="header-section-number">25.5</span> Development Of The Nervous System</h2>
<p>All bilaterian animals at an early stage of development form a gastrula, which is polarized, with one end called the animal pole and the other the vegetal pole. The gastrula has the shape of a disk with three layers of cells, an inner layer called the endoderm, which gives rise to the lining of most internal organs, a middle layer called the mesoderm, which gives rise to the bones and muscles, and an outer layer called the ectoderm, which gives rise to the skin and nervous system.</p>
<p>In vertebrates, the first sign of the nervous system is the appearance of a thin strip of cells along the center of the back, called the neural plate. The inner portion of the neural plate (along the midline) is destined to become the central nervous system (CNS), the outer portion the peripheral nervous system (PNS). As development proceeds, a fold called the neural groove appears along the midline. This fold deepens, and then closes up at the top. At this point the future CNS appears as a cylindrical structure called the neural tube, whereas the future PNS appears as two strips of tissue called the neural crest, running lengthwise above the neural tube. The sequence of stages from neural plate to neural tube and neural crest is known as neurulation.</p>
<p>In the early 20<sup>th</sup> century, a set of famous experiments by <a href="https://en.wikipedia.org/wiki/Hans_Spemann">Hans Spemann</a> and <a href="https://en.wikipedia.org/wiki/Hilde_Mangold">Hilde Mangold</a> showed that the formation of nervous tissue is “induced” by signals from a group of mesodermal cells called the organizer region. For decades, though, the nature of neural induction defeated every attempt to figure it out, until finally it was resolved by genetic approaches in the 1990s. Induction of neural tissue requires inhibition of the gene for a so-called bone morphogenetic protein, or BMP. Specifically the protein BMP4 appears to be involved. Two proteins called Noggin and Chordin, both secreted by the mesoderm, are capable of inhibiting BMP4 and thereby inducing ectoderm to turn into neural tissue. It appears that a similar molecular mechanism is involved for widely disparate types of animals, including arthropods as well as vertebrates. In some animals, however, another type of molecule called Fibroblast Growth Factor or FGF may also play an important role in induction.</p>
<p>Induction of neural tissues causes formation of neural precursor cells, called neuroblasts. In drosophila, neuroblasts divide asymmetrically, so that one product is a “ganglion mother cell” (GMC), and the other is a neuroblast. A GMC divides once, to give rise to either a pair of neurons or a pair of glial cells. In all, a neuroblast is capable of generating an indefinite number of neurons or glia.</p>
<p>One factor common to all bilateral organisms (including humans) is a family of secreted signaling molecules called neurotrophins which regulate the growth and survival of neurons. Because neurotrophins have now been identified in both vertebrate and invertebrates, this evidence suggests that neurotrophins were present in an ancestor common to bilateral organisms and may represent a common mechanism for nervous system formation.</p>
</div>
<div id="the-function-of-the-nervous-system" class="section level2">
<h2><span class="header-section-number">25.6</span> The Function Of The Nervous System</h2>
<p>Organisms need information to solve at least three kinds of problems: (a) to maintain an appropriate environment, i.e., homeostasis; (b) to time activities (e.g., seasonal changes in behavior) or synchronize activities with those of conspecifics; and (c) to locate and respond to resources or threats (e.g., by moving towards resources or evading or attacking threats). Organisms also need to transmit information in order to influence another’s behavior: to identify themselves, warn conspecifics of danger, coordinate activities, or deceive.</p>
<p>At the most basic level, the function of the nervous system is to send signals from one cell to others, or from one part of the body to others. There are multiple ways that a cell can send signals to other cells. One is by releasing chemicals called hormones into the internal circulation, so that they can diffuse to distant sites. In contrast to this “broadcast” mode of signaling, the nervous system provides “point-to-point” signals—neurons project their axons to specific target areas and make synaptic connections with specific target cells. Thus, neural signaling is capable of a much higher level of specificity than hormonal signaling. It is also much faster: the fastest nerve signals travel at speeds that exceed 100 meters per second.</p>
<p>At a more integrative level, the primary function of the nervous system is to control the body. It does this by extracting information from the environment using sensory receptors, sending signals that encode this information into the central nervous system, processing the information to determine an appropriate response, and sending output signals to muscles or glands to activate the response. The evolution of a complex nervous system has made it possible for various animal species to have advanced perception abilities such as vision, complex social interactions, rapid coordination of organ systems, and integrated processing of concurrent signals. In humans, the sophistication of the nervous system makes it possible to have language, abstract representation of concepts, transmission of culture, and many other features of human society that would not exist without the human brain.</p>
</div>
<div id="the-sensory-system" class="section level2">
<h2><span class="header-section-number">25.7</span> The Sensory System</h2>
<p>The sensory nervous system is a part of the nervous system responsible for processing sensory information. A sensory system consists of sensory neurons (including the sensory receptor cells), neural pathways, and parts of the brain involved in sensory perception. Commonly recognized sensory systems are those for vision, hearing, touch, taste, smell, and balance. In short, senses are transducers from the physical world to the realm of the mind where we interpret the information, creating our perception of the world around us.</p>
<p>Sensory systems code for four aspects of a stimulus; type (modality), intensity, location, and duration. Arrival time of a sound pulse and phase differences of continuous sound are used for sound localization. Certain receptors are sensitive to certain types of stimuli (for example, different mechanoreceptors respond best to different kinds of touch stimuli, like sharp or blunt objects). Receptors send impulses in certain patterns to send information about the intensity of a stimulus (for example, how loud a sound is). The location of the receptor that is stimulated gives the brain information about the location of the stimulus (for example, stimulating a mechanoreceptor in a finger will send information to the brain about that finger). The duration of the stimulus (how long it lasts) is conveyed by firing patterns of receptors. These impulses are transmitted to the brain through afferent neurons.</p>
<p>While debate exists among neurologists as to the specific number of senses due to differing definitions of what constitutes a sense, Gautama Buddha and Aristotle classified five ‘traditional’ human senses which have become universally accepted: touch, taste, smell, sight, and hearing. Other senses that have been well-accepted in most mammals, including humans, include nociception, equilibrioception, kinaesthesia, and thermoception. Furthermore, some nonhuman animals have been shown to possess alternate senses, including magnetoception and electroreception.</p>
<p>The human sensory system consists of the following subsystems:</p>
<ul>
<li>Somatosensory system consists of the receptors, transmitters (pathways) leading to area S1, and area S1 in the cortex that is involved in creating the conscious experience of the sensations labelled as touch or pressure, temperature (warm or cold), pain (including itch and tickle), and the sensations of muscle movement and joint position including posture, movement, and facial expression (collectively also called proprioception)</li>
<li>Visual system</li>
<li>Auditory system</li>
<li>Vestibular system</li>
<li>Olfactory system</li>
<li>Gustatory system</li>
</ul>
<p>The receptive field is the area of the body or environment to which a receptor organ and receptor cells respond. For instance, the part of the world an eye can see, is its receptive field; the light that each rod or cone can see, is its receptive field. Receptive fields have been identified for the visual system, auditory system and somatosensory system.</p>
</div>
<div id="the-motor-system" class="section level2">
<h2><span class="header-section-number">25.8</span> The Motor System</h2>
<p>The motor system is the set of central and peripheral structures in the nervous system that support motor functions, i.e. movement. Peripheral structures may include skeletal muscles and neural connections with muscle tissues. Central structures include cerebral cortex, brainstem, spinal cord, pyramidal system including the upper motor neurons, extrapyramidal system, cerebellum, and the lower motor neurons in the brainstem and the spinal cord.</p>
<p>The pyramidal motor system, also called the pyramidal tract or the corticospinal tract, start in the motor center of the cerebral cortex. There are upper and lower motor neurons in the corticospinal tract. The motor impulses originate in the giant pyramidal cells or Betz cells of the motor area; i.e., precentral gyrus of cerebral cortex. These are the upper motor neurons (UMN) of the corticospinal tract. The axons of these cells pass in the depth of the cerebral cortex to the corona radiata and then to the internal capsule passing through the posterior branch of internal capsule and continue to descend in the midbrain and the medulla oblongata. In the lower part of medulla oblongata 80 to 85% of these fibers decussate (pass to the opposite side) and descend in the white matter of the lateral funiculus of the spinal cord on the opposite side. The remaining 15 to 20% pass to the same side. Fibers for the extremities (limbs) pass 100% to the opposite side. The fibers of the corticospinal tract terminate at different levels in the anterior horn of the grey matter of the spinal cord. Here the lower motor neurons (LMN) of the spinal cord are located. Peripheral motor nerves carry the motor impulses from the anterior horn to the voluntary muscles.</p>
<p>The extrapyramidal system is called extrapyramidal to distinguish it from the tracts of the motor cortex that reach their targets by traveling through the pyramids of the medulla. The pyramidal tracts (corticospinal tract and corticobulbar tracts) may directly innervate motor neurons of the spinal cord or brainstem (anterior (ventral) horn cells or certain cranial nerve nuclei), whereas the extrapyramidal system centers on the modulation and regulation (indirect control) of anterior (ventral) horn cells.</p>
<p>Extrapyramidal tracts are chiefly found in the reticular formation of the pons and medulla, and target lower motor neurons in the spinal cord that are involved in reflexes, locomotion, complex movements, and postural control. These tracts are in turn modulated by various parts of the central nervous system, including the nigrostriatal pathway, the basal ganglia, the cerebellum, the vestibular nuclei, and different sensory areas of the cerebral cortex. All of these regulatory components can be considered part of the extrapyramidal system, in that they modulate motor activity without directly innervating motor neurons.</p>
</div>
<div id="neuronal-signalling" class="section level2">
<h2><span class="header-section-number">25.9</span> Neuronal Signalling</h2>
<p>Most neurons send signals via their axons, although some types are capable of dendrite-to-dendrite communication. (In fact, the types of neurons in the retina of the eye called amacrine cells have no axons, and communicate only via their dendrites.) Neural signals propagate along an axon in the form of electrochemical waves called action potentials, which produce cell-to-cell signals at points where axon terminals make synaptic contact with other cells.</p>
<p>Synapses may be electrical or chemical. Electrical synapses make direct electrical connections between neurons, but chemical synapses are much more common, and much more diverse in function. At a chemical synapse, the cell that sends signals is called presynaptic, and the cell that receives signals is called postsynaptic. Both the presynaptic and postsynaptic areas are full of molecular machinery that carries out the signalling process. The presynaptic area contains large numbers of tiny spherical vessels called synaptic vesicles, packed with neurotransmitter chemicals. When the presynaptic terminal is electrically stimulated, an array of molecules embedded in the membrane are activated, and cause the contents of the vesicles to be released into the narrow space between the presynaptic and postsynaptic membranes, called the synaptic cleft. The neurotransmitter then binds to receptors embedded in the postsynaptic membrane, causing them to enter an activated state. Depending on the type of receptor, the resulting effect on the postsynaptic cell may be excitatory, inhibitory, or modulatory in more complex ways. For example, release of the neurotransmitter acetylcholine at a synaptic contact between a motor neuron and a muscle cell induces rapid contraction of the muscle cell. The entire synaptic transmission process takes only a fraction of a millisecond, although the effects on the postsynaptic cell may last much longer (even indefinitely, in cases where the synaptic signal leads to the formation of a memory trace).</p>
<p>There are literally hundreds of different types of synapses. In fact, there are over a hundred known neurotransmitters, and many of them have multiple types of receptors. Many synapses use more than one neurotransmitter—a common arrangement is for a synapse to use one fast-acting small-molecule neurotransmitter such as glutamate or GABA, along with one or more peptide neurotransmitters that play slower-acting modulatory roles. Molecular neuroscientists generally divide receptors into two broad groups: chemically gated ion channels and second messenger systems. When a chemically gated ion channel is activated, it forms a passage that allows specific types of ions to flow across the membrane. Depending on the type of ion, the effect on the target cell may be excitatory or inhibitory. When a second messenger system is activated, it starts a cascade of molecular interactions inside the target cell, which may ultimately produce a wide variety of complex effects, such as increasing or decreasing the sensitivity of the cell to stimuli, or even altering gene transcription.</p>
<p>According to a rule called Dale’s principle, which has only a few known exceptions, a neuron releases the same neurotransmitters at all of its synapses. This does not mean, though, that a neuron exerts the same effect on all of its targets, because the effect of a synapse depends not on the neurotransmitter, but on the receptors that it activates. Because different targets can (and frequently do) use different types of receptors, it is possible for a neuron to have excitatory effects on one set of target cells, inhibitory effects on others, and complex modulatory effects on others still. Nevertheless, it happens that the two most widely used neurotransmitters, glutamate and GABA, each have largely consistent effects. Glutamate has several widely occurring types of receptors, but all of them are excitatory or modulatory. Similarly, GABA has several widely occurring receptor types, but all of them are inhibitory. Because of this consistency, glutamatergic cells are frequently referred to as “excitatory neurons”, and GABAergic cells as “inhibitory neurons”. Strictly speaking, this is an abuse of terminology—it is the receptors that are excitatory and inhibitory, not the neurons—but it is commonly seen even in scholarly publications.</p>
<p>One very important subset of synapses are capable of forming memory traces by means of long-lasting activity-dependent changes in synaptic strength. The best-known form of neural memory is a process called long-term potentiation (abbreviated LTP), which operates at synapses that use the neurotransmitter glutamate acting on a special type of receptor known as the NMDA receptor. The NMDA receptor functions as a molecular “conincidence detector”: although the NMDA-receptor associated ion-channel opens upon binding of glutamate, extracellular Mg<sup>2+</sup> ions will enter and block the channel immediately. Only concomitant membrane depolarization (e.g. induced by Na<sup>+</sup> influx via concomittantly stimulated non-NMDA (AMPA) type glutamate receptors in the same cell), will overcome the Mg<sup>2+</sup> block and allow Na<sup>+</sup> and Ca<sup>2+</sup> ions to enter the cell thourgh the NMDA-receptor. Calcium entering the postsynaptic cell via NMDA receptors then initiates a second messenger cascade that ultimately leads to an increase in the number of AMPA-type glutamate receptors in the target cell, thereby increasing the effective strength of the synapse. This change in strength can last for weeks or longer. Besides the NMDA-receptor based processes, further cellular mechanisms allow of the association between two different input signals converging on the same neuron, in a defined timeframe. Upon a simultaneous increase in the intracellular concentrations of cAMP and Ca2+, a transcriptional coactivator called TORC1 (CRTC1) becomes activated, that converts the temporal coincidence of the two second messengers into long term changes such as LTP. This cellular mechanism, through calcium-dependent adenylate cyclase activation, might also account for the detection of the repetitive stimulation of a given synapse.</p>
<p>In 1949, <a href="https://en.wikipedia.org/wiki/Donald_O._Hebb">Donald Hebb</a> postulated that synaptic efficiency will increase through repeated and persistent stimulation of a postsynaptic cell by a presynaptic cell. This is often informally summarized as “cells that fire together, wire together”. The theory was validated in part by the discovery of long-term potentiation. Studies of LTP on multiple presynaptic cells stimulating a postsynaptic cell uncovered the property of associativity. A weak neuronal stimulation onto a pyramidal neuron may not induce long-term potentiation. However, this same stimulation paired with a simultaneous strong stimulation from another neuron will strengthen both synapses. This process suggests that two neuronal pathways converging on the same cell may both strengthen if stimulated coincidentally.</p>
<p>Since the discovery of LTP in 1973, many other types of synaptic memory traces have been found, involving increases or decreases in synaptic strength that are induced by varying conditions, and last for variable periods of time. The reward system, that reinforces desired behaviour for example, depends on a variant form of LTP that is conditioned on an extra input coming from a reward-signalling pathway that uses dopamine as neurotransmitter. All these forms of synaptic modifiability, taken collectively, give rise to neural plasticity, that is, to a capability for the nervous system to adapt itself to variations in the environment.</p>
</div>
<div id="neurons-and-glial-cells" class="section level2">
<h2><span class="header-section-number">25.10</span> Neurons And Glial Cells</h2>
<div id="neurons" class="section level3">
<h3><span class="header-section-number">25.10.1</span> Neurons</h3>
<p>The neuron doctrine is the now fundamental idea that neurons are the basic structural and functional units of the nervous system. The theory was put forward by Santiago Ramón y Cajal in the late 19th century. It held that neurons are discrete cells (not connected in a meshwork), acting as metabolically distinct units.</p>
<p>Later discoveries yielded refinements to the doctrine. For example, glial cells, which are not considered neurons, play an essential role in information processing. Also, electrical synapses are more common than previously thought, comprising direct, cytoplasmic connections between neurons. In fact, neurons can form even tighter couplings: the squid giant axon arises from the fusion of multiple axons.</p>
<p>Ramón y Cajal also postulated the Law of Dynamic Polarization, which states that a neuron receives signals at its dendrites and cell body and transmits them, as action potentials, along the axon in one direction: away from the cell body. The Law of Dynamic Polarization has important exceptions; dendrites can serve as synaptic output sites of neurons and axons can receive synaptic inputs.</p>
<p>The number of neurons in the brain varies dramatically from species to species. In a human, there are an estimated 10–20 billion neurons in the cerebral cortex and 55–70 billion neurons in the cerebellum. By contrast, the nematode worm <em>Caenorhabditis elegans</em> has just 302 neurons, making it an ideal model organism as scientists have been able to map all of its neurons. The fruit fly <em>Drosophila melanogaster</em>, a common subject in biological experiments, has around 100,000 neurons and exhibits many complex behaviors. Many properties of neurons, from the type of neurotransmitters used to ion channel composition, are maintained across species, allowing scientists to study processes occurring in more complex organisms in much simpler experimental systems.</p>
<p>A neuron, neurone (old British spelling) or nerve cell, is an electrically excitable cell that communicates with other cells via specialized connections called synapses. It is the main component of nervous tissue. All animals except sponges and placozoans have neurons, but other multicellular organisms such as plants do not.</p>
<p>Neurons are typically classified into three types based on their function. Sensory neurons respond to stimuli such as touch, sound, or light that affect the cells of the sensory organs, and they send signals to the spinal cord or brain. Motor neurons receive signals from the brain and spinal cord to control everything from muscle contractions to glandular output. Interneurons connect neurons to other neurons within the same region of the brain or spinal cord. A group of connected neurons is called a neural circuit.</p>
<p>A typical neuron consists of a cell body (soma), dendrites, and a single axon. The soma is usually compact. The axon and dendrites are filaments that extrude from it. Dendrites typically branch profusely and extend a few hundred micrometers from the soma. The axon leaves the soma at a swelling called the axon hillock, and travels for as far as 1 meter in humans or more in other species. It branches but usually maintains a constant diameter. At the farthest tip of the axon’s branches are axon terminals, where the neuron can transmit a signal across the synapse to another cell. Neurons may lack dendrites or have no axon. The term neurite is used to describe either a dendrite or an axon, particularly when the cell is undifferentiated.</p>
<p>Most neurons receive signals via the dendrites and soma and send out signals down the axon. At the majority of synapses, signals cross from the axon of one neuron to a dendrite of another. However, synapses can connect an axon to another axon or a dendrite to another dendrite.</p>
<p>The signaling process is partly electrical and partly chemical. Neurons are electrically excitable, due to maintenance of voltage gradients across their membranes. If the voltage changes by a large enough amount over a short interval, the neuron generates an all-or-nothing electrochemical pulse called an action potential. This potential travels rapidly along the axon, and activates synaptic connections as it reaches them. Synaptic signals may be excitatory or inhibitory, increasing or reducing the net voltage that reaches the soma.</p>
<p>In most cases, neurons are generated by neural stem cells during brain development and childhood. Neurogenesis largely ceases during adulthood in most areas of the brain. However, strong evidence supports generation of substantial numbers of new neurons in the hippocampus and olfactory bulb.</p>
<p>Neurons are highly specialized for the processing and transmission of cellular signals. Given their diversity of functions performed in different parts of the nervous system, there is a wide variety in their shape, size, and electrochemical properties. For instance, the soma of a neuron can vary from 4 to 100 micrometers in diameter.</p>
<p>The soma is the body of the neuron. As it contains the nucleus, most protein synthesis occurs here. The nucleus can range from 3 to 18 micrometers in diameter.
The dendrites of a neuron are cellular extensions with many branches. This overall shape and structure is referred to metaphorically as a dendritic tree. This is where the majority of input to the neuron occurs via the dendritic spine.</p>
<p>The axon is a finer, cable-like projection that can extend tens, hundreds, or even tens of thousands of times the diameter of the soma in length. The axon primarily carries nerve signals away from the soma, and carries some types of information back to it. Many neurons have only one axon, but this axon may—and usually will—undergo extensive branching, enabling communication with many target cells. The part of the axon where it emerges from the soma is called the axon hillock. Besides being an anatomical structure, the axon hillock also has the greatest density of voltage-dependent sodium channels. This makes it the most easily excited part of the neuron and the spike initiation zone for the axon. In electrophysiological terms, it has the most negative threshold potential. While the axon and axon hillock are generally involved in information outflow, this region can also receive input from other neurons.</p>
<p>The axon terminal is found at the end of the axon farthest from the soma and contains synapses. Synaptic boutons are specialized structures where neurotransmitter chemicals are released to communicate with target neurons. In addition to synaptic boutons at the axon terminal, a neuron may have en passant boutons, which are located along the length of the axon.</p>
<p>The accepted view of the neuron attributes dedicated functions to its various anatomical components; however, dendrites and axons often act in ways contrary to their so-called main function.</p>
<p>Axons and dendrites in the central nervous system are typically only about one micrometer thick, while some in the peripheral nervous system are much thicker. The soma is usually about 10–25 micrometers in diameter and often is not much larger than the cell nucleus it contains. The longest axon of a human motor neuron can be over a meter long, reaching from the base of the spine to the toes.</p>
<p>Sensory neurons can have axons that run from the toes to the posterior column of the spinal cord, over 1.5 meters in adults. Giraffes have single axons several meters in length running along the entire length of their necks. Much of what is known about axonal function comes from studying the squid giant axon, an ideal experimental preparation because of its relatively immense size (0.5–1 millimeters thick, several centimeters long).</p>
<p>Fully differentiated neurons are permanently postmitotic however, stem cells present in the adult brain may regenerate functional neurons throughout the life of an organism. Astrocytes are star-shaped glial cells. They have been observed to turn into neurons by virtue of their stem cell-like characteristic of pluripotency.</p>
<p>Like all animal cells, the cell body of every neuron is enclosed by a plasma membrane, a bilayer of lipid molecules with many types of protein structures embedded in it. A lipid bilayer is a powerful electrical insulator, but in neurons, many of the protein structures embedded in the membrane are electrically active. These include ion channels that permit electrically charged ions to flow across the membrane and ion pumps that transport ions from one side of the membrane to the other. Most ion channels are permeable only to specific types of ions. Some ion channels are voltage gated, meaning that they can be switched between open and closed states by altering the voltage difference across the membrane. Others are chemically gated, meaning that they can be switched between open and closed states by interactions with chemicals that diffuse through the extracellular fluid. The ions include sodium, potassium, chloride, and calcium. The interactions between ion channels and ion pumps produce a voltage difference across the membrane, typically a bit less than 1/10 of a volt at baseline. This voltage has two functions: first, it provides a power source for an assortment of voltage-dependent protein machinery that is embedded in the membrane; second, it provides a basis for electrical signal transmission between different parts of the membrane.</p>
<p>Numerous microscopic clumps called Nissl bodies (or Nissl substance) are seen when nerve cell bodies are stained with a basophilic (“base-loving”) dye. These structures consist of rough endoplasmic reticulum and associated ribosomal RNA. Named after German psychiatrist and neuropathologist <a href="https://en.wikipedia.org/wiki/Franz_Nissl">Franz Nissl</a> (1860–1919), they are involved in protein synthesis and their prominence can be explained by the fact that nerve cells are very metabolically active. Basophilic dyes such as aniline or (weakly) haematoxylin highlight negatively charged components, and so bind to the phosphate backbone of the ribosomal RNA.</p>
<p>The cell body of a neuron is supported by a complex mesh of structural proteins called neurofilaments, which together with neurotubules (neuronal microtubules) are assembled into larger neurofibrils. Some neurons also contain pigment granules, such as neuromelanin (a brownish-black pigment that is byproduct of synthesis of catecholamines), and lipofuscin (a yellowish-brown pigment), both of which accumulate with age. Other structural proteins that are important for neuronal function are actin and the tubulin of microtubules. Class III β-tubulin is found almost exclusively in neurons. Actin is predominately found at the tips of axons and dendrites during neuronal development. There the actin dynamics can be modulated via an interplay with microtubule.</p>

<div class="figure" style="text-align: center"><span id="fig:motorneuron"></span>
<img src="figures/cells/Complete_neuron_cell_diagram_en.svg" alt="Diagram of a myelinated vertebrate motor neuron." width="70%" />
<p class="caption">
Figure 25.12: <a href="https://commons.wikimedia.org/wiki/File:Complete_neuron_cell_diagram_en.svg">Diagram of a myelinated vertebrate motor neuron.</a>
</p>
</div>
<p>There are different internal structural characteristics between axons and dendrites. Typical axons almost never contain ribosomes, except some in the initial segment. Dendrites contain granular endoplasmic reticulum or ribosomes, in diminishing amounts as the distance from the cell body increases.</p>
<p>Neurons vary in shape and size and can be classified by their morphology and function. The anatomist <a href="https://en.wikipedia.org/wiki/Camillo_Golgi">Camillo Golgi</a> grouped neurons into two types; type I with long axons used to move signals over long distances and type II with short axons, which can often be confused with dendrites. Type I cells can be further classified by the location of the soma. The basic morphology of type I neurons, represented by spinal motor neurons, consists of a cell body called the soma and a long thin axon covered by a myelin sheath.</p>

<div class="figure" style="text-align: center"><span id="fig:axonmyelinated"></span>
<img src="figures/cells/CajalMyelinAxon.jpg" alt="A Golgi type I neuron from the electric lobe in the brain of the electric ray (Torpedo). A single, thick axon emerges from the cell body unmyelinated (middle, left) but becomes surrounded by a think layer of myelin (dark grey). Multiple, thin dendrites emerge from the remainder of the cell body. Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay." width="70%" />
<p class="caption">
Figure 25.13: A Golgi type I neuron from the electric lobe in the brain of the electric ray (<em>Torpedo</em>). A single, thick axon emerges from the cell body unmyelinated (middle, left) but becomes surrounded by a think layer of myelin (dark grey). Multiple, thin dendrites emerge from the remainder of the cell body. <a href="https://wellcomelibrary.org/item/b2129592x#?c=0&amp;m=0&amp;s=0&amp;cv=14&amp;z=0%2C-3.48%2C1%2C8.6591">Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier</a> (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay.
</p>
</div>
<p>The dendritic tree wraps around the cell body and receives signals from other neurons. The end of the axon has branching terminals (axon terminal) that release neurotransmitters into a gap called the synaptic cleft between the terminals and the dendrites of the next neuron.</p>
<p>Most neurons can be anatomically characterized as:</p>
<ul>
<li>Unipolar: single process</li>
<li>Bipolar: 1 axon and 1 dendrite</li>
<li>Multipolar: 1 axon and 2 or more dendrites</li>
<li>Golgi I: neurons with projecting axonal processes; examples are pyramidal cells, Purkinje cells, and anterior horn cells</li>
<li>Golgi II: neurons whose axonal process projects locally; the best example is the granule cell</li>
<li>Anaxonic: where the axon cannot be distinguished from the dendrite(s)</li>
<li>Pseudounipolar: 1 process which then serves as both an axon and a dendrite</li>
<li>Other</li>
</ul>

<div class="figure" style="text-align: center"><span id="fig:neurontypes"></span>
<img src="figures/cells/Cajal_neurons.jpg" alt="Morpholoigcally distinct types of neurons after Cajal. A) Unipolar neurons; B) bipolar neurons; Golgi I neurons: C) a Purkinje cell; D) spinal motor neuron E) a pyramidal cell; F) Golgi II neuron. Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay." width="70%" />
<p class="caption">
Figure 25.14: Morpholoigcally distinct types of neurons after Cajal. A) Unipolar neurons; B) bipolar neurons; Golgi I neurons: C) a Purkinje cell; D) spinal motor neuron E) a pyramidal cell; F) Golgi II neuron. <a href="https://wellcomelibrary.org/item/b2129592x#?c=0&amp;m=0&amp;s=0&amp;cv=14&amp;z=0%2C-3.48%2C1%2C8.6591">Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier</a> (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay.
</p>
</div>
<p>Some unique neuronal types can be identified according to their location in the nervous system and distinct shape. Some examples are:</p>
<ul>
<li>Basket cells, interneurons that form a dense plexus of terminals around the soma of target cells, found in the cortex and cerebellum</li>
<li>Betz cells, large motor neurons</li>
<li>Lugaro cells, interneurons of the cerebellum</li>
<li>Medium spiny neurons, most neurons in the corpus striatum</li>
<li>Purkinje cells, huge neurons in the cerebellum, a type of Golgi I multipolar neuron</li>
<li>Pyramidal cells, neurons with triangular soma, a type of Golgi I</li>
<li>Renshaw cells, neurons with both ends linked to alpha motor neurons</li>
<li>Unipolar brush cells, interneurons with unique dendrite ending in a brush-like tuft</li>
<li>Granule cells, a type of Golgi II neuron</li>
<li>Anterior horn cells, motoneurons located in the spinal cord</li>
<li>Spindle cells, interneurons that connect widely separated areas of the brain</li>
</ul>
<p>Neurons can also be characterized based on various aspects of their function:</p>
<ul>
<li>Afferent neurons convey information from tissues and organs into the central nervous system and are also called sensory neurons.</li>
<li>Efferent neurons (motor neurons) transmit signals from the central nervous system to the effector cells.</li>
<li>Interneurons connect neurons within specific regions of the central nervous system.
Afferent and efferent also refer generally to neurons that, respectively, bring information to or send information from the brain.</li>
</ul>
<p>The axons of neurons in the human peripheral nervous system can be classified based on their physical features and signal conduction properties. Axons were known to have different thicknesses (from 0.1 to 20 µm) and these differences were thought to relate to the speed at which an action potential could travel along the axon – its conductance velocity. Erlanger and Gasser proved this hypothesis, and identified several types of nerve fiber, establishing a relationship between the diameter of an axon and its nerve conduction velocity. They published their findings in 1941 giving the first classification of axons.</p>
<p>Neurons have intrinsic electroresponsive properties like intrinsic transmembrane voltage oscillatory patterns. So neurons can be classified according to their electrophysiological characteristics:</p>
<ul>
<li>Tonic or regular spiking. Some neurons are typically constantly (tonically) active, typically firing at a constant frequency. Example: interneurons in neurostriatum.</li>
<li>Phasic or bursting. Neurons that fire in bursts are called phasic.</li>
<li>Fast spiking. Some neurons are notable for their high firing rates, for example some types of cortical inhibitory interneurons, cells in globus pallidus, retinal ganglion cells.</li>
</ul>
<p>Neurons communicate with each another via synapses, where either the axon terminal of one cell contacts another neuron’s dendrite, soma or, less commonly, axon. Neurons such as Purkinje cells in the cerebellum can have over 1000 dendritic branches, making connections with tens of thousands of other cells; other neurons, such as the magnocellular neurons of the supraoptic nucleus, have only one or two dendrites, each of which receives thousands of synapses.</p>
<p>Synapses can be excitatory or inhibitory, either increasing or decreasing activity in the target neuron, respectively. Some neurons also communicate via electrical synapses, which are direct, electrically conductive junctions between cells.</p>
<p>When an action potential reaches the axon terminal, it opens voltage-gated calcium channels, allowing calcium ions to enter the terminal. Calcium causes synaptic vesicles filled with neurotransmitter molecules to fuse with the membrane, releasing their contents into the synaptic cleft. The neurotransmitters diffuse across the synaptic cleft and activate receptors on the postsynaptic neuron. High cytosolic calcium in the axon terminal triggers mitochondrial calcium uptake, which, in turn, activates mitochondrial energy metabolism to produce ATP to support continuous neurotransmission.</p>
<p>An autapse is a synapse in which a neuron’s axon connects to its own dendrites.</p>
<p>The human brain has some 8.6 x 10<sup>10</sup> (eighty six billion) neurons. Each neuron has on average 7,000 synaptic connections to other neurons. It has been estimated that the brain of a three-year-old child has about 10<sup>15</sup> synapses (1 quadrillion). This number declines with age, stabilizing by adulthood. Estimates vary for an adult, ranging from 10<sup>14</sup> to 5 x 10<sup>14</sup> synapses (100 to 500 trillion).</p>
<p>The two most common neurotransmitters in the brain, glutamate and GABA, have largely consistent actions. Glutamate acts on several types of receptors, and has effects that are excitatory at ionotropic receptors and a modulatory effect at metabotropic receptors. Similarly, GABA acts on several types of receptors, but all of them have inhibitory effects (in adult animals, at least). Because of this consistency, it is common for neuroscientists to refer to cells that release glutamate as “excitatory neurons”, and cells that release GABA as “inhibitory neurons”. Some other types of neurons have consistent effects, for example, “excitatory” motor neurons in the spinal cord that release acetylcholine, and “inhibitory” spinal neurons that release glycine.</p>
<p>The distinction between excitatory and inhibitory neurotransmitters is not absolute. Rather, it depends on the class of chemical receptors present on the postsynaptic neuron. In principle, a single neuron, releasing a single neurotransmitter, can have excitatory effects on some targets, inhibitory effects on others, and modulatory effects on others still. For example, photoreceptor cells in the retina constantly release the neurotransmitter glutamate in the absence of light. So-called OFF bipolar cells are, like most neurons, excited by the released glutamate. However, neighboring target neurons called ON bipolar cells are instead inhibited by glutamate, because they lack typical ionotropic glutamate receptors and instead express a class of inhibitory metabotropic glutamate receptors. When light is present, the photoreceptors cease releasing glutamate, which relieves the ON bipolar cells from inhibition, activating them; this simultaneously removes the excitation from the OFF bipolar cells, silencing them.</p>
<p>Neurons can also be classified based on the neurotransmitter they release at synapses:</p>
<ul>
<li>Cholinergic neurons—acetylcholine. Acetylcholine is released from presynaptic neurons into the synaptic cleft. It acts as a ligand for both ligand-gated ion channels and metabotropic (GPCRs) muscarinic receptors. Nicotinic receptors are pentameric ligand-gated ion channels composed of alpha and beta subunits that bind nicotine. Ligand binding opens the channel causing influx of Na<sup>+</sup> depolarization and increases the probability of presynaptic neurotransmitter release. Acetylcholine is synthesized from choline and acetyl coenzyme A.</li>
<li>GABAergic neurons—gamma aminobutyric acid. GABA is one of two neuroinhibitors in the central nervous system (CNS), along with glycine. GABA has a homologous function to ACh, gating anion channels that allow Cl<sup>−</sup> ions to enter the post synaptic neuron. Cl<sup>−</sup> causes hyperpolarization within the neuron, decreasing the probability of an action potential firing as the voltage becomes more negative (for an action potential to fire, a positive voltage threshold must be reached). GABA is synthesized from glutamate by the enzyme glutamate decarboxylase.</li>
<li>Glutamatergic neurons—glutamate. Glutamate is one of two primary excitatory amino acid neurotransmitters, along with aspartate. Glutamate can cause excitotoxicity when blood flow to the brain is interrupted, resulting in brain damage. When blood flow is suppressed, glutamate is released from presynaptic neurons, causing abnormal NMDA and AMPA receptor activation, leading to elevated Ca<sup>2+</sup> and Na<sup>+</sup> entering the post synaptic neuron and cell damage. Glutamate is synthesized from the amino acid glutamine by the enzyme glutamate synthase. There are three main types of ionotropic glutamate receptors and three groups of metabotropic (G-protein coupled) receptors:
<ul>
<li>AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid) receptors are composed of four types of subunits encoded by different genes, designated as GRIA1 (also named GluA1 or GluR1), GRIA2 (also named GluA2 or GluR2), GRIA3 (Also named GluA3 or GluR3), and GRIA4 (also called GluA4 or GluRA-D2 ), which combine to form tetramers. Most AMPARs are heterotetrameric, consisting of symmetric ‘dimer of dimers’ of GluA2 and either GluA1, GluA3 or GluA4. Each AMPAR has four sites to which an agonist (such as glutamate) can bind, one for each subunit. The binding site is believed to be formed by the N-terminal tail and the extracellular loop between transmembrane domains three and four. When an agonist binds, these two loops move towards each other, opening the pore. The channel opens when two sites are occupied, and increases its current as more binding sites are occupied. Once open, the channel may undergo rapid desensitization, stopping the current. The mechanism of desensitization is believed to be due to a small change in angle of one of the parts of the binding site, closing the pore. AMPARs open and close quickly (1ms), and are thus responsible for most of the fast excitatory synaptic transmission in the central nervous system. The AMPAR’s permeability to calcium and other cations, such as sodium and potassium, is governed by the GluA2 subunit. If an AMPAR lacks a GluA2 subunit, then it will be permeable to sodium, potassium, and calcium. The presence of a GluA2 subunit will almost always render the channel impermeable to calcium. This is determined by post-transcriptional modification — RNA editing — of the Q-to-R editing site of the GluA2 mRNA. Here, A→I editing alters the uncharged amino acid glutamine (Q) to the positively charged arginine (R) in the receptor’s ion channel. The positively charged amino acid at the critical point makes it energetically unfavourable for calcium to enter the cell through the pore. Almost all of the GluA2 subunits in CNS are edited to the GluA2(R) form. This means that the principal ions gated by AMPARs are sodium and potassium, distinguishing AMPARs from NMDA receptors (the other main ionotropic glutamate receptors in the brain), which also permit calcium influx.</li>
<li>Kainate receptors were first identified as a distinct glutamate receptor type through their selective activation by the agonist kainate, a drug first isolated from the red alga <em>Digenea simplex</em>. There are five types of kainate receptor subunits, GluR5 (GRIK1), GluR6 (GRIK2), GluR7 (GRIK3), KA1 (GRIK4) and KA2 (GRIK5), which are similar to AMPA and NMDA receptor subunits and can be arranged in different ways to form a tetramer, a four subunit receptor. GluR5-7 can form homomers (ex. a receptor composed entirely of GluR5) and heteromers (ex. a receptor composed of both GluR5 and GluR6), however, KA1 and KA2 can only form functional receptors by combining with one of the GluR5-7 subunits. Since 2009 the kainate receptor subunits have been renamed to correspond with their gene name. Hence GluR5-7 are now GluK1-3 and KA1 and KA2 are GluK4 and GluK5 respectively.The ion channel formed by kainate receptors is permeable to sodium and potassium ions. The single channel conductance of kainate receptor channels is similar to that of AMPA channels, at about 20 pS. However, rise and decay times for postsynaptic potentials generated by KARs are slower than for AMPA postsynaptic potentials. Their permeability to Ca2+ is usually very slight but varies with subunits and RNA editing at the tip of the p loop.</li>
<li>NMDA receptors are activated when glycine and glutamate bind to it. The receptor is a heteromeric complex that interacts with multiple intracellular proteins by three different subunits: GluN1, GluN2 and GluN3. GluN1 has eight different isoforms due to alternative splicing of the gene GRIN1. There are four different GluN2 subunits (A-D) and two different Glun3 subunits (A and B). Six separate genes encode for GluN2 and GluN3. All the subunits share a common membrane topology that is dominated by a large extracellular N-terminus, a membrane region comprising three transmembrane segments, a re-entrant pore loop, an extracellular loop between the transmembrane segments that are structurally not well known, and an intracellular C-terminus, which are different in size depending on the subunit and provide multiple sites of interaction with many intracellular proteins.Mg2+ blocks the NMDA receptor channel in a voltage-dependent manner. The channels are also highly permeable to Ca2+. Activation of the receptor depends on glutamate binding, D-serine or glycine binding at its GluN1-linked binding site and AMPA receptor-mediated depolarization of the postsynaptic membrane, which relieves the voltage-dependent channel block by Mg2+. Activation and opening of the receptor channel thus allows the flow of K+, Na+ and Ca2+ ions, and the influx of Ca2+ triggers intracellular signaling pathways. Allosteric receptor binding sites for zinc, proteins and the polyamines spermidine and spermine are also modulators for the NMDA receptor channels.The activity of the NMDA receptor is affected by many psychoactive drugs such as phencyclidine (PCP), alcohol (ethanol) and dextromethorphan (DXM).</li>
<li>Metabotropic glutamate receptors are members of the group C family of G-protein-coupled receptors (GPCRs).
Eight different types of mGluRs, labeled mGluR1 to mGluR8 (GRM1 to GRM8), are divided into groups I, II, and III. Receptor types are grouped based on receptor structure and physiological activity. The mGluRs are further divided into subtypes, such as mGluR7a and mGluR7b.The mGluRs perform a variety of functions in the central and peripheral nervous systems: For example, they are involved in learning, memory, anxiety, and the perception of pain. They are found in pre- and postsynaptic neurons in synapses of retina, the hippocampus, cerebellum, and the cerebral cortex, as well as other parts of the brain and in peripheral tissues.</li>
</ul></li>
<li>Dopaminergic neurons—dopamine. Dopamine is a neurotransmitter that acts on D1 type (D1 and D5) Gs-coupled receptors, which stimulate the production of cAMP which stimulates protein kinase A (PKA), and D2 type (D2, D3, and D4) receptors, which activate Gi-coupled receptors that decrease cAMP and PKA. Dopamine is connected to mood and behavior and modulates both pre- and post-synaptic neurotransmission. Loss of dopamine neurons in the substantia nigra has been linked to Parkinson’s disease. Dopamine is synthesized from the amino acid tyrosine. Tyrosine is converted into levadopa (or L-DOPA) by tyrosine hydroxlase, and levadopa is then converted into dopamine by the aromatic amino acid decarboxylase.</li>
<li>Serotonergic neurons—serotonin. Serotonin (5-hydroxytryptamine, 5-HT) can act as excitatory or inhibitory. Of its four 5-HT receptor classes, 3 are GPCR and 1 is a ligand-gated cation channel. Serotonin is synthesized from tryptophan by tryptophan hydroxylase, and then further by decarboxylase. A lack of 5-HT at postsynaptic neurons has been linked to depression. Drugs that block the presynaptic serotonin transporter are used for treatment, such as Prozac and Zoloft.</li>
</ul>
</div>
<div id="glia" class="section level3">
<h3><span class="header-section-number">25.10.2</span> Glia</h3>
<p>Glia, also called glial cells or neuroglia, are non-neuronal cells in the central nervous system (brain and spinal cord) and the peripheral nervous system that do not produce electrical impulses. They maintain homeostasis, form myelin, and provide support and protection for neurons. In the central nervous system, glial cells include oligodendrocytes, astrocytes, ependymal cells, and microglia, and in the peripheral nervous system glial cells include Schwann cells and satellite cells. They have four main functions: (1) to surround neurons and hold them in place; (2) to supply nutrients and oxygen to neurons; (3) to insulate one neuron from another; (4) to destroy pathogens and remove dead neurons.</p>
<p>Glial cells exhibit great cellular and functional diversity. Glial cells can respond to and manipulate neurotransmission in many ways.</p>
<p>Glia were discovered in 1856, by the pathologist Rudolf Virchow in his search for a “connective tissue” in the brain. The term derives from Greek γλία and γλοία “glue”, and suggests the original impression that they were the glue of the nervous system.</p>
<p>In general, neuroglial cells are smaller than neurons. There are approximately 85 billion glia cells in the human brain, about the same number as neurons. Glial cells make up about half the total volume of the brain and spinal cord. The glia to neuron-ratio varies from one part of the brain to another. The glia to neuron-ratio in the cerebral cortex is 3.72 (60.84 billion glia (72%); 16.34 billion neurons), while that of the cerebellum is only 0.23 (16.04 billion glia; 69.03 billion neurons). The ratio in the cerebral cortex gray matter is 1.48, with 3.76 for the gray and white matter combined. The ratio of the basal ganglia, diencephalon and brainstem combined is 11.35.</p>
<p>The total number of glia cells in the human brain is distributed into the different types with oligodendrocytes being the most frequent (45–75%), followed by astrocytes (19–40%) and microglia (about 10% or less).</p>
<p>Most glia are derived from ectodermal tissue of the developing embryo, in particular the neural tube and crest. The exception is microglia, which are derived from hemopoietic stem cells. In the adult, microglia are largely a self-renewing population and are distinct from macrophages and monocytes, which infiltrate an injured and diseased CNS.</p>
<p>In the central nervous system, glia develop from the ventricular zone of the neural tube. These glia include the oligodendrocytes, ependymal cells, and astrocytes. In the peripheral nervous system, glia derive from the neural crest. These PNS glia include Schwann cells in nerves and satellite glial cells in ganglia.</p>
<p>Glia retain the ability to undergo cell division in adulthood, whereas most neurons cannot. The view is based on the general inability of the mature nervous system to replace neurons after an injury, such as a stroke or trauma, where very often there is a substantial proliferation of glia, or gliosis, near or at the site of damage. However, detailed studies have found no evidence that ‘mature’ glia, such as astrocytes or oligodendrocytes, retain mitotic capacity. Only the resident oligodendrocyte precursor cells seem to keep this ability once the nervous system matures.</p>
<p>Some glial cells function primarily as the physical support for neurons. Others provide nutrients to neurons and regulate the extracellular fluid of the brain, especially surrounding neurons and their synapses. During early embryogenesis, glial cells direct the migration of neurons and produce molecules that modify the growth of axons and dendrites.</p>
<p>Glia are crucial in the development of the nervous system and in processes such as synaptic plasticity and synaptogenesis. Glia have a role in the regulation of repair of neurons after injury. In the central nervous system (CNS), glia suppress repair. Glial cells known as astrocytes enlarge and proliferate to form a scar and produce inhibitory molecules that inhibit regrowth of a damaged or severed axon. In the peripheral nervous system (PNS), glial cells known as Schwann cells promote repair. After axonal injury, Schwann cells regress to an earlier developmental state to encourage regrowth of the axon. This difference between the CNS and the PNS, raises hopes for the regeneration of nervous tissue in the CNS. For example, a spinal cord may be able to be repaired following injury or severance. Schwann cells are also known as neuri-lemmocytes. These cells envelop nerve fibers of the PNS by winding repeatedly around a nerve fiber with the nucleus inside of it. This process creates a myelin sheath, which not only aids in conductivity but also assists in the regeneration of damaged fibers.</p>
<p>Astrocytes are crucial participants in the tripartite synapse. They have several crucial functions, including clearance of neurotransmitters from within the synaptic cleft, which aids in distinguishing between separate action potentials and prevents toxic build-up of certain neurotransmitters such as glutamate, which would otherwise lead to excitotoxicity. Furthermore, astrocytes release gliotransmitters such as glutamate, ATP, and D-serine in response to stimulation.</p>

<div class="figure" style="text-align: center"><span id="fig:astrocytes"></span>
<img src="figures/cells/Cajal_glia.jpg" alt="Astrocytes (A) and oligodendrocytes (B) are the major types of macroglia in the grey and white matter of the brain, respectively. Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay." width="70%" />
<p class="caption">
Figure 25.15: Astrocytes (A) and oligodendrocytes (B) are the major types of macroglia in the grey and white matter of the brain, respectively. <a href="https://wellcomelibrary.org/item/b2129592x#?c=0&amp;m=0&amp;s=0&amp;cv=14&amp;z=0%2C-3.48%2C1%2C8.6591">Histologie du système nerveux de l’homme &amp; des vertébrés, Tome Premier</a> (1909) by Santiago Ramón y Cajal translated from Spanish by Dr. L. Azoulay.
</p>
</div>
<p>Oligodendrocytes are found in the CNS and resemble an octopus: they have bulbous cell bodies with up to fifteen arm-like processes. Each “arm” reaches out to a nerve fiber and spirals around it, creating a myelin sheath. The myelin sheath insulates the nerve fiber from the extracellular fluid and speeds up signal conduction along the nerve fiber.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:macroglia">Table 25.1: </span>Types of macroglia and their location and function.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Location
</th>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 5em; vertical-align: middle !important;" rowspan="7">
CNS
</td>
<td style="text-align:left;width: 10em; vertical-align: middle !important;" rowspan="4">
Astrocytes
</td>
<td style="text-align:left;width: 20em; ">
The most abundant type of macroglial cell in the CNS, astrocytes (also called astroglia) have numerous projections that link neurons to their blood supply while forming the blood-brain barrier. They regulate the external chemicalenvironment of neurons by removing excess potassium ions, and recycling neurotransmitters released during synaptic transmission. Astrocytes may regulate vasoconstriction and vasodilation by producing substances such as arachidonic acid, whose metabolites are vasoactive.
</td>
</tr>
<tr>
<td style="text-align:left;width: 20em; ">
Astrocytes signal each other using ATP. The gap junctions (also known as electrical synapses) between astrocytes allow the messenger molecule IP3 to diffuse from one astrocyte to another. IP3 activates calcium channels on cellular organelles, releasing calcium into the cytoplasm. This calcium may stimulate the production of more IP3 and cause release of ATP through channels in the membrane made of pannexins. The net effect is a calcium wave that propagates from cell to cell. Extracellular release of ATP, and consequent activation of purinergic receptors on other astrocytes, may also mediate calcium waves in some cases.
</td>
</tr>
<tr>
<td style="text-align:left;width: 20em; ">
In general, there are two types of astrocytes, protoplasmic and fibrous, similar in function but distinct in morphology and distribution. Protoplasmic astrocytes have short, thick, highly branched processes and are typically found in gray matter. Fibrous astrocytes have long, thin, less branched processes and are more commonly found in white matter.
</td>
</tr>
<tr>
<td style="text-align:left;width: 20em; ">
It has recently been shown that astrocyte activity is linked to blood flow in the brain, and that this is what is actually being measured in fMRI. They also have been involved in neuronal circuits playing an inhibitory role after sensing changes in extracellular calcium.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Oligodendrocytes
</td>
<td style="text-align:left;width: 20em; ">
Oligodendrocytes are cells that coat axons in the central nervous system (CNS) with their cell membrane, forming a specialized membrane differentiation called myelin, producing the myelin sheath. The myelin sheath provides insulation to the axon that allows electrical signals to propagate more efficiently.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Ependymal cells
</td>
<td style="text-align:left;width: 20em; ">
Ependymal cells, also named ependymocytes, line the spinal cord and the ventricular system of the brain. These cells are involved in the creation and secretion of cerebrospinal fluid (CSF) and beat their cilia to help circulate the CSF and make up the blood-CSF barrier. They are also thought to act as neural stem cells.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Radial glia
</td>
<td style="text-align:left;width: 20em; ">
Radial glia cells arise from neuroepithelial cells after the onset of neurogenesis. Their differentiation abilities are more restricted than those of neuroepithelial cells. In the developing nervous system, radial glia function both as neuronal progenitors and as a scaffold upon which newborn neurons migrate. In the mature brain, the cerebellum and retina retain characteristic radial glial cells. In the cerebellum, these are Bergmann glia, which regulate synaptic plasticity. In the retina, the radial Müller cell is the glial cell that spans the thickness of the retina and, in addition to astroglial cells,[14] participates in a bidirectional communication with neurons.
</td>
</tr>
<tr>
<td style="text-align:left;width: 5em; vertical-align: middle !important;" rowspan="3">
PNS
</td>
<td style="text-align:left;width: 10em; ">
Schwann cells
</td>
<td style="text-align:left;width: 20em; ">
Similar in function to oligodendrocytes, Schwann cells provide myelination to axons in the peripheral nervous system(PNS). They also have phagocytotic activity and clear cellular debris that allows for regrowth of PNS neurons.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Satellite cells
</td>
<td style="text-align:left;width: 20em; ">
Satellite glial cells are small cells that surround neurons in sensory, sympathetic, and parasympathetic ganglia.[17]These cells help regulate the external chemical environment. Like astrocytes, they are interconnected by gap junctions and respond to ATP by elevating intracellular concentration of calcium ions. They are highly sensitive to injury and inflammation, and appear to contribute to pathological states, such as chronic pain.
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; ">
Enteric glial cells
</td>
<td style="text-align:left;width: 20em; ">
Are found in the intrinsic ganglia of the digestive system. They are thought to have many roles in the enteric system, some related to homeostasis and muscular digestive processes.
</td>
</tr>
</tbody>
</table>
<p>Microglia are specialized macrophages capable of phagocytosis that protect neurons of the central nervous system. They are derived from the earliest wave of mononuclear cells that originate in yolk sac blood islands early in development, and colonize the brain shortly after the neural precursors begin to differentiate.</p>
<p>These cells are found in all regions of the brain and spinal cord. Microglial cells are small relative to macroglial cells, with changing shapes and oblong nuclei. They are mobile within the brain and multiply when the brain is damaged. In the healthy central nervous system, microglia processes constantly sample all aspects of their environment (neurons, macroglia and blood vessels). In a healthy brain, microglia direct the immune response to brain damage and play an important role in the inflammation that accompanies the damage. Many diseases and disorders are associated with deficient microglia, such as Alzheimer’s disease, Parkinson’s disease, and ALS.During developmental wiring of the brain, microglial cells play a large role regulating numbers of neural precursor cells and removing apoptotic neurons. There is also evidence that microglia can refine synaptic circuitry by engulfing and eliminating synapses. Post development, the majority of dead or apoptotic cells are found in the cerebral cortex and the subcortical white matter. This may explain why the majority of ameboid microglial cells are found within the “fountains of microglia” in the cerebral cortex.</p>
</div>
</div>
<div id="electrical-basis-of-neuronal-function" class="section level2">
<h2><span class="header-section-number">25.11</span> Electrical Basis Of Neuronal Function</h2>
<div id="the-membrane-potential" class="section level3">
<h3><span class="header-section-number">25.11.1</span> The Membrane Potential</h3>
<p>Membrane potential (also transmembrane potential or membrane voltage) is the difference in electric potential between the interior and the exterior of a biological cell. With respect to the exterior of the cell, typical values of membrane potential, normally given in units of millivolts and denoted as mV, ranges from –40 mV to –80 mV.</p>
<p>All animal cells are surrounded by a membrane composed of a lipid bilayer with proteins embedded in it. The membrane serves as both an insulator and a diffusion barrier to the movement of ions. Transmembrane proteins, also known as ion transporter or ion pump proteins, actively push ions across the membrane and establish concentration gradients across the membrane, and ion channels allow ions to move across the membrane down those concentration gradients. Ion pumps and ion channels are electrically equivalent to a set of batteries and resistors inserted in the membrane, and therefore create a voltage between the two sides of the membrane.</p>
<p>Almost all plasma membranes have an electrical potential across them, with the inside usually negative with respect to the outside. The membrane potential has two basic functions. First, it allows a cell to function as a battery, providing power to operate a variety of “molecular devices” embedded in the membrane. Second, in electrically excitable cells such as neurons and muscle cells, it is used for transmitting signals between different parts of a cell. Signals are generated by opening or closing of ion channels at one point in the membrane, producing a local change in the membrane potential. This change in the electric field can quickly affect adjacent and more distant ion channels in the membrane. Those ion channels can then open or close as a result of the potential change, reproducing the signal.</p>
<p>In non-excitable cells, and in excitable cells in their baseline states, the membrane potential is held at a relatively stable value, called the resting potential. For neurons, typical values of the resting potential range from –70 to –80 millivolts; that is, the interior of a cell has a negative baseline voltage of a bit less than one-tenth of a volt. The opening and closing of ion channels can induce a departure from the resting potential. This is called a depolarization if the interior voltage becomes less negative (say from –70 mV to –60 mV), or a hyperpolarization if the interior voltage becomes more negative (say from –70 mV to –80 mV). In excitable cells, a sufficiently large depolarization can evoke an action potential, in which the membrane potential changes rapidly and significantly for a short time (on the order of 1 to 100 milliseconds), often reversing its polarity. Action potentials are generated by the activation of certain voltage-gated ion channels.</p>
<p>In neurons, the factors that influence the membrane potential are diverse. They include numerous types of ion channels, some of which are chemically gated and some of which are voltage-gated. Because voltage-gated ion channels are controlled by the membrane potential, while the membrane potential itself is influenced by these same ion channels, feedback loops that allow for complex temporal dynamics arise, including oscillations and regenerative events such as action potentials.</p>
<p>The membrane potential in a cell derives ultimately from two factors: electrical force and diffusion. Electrical force arises from the mutual attraction between particles with opposite electrical charges (positive and negative) and the mutual repulsion between particles with the same type of charge (both positive or both negative). Diffusion arises from the statistical tendency of particles to redistribute from regions where they are highly concentrated to regions where the concentration is low.</p>
<p>Voltage, which is synonymous with difference in electrical potential, is the ability to drive an electric current across a resistance. Indeed, the simplest definition of a voltage is given by Ohm’s law: V=IR, where V is voltage, I is current and R is resistance. If a voltage source such as a battery is placed in an electrical circuit, the higher the voltage of the source the greater the amount of current that it will drive across the available resistance. The functional significance of voltage lies only in potential differences between two points in a circuit. The idea of a voltage at a single point is meaningless. It is conventional in electronics to assign a voltage of zero to some arbitrarily chosen element of the circuit, and then assign voltages for other elements measured relative to that zero point. There is no significance in which element is chosen as the zero point—the function of a circuit depends only on the differences not on voltages per se. However, in most cases and by convention, the zero level is most often assigned to the portion of a circuit that is in contact with ground.</p>
<p>The same principle applies to voltage in cell biology. In electrically active tissue, the potential difference between any two points can be measured by inserting an electrode at each point, for example one inside and one outside the cell, and connecting both electrodes to the leads of what is in essence a specialized voltmeter. By convention, the zero potential value is assigned to the outside of the cell and the sign of the potential difference between the outside and the inside is determined by the potential of the inside relative to the outside zero.</p>
</div>
</div>
<div id="ions-and-the-forces-driving-their-motion" class="section level2">
<h2><span class="header-section-number">25.12</span> Ions And The Forces Driving Their Motion</h2>
<p>Electrical signals within biological organisms are, in general, driven by ions. An ion is an atom or molecule that has a net electrical charge. Since the charge of the electron (considered negative by convention) is equal and opposite to that of the proton (considered positive by convention), the net charge of an ion is non-zero due to its total number of electrons being unequal to its total number of protons. A cation is a positively charged ion, with fewer electrons than protons, while an anion is negatively charged, with more electrons than protons. Because of their opposite electric charges, cations and anions attract each other and readily form ionic compounds.</p>
<p>The word ion comes from the Greek word ἰόν, ion, “going”, the present participle of ἰέναι, ienai, “to go”. This term was introduced (after a suggestion by <a href="https://en.wikipedia.org/wiki/William_Whewell">William Whewell</a>) by English physicist and chemist <a href="https://en.wikipedia.org/wiki/Michael_Faraday">Michael Faraday</a> in 1834 for the then-unknown species that goes from one electrode to the other through an aqueous medium. Faraday did not know the nature of these species, but he knew that since metals dissolved into and entered a solution at one electrode and new metal came forth from a solution at the other electrode; that some kind of substance has moved through the solution in a current. This conveys matter from one place to the other. In correspondence with Faraday, Whewell also coined the words anode and cathode, as well as anion (from the Greek word ἄνω (ánō), meaning “up)”and cation (from the Greek word κάτω (káto), meaning “down”) as ions that are attracted to the respective electrodes.</p>
<p><a href="https://en.wikipedia.org/wiki/Svante_Arrhenius">Svante Arrhenius</a> put forth, in his 1884 dissertation, his explanation of the fact that solid crystalline salts dissociate into paired charged particles when dissolved, for which he would win the 1903 Nobel Prize in Chemistry. Arrhenius’ explanation was that in forming a solution, the salt dissociates into Faraday’s ions. Arrhenius proposed that ions formed even in the absence of an electric current.</p>
<p>The most important cations for the action potential are sodium (Na<sup>+</sup>) and potassium (K<sup>+</sup>). Both of these are monovalent cations that carry a single positive charge. Action potentials can also involve calcium (Ca<sup>2+</sup>), which is a divalent cation that carries a double positive charge. The chloride anion (Cl<sup>−</sup>) plays a major role in the action potentials of some algae, but plays a negligible role in the action potentials of most animals.</p>

<div class="figure" style="text-align: center"><span id="fig:ionsize"></span>
<img src="figures/potential/Atomic_and_ionic_radii.svg" alt=": Radii of some elements as neutral atoms, and cations (red) and anions (blue) derived from those atoms. Radii are given in picometers." width="70%" />
<p class="caption">
Figure 25.16: : <a href="https://commons.wikimedia.org/wiki/File:Atomic_%26_ionic_radii.svg">Radii of some elements as neutral atoms, and cations (red) and anions (blue) derived from those atoms. Radii are given in picometers.</a>
</p>
</div>
<p>Ions cross the cell membrane under two influences: diffusion and electric fields. A simple example wherein two solutions—A and B—are separated by a porous barrier illustrates that diffusion will ensure that they will eventually mix into equal solutions. This mixing occurs because of the difference in their concentrations. The region with high concentration will diffuse out toward the region with low concentration. To extend the example, let solution A have 30 sodium ions and 30 chloride ions. Also, let solution B have only 20 sodium ions and 20 chloride ions. Assuming the barrier allows both types of ions to travel through it, then a steady state will be reached whereby both solutions have 25 sodium ions and 25 chloride ions. If, however, the porous barrier is selective to which ions are let through, then diffusion alone will not determine the resulting solution. Returning to the previous example, let’s now construct a barrier that is permeable only to sodium ions. Now, only sodium is allowed to diffuse cross the barrier from its higher concentration in solution A to the lower concentration in solution B. This will result in a greater accumulation of sodium ions than chloride ions in solution B and a lesser number of sodium ions than chloride ions in solution A.</p>
<p>This means that there is a net positive charge in solution B from the higher concentration of positively charged sodium ions than negatively charged chloride ions. Likewise, there is a net negative charge in solution A from the greater concentration of negative chloride ions than positive sodium ions. Since opposite charges attract and like charges repel, the ions are now also influenced by electrical fields as well as forces of diffusion. Therefore, positive sodium ions will be less likely to travel to the now-more-positive B solution and remain in the now-more-negative A solution. The point at which the forces of the electric fields completely counteract the force due to diffusion is called the equilibrium potential. At this point, the net flow of the specific ion (in this case sodium) is zero.</p>
</div>
<div id="the-plasma-membrane" class="section level2">
<h2><span class="header-section-number">25.13</span> The Plasma Membrane</h2>
<p>The cell membrane, also called the plasma membrane or plasmalemma, is a semipermeable lipid bilayer common to all living cells. It contains a variety of biological molecules, primarily proteins and lipids, which are involved in a vast array of cellular processes.</p>

<div class="figure" style="text-align: center"><span id="fig:cellmembrane"></span>
<img src="figures/potential/cell_membrane_rhodopsin_water_Cl_K.png" alt="Picture of a molecular dynamics simulation of a cell membrane/protein complex consisting of bovine rhodopsin incorporated of a phosphatidylcholine (1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine, POPC) lipid bylayer. POPC and water molecules are depicted as sticks. The lipid layers facing the extracellular and cytoplasmic spaces are shown in white and blue, respectively. Both the extra- and intracellular interfaces are covered with layers of water. The secondary structure of rhodopsin is depicted in rainbow colored cartoon representation. Potassium and chloride ions are shown as spheres (colored in cyan and green, respectively). Image generated from PDB file obtained from the CHARMM-GUI Archive - Protein/Membrane Complex Library using the open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 25.17: Picture of a <a href="https://doi.org/10.1371/journal.pone.0000880">molecular dynamics simulation</a> of a cell membrane/protein complex consisting of bovine <a href="https://en.wikipedia.org/wiki/Rhodopsin">rhodopsin</a> incorporated of a phosphatidylcholine (1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine, POPC) lipid bylayer. POPC and water molecules are depicted as sticks. The lipid layers facing the extracellular and cytoplasmic spaces are shown in white and blue, respectively. Both the extra- and intracellular interfaces are covered with layers of water. The secondary structure of rhodopsin is depicted in rainbow colored cartoon representation. Potassium and chloride ions are shown as spheres (colored in cyan and green, respectively). Image generated from <a href="http://www.charmm-gui.org/archive/complex/1gzm_rect_popc.pdb">PDB file</a> obtained from the <a href="http://www.charmm-gui.org/?doc=archive&amp;lib=complex">CHARMM-GUI Archive - Protein/Membrane Complex Library</a> using the open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p>Every animal cell is enclosed in a plasma membrane, which has the structure of a lipid bilayer with many types of large molecules embedded in it. Because it is made of lipid molecules, the plasma membrane intrinsically has a high electrical resistivity, in other words a low intrinsic permeability to ions. However, some of the molecules embedded in the membrane are capable either of actively transporting ions from one side of the membrane to the other or of providing channels through which they can move.</p>
<p>In electrical terminology, the plasma membrane functions as a combined resistor and capacitor. Resistance arises from the fact that the membrane impedes the movement of charges across it. Capacitance arises from the fact that the lipid bilayer is so thin that an accumulation of charged particles on one side gives rise to an electrical force that pulls oppositely charged particles toward the other side. The capacitance of the membrane is relatively unaffected by the molecules that are embedded in it, so it has a more or less invariant value estimated at about 2 μF/cm<sup>2</sup> (the total capacitance of a patch of membrane is proportional to its area). The conductance of a pure lipid bilayer is so low, on the other hand, that in biological situations it is always dominated by the conductance of alternative pathways provided by embedded molecules. Thus, the capacitance of the membrane is more or less fixed, but the resistance is highly variable.</p>
<p>The thickness of a plasma membrane is estimated to be about 7-8 nanometers. Because the membrane is so thin, it does not take a very large transmembrane voltage to create a strong electric field within it. Typical membrane potentials in animal cells are on the order of 100 millivolts (that is, one tenth of a volt), but calculations show that this generates an electric field close to the maximum that the membrane can sustain—it has been calculated that a voltage difference much larger than 200 millivolts could cause dielectric breakdown, that is, arcing across the membrane.</p>
<p>The resistance of a pure lipid bilayer to the passage of ions across it is very high, but structures embedded in the membrane can greatly enhance ion movement, either actively or passively, via mechanisms called facilitated transport and facilitated diffusion. The two types of structure that play the largest roles are ion channels and ion pumps, both usually formed from assemblages of protein molecules. Ion channels provide passageways through which ions can move. In most cases, an ion channel is permeable only to specific types of ions (for example, sodium and potassium but not chloride or calcium), and sometimes the permeability varies depending on the direction of ion movement. Ion pumps, also known as ion transporters or carrier proteins, actively transport specific types of ions from one side of the membrane to the other, using energy derived from metabolic processes to do so.</p>
</div>
<div id="ion-pumps" class="section level2">
<h2><span class="header-section-number">25.14</span> Ion Pumps</h2>
<p>Ion pumps are integral membrane proteins that carry out active transport, i.e., use cellular energy (ATP) to “pump” the ions against their concentration gradient. Such ion pumps take in ions from one side of the membrane (decreasing its concentration there) and release them on the other side (increasing its concentration there).</p>
<p>The ion pump most relevant to the action potential is the sodium–potassium pump, which transports three sodium ions out of the cell and two potassium ions in. As a consequence, the concentration of potassium ions K<sup>+</sup> inside the neuron is roughly 20-fold larger than the outside concentration, whereas the sodium concentration outside is roughly ninefold larger than inside. In a similar manner, other ions have different concentrations inside and outside the neuron, such as calcium, chloride and magnesium.</p>

<div class="figure" style="text-align: center"><span id="fig:nakpump"></span>
<img src="figures/potential/Na_K_pump.png" alt="A cartoon representation of the molecular structure of the sodium - potassium pump based on atomic coordinates of PDB 2ZXE, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 25.18: A cartoon representation of the molecular structure of the sodium - potassium pump based on atomic coordinates of <a href="https://www.rcsb.org/structure/2ZXE">PDB 2ZXE</a>, rendered with open source molecular visualization tool PyMol.
</p>
</div>
<p>If the numbers of each type of ion were equal, the sodium–potassium pump would be electrically neutral, but, because of the three-for-two exchange, it gives a net movement of one positive charge from intracellular to extracellular for each cycle, thereby contributing to a positive voltage difference. The pump has three effects: (1) it makes the sodium concentration high in the extracellular space and low in the intracellular space; (2) it makes the potassium concentration high in the intracellular space and low in the extracellular space; (3) it gives the intracellular space a negative voltage with respect to the extracellular space.</p>
<p>The sodium-potassium pump is relatively slow in operation. If a cell were initialized with equal concentrations of sodium and potassium everywhere, it would take hours for the pump to establish equilibrium. The pump operates constantly, but becomes progressively less efficient as the concentrations of sodium and potassium available for pumping are reduced.</p>
<p>Ion pumps influence the action potential only by establishing the relative ratio of intracellular and extracellular ion concentrations. The action potential involves mainly the opening and closing of ion channels not ion pumps. If the ion pumps are turned off by removing their energy source, or by adding an inhibitor such as ouabain, the axon can still fire hundreds of thousands of action potentials before their amplitudes begin to decay significantly. In particular, ion pumps play no significant role in the repolarization of the membrane after an action potential.</p>
<p>Another functionally important ion pump is the sodium-calcium exchanger. This pump operates in a conceptually similar way to the sodium-potassium pump, except that in each cycle it exchanges three Na<sup>+</sup> from the extracellular space for one Ca<sup>2+</sup> from the intracellular space. Because the net flow of charge is inward, this pump runs “downhill”, in effect, and therefore does not require any energy source except the membrane voltage. Its most important effect is to pump calcium outward—it also allows an inward flow of sodium, thereby counteracting the sodium-potassium pump, but, because overall sodium and potassium concentrations are much higher than calcium concentrations, this effect is relatively unimportant. The net result of the sodium-calcium exchanger is that in the resting state, intracellular calcium concentrations become very low.</p>
</div>
<div id="ion-channels" class="section level2">
<h2><span class="header-section-number">25.15</span> Ion Channels</h2>
<p>Ion channels are integral membrane proteins with a pore through which ions can travel between extracellular space and cell interior. Most channels are specific (selective) for one ion; for example, most potassium channels are characterized by 1000:1 selectivity ratio for potassium over sodium, though potassium and sodium ions have the same charge and differ only slightly in their radius. The channel pore is typically so small that ions must pass through it in single-file order. Channel pores can be either open or closed for ion passage, although a number of channels demonstrate various sub-conductance levels. When a channel is open, ions permeate through the channel pore down the transmembrane concentration gradient for that particular ion. Rate of ionic flow through the channel, i.e. single-channel current amplitude, is determined by the maximum channel conductance and electrochemical driving force for that ion, which is the difference between the instantaneous value of the membrane potential and the value of the reversal potential.</p>
<p>The fundamental properties of currents mediated by ion channels were analyzed by the British biophysicists <a href="https://en.wikipedia.org/wiki/Alan_Hodgkin">Alan Hodgkin</a> and <a href="https://en.wikipedia.org/wiki/Andrew_Huxley">Andrew Huxley</a> as part of their Nobel Prize-winning research on the action potential, published in 1952. They built on the work of other physiologists, such as <a href="https://en.wikipedia.org/wiki/Kenneth_Stewart_Cole">Kenneth Stewart Cole’s</a> research into voltage-gated membrane pores from 1941. The existence of ion channels was confirmed in the 1970s by <a href="https://en.wikipedia.org/wiki/Bernard_Katz">Bernard Katz</a> and <a href="https://en.wikipedia.org/wiki/Ricardo_Miledi">Ricardo Miledi</a> using noise analysis. It was then shown more directly with an electrical recording technique known as the “patch clamp”, which led to a Nobel Prize to <a href="https://en.wikipedia.org/wiki/Erwin_Neher">Erwin Neher</a> and <a href="https://en.wikipedia.org/wiki/Bert_Sakmann">Bert Sakmann</a>, the technique’s inventors. Hundreds if not thousands of researchers continue to pursue a more detailed understanding of how these proteins work.</p>
<p>Channels differ with respect to the ion they let pass (for example, Na<sup>+</sup>, K<sup>+</sup>, Cl<sup>−</sup>), the ways in which they may be regulated, the number of subunits of which they are composed and other aspects of structure. Channels belonging to the largest class, which includes the voltage-gated channels that underlie the nerve impulse, consists of four subunits with six transmembrane helices each. On activation, these helices move about and open the pore. Two of these six helices are separated by a loop that lines the pore and is the primary determinant of ion selectivity and conductance in this channel class and some others. The existence and mechanism for ion selectivity was first postulated in the late 1960s by <a href="https://en.wikipedia.org/wiki/Bertil_Hille">Bertil Hille</a> and <a href="https://en.wikipedia.org/wiki/Clay_Armstrong">Clay Armstrong</a>. The idea of the ionic selectivity for potassium channels was that the carbonyl oxygens of the protein backbones of the “selectivity filter” (named by Bertil Hille) could efficiently replace the water molecules that normally shield potassium ions, but that sodium ions were smaller and cannot be completely dehydrated to allow such shielding, and therefore could not pass through. This mechanism was finally confirmed when the first structure of an ion channel was elucidated. A bacterial potassium channel KcsA, consisting of just the selectivity filter, “P” loop and two transmembrane helices was used as a model to study the permeability and the selectivity of ion channels. The determination of the molecular structure of KcsA by <a href="https://en.wikipedia.org/wiki/Roderick_MacKinnon">Roderick MacKinnon</a> using X-ray crystallography won a share of the 2003 Nobel Prize in Chemistry.</p>

<div class="figure" style="text-align: center"><span id="fig:kchannel"></span>
<img src="figures/potential/Kcsa.png" alt="The structure of the potassium channel KcsA fromn Streptomyces lividans determined by X-ray crystallography.Top: side view; bottom: top view. KcsA shares sequence similarity with all known K+ channels and was the first ion channel to have its structure solved at atomic resolution. It consists of four identical subunits that together form a cone shaped structure (top) with a ion selectivity filter at its outer end. Three K+ ions (red spheres) are present in the channel, two of which are held 7.5 angstroms apart by the selectivity filter, a third K+ ion shown in the channel pore below. Data from PDB 1BL8, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 25.19: The structure of the potassium channel KcsA fromn <em>Streptomyces lividans</em> determined by X-ray crystallography.Top: side view; bottom: top view. KcsA shares sequence similarity with all known K<sup>+</sup> channels and was the first ion channel to have its structure solved at atomic resolution. It consists of four identical subunits that together form a cone shaped structure (top) with a ion selectivity filter at its outer end. Three K<sup>+</sup> ions (red spheres) are present in the channel, two of which are held 7.5 angstroms apart by the selectivity filter, a third K<sup>+</sup> ion shown in the channel pore below. Data from <a href="https://www.rcsb.org/structure/1BL8">PDB 1BL8</a>, rendered with open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p>Because of their small size and the difficulty of crystallizing integral membrane proteins for X-ray analysis, it is only very recently that scientists have been able to directly examine what channels “look like.” Most of what researchers have deduced about channel operation so far they have established through electrophysiology, biochemistry, gene sequence comparison and mutagenesisi and structural studies (X-ray crystallography and cryoelectronmicroscopy).</p>
<p>Channels can have single (e.g. members of the Chloride Intracellular Ion Channel family) or multiple transmembrane (K<sup>+</sup> channels, P2X receptors, Na<sup>+</sup> channels) domains which span the plasma membrane to form pores.</p>
<p>Ion channels can be classified by how they respond to their environment. For example, the ion channels involved in the action potential are voltage-sensitive channels; they open and close in response to the voltage across the membrane. Ligand-gated channels form another important class; these ion channels open and close in response to the binding of a ligand molecule, such as a neurotransmitter. Other ion channels open and close with mechanical forces. Still other ion channels—such as those of sensory neurons—open and close in response to other stimuli, such as light, temperature or pressure.</p>
<p>Leakage channels are the simplest type of ion channel, in that their permeability is more or less constant. The types of leakage channels that have the greatest significance in neurons are potassium and chloride channels. Even these are not perfectly constant in their properties: First, most of them are voltage-dependent in the sense that they conduct better in one direction than the other (in other words, they are rectifiers); second, some of them are capable of being shut off by chemical ligands even though they do not require ligands in order to operate.</p>
<p><a href="https://en.wikipedia.org/wiki/Voltage-gated_ion_channel">Voltage-gated ion channels</a> are a class of transmembrane proteins that form ion channels that are activated by changes in the electrical membrane potential near the channel. The membrane potential alters the conformation of the channel proteins, regulating their opening and closing. Cell membranes are generally impermeable to ions, thus they must diffuse through the membrane through transmembrane protein channels. They have a crucial role in excitable cells such as neuronal and muscle tissues, allowing a rapid and co-ordinated depolarization in response to triggering voltage change. Found along the axon and at the synapse, voltage-gated ion channels directionally propagate electrical signals. Voltage-gated ion-channels are usually ion-specific, and channels specific to sodium (Na<sup>+</sup>), potassium (K<sup>+</sup>), calcium (Ca<sup>2+</sup>), and chloride (Cl−) ions have been identified. The opening and closing of the channels are triggered by changing ion concentration, and hence charge gradient, between the sides of the cell membrane.</p>
<p>Voltage-gated ion channels are generally composed of several subunits arranged in such a way that there is a central pore through which ions can travel down their electrochemical gradients. The channels tend to be ion-specific, although similarly sized and charged ions may sometimes travel through them. The functionality of voltage-gated ion channels is attributed to its three main discrete units: the voltage sensor, the pore or conducting pathway, and the gate. Na<sup>+</sup>, K<sup>+</sup>, and Ca<sup>2+</sup> channels are composed of four transmembrane domains arranged around a central pore; these four domains are part of a single α-subunit in the case of most Na<sup>+</sup> and Ca<sup>2+</sup> channels, whereas there are four α-subunits, each contributing one transmembrane domain, in most K<sup>+</sup> channels. The membrane-spanning segments, designated S1-S6, all take the form of alpha helices with specialized functions. The fifth and sixth transmembrane segments (S5 and S6) and pore loop serve the principal role of ion conduction, comprising the gate and pore of the channel, while S1-S4 serve as the voltage-sensing region. The four subunits may be identical, or different from one another. In addition to the four central α-subunits, there are also regulatory β-subunits, with oxidoreductase activity, which are located on the inner surface of the cell membrane and do not cross the membrane, and which are coassembled with the α-subunits in the endoplasmic reticulum.</p>
<p>Crystallographic structural studies of a potassium channel have shown that, when a potential difference is introduced over the membrane, the associated electric field induces a conformational change in the potassium channel. The conformational change distorts the shape of the channel proteins sufficiently such that the cavity, or channel, opens to allow influx or efflux to occur across the membrane. This movement of ions down their concentration gradients subsequently generates an electric current sufficient to depolarize the cell membrane.</p>
<p>Voltage-gated sodium channels and calcium channels are made up of a single polypeptide with four homologous domains. Each domain contains 6 membrane spanning alpha helices. One of these helices, S4, is the voltage sensing helix. The S4 segment contains many positive charges such that a high positive charge outside the cell repels the helix, keeping the channel in its closed state.</p>
<p>In general, the voltage sensing portion of the ion channel is responsible for the detection of changes in transmembrane potential that trigger the opening or closing of the channel. The S1-4 alpha helices are generally thought to serve this role. In potassium and sodium channels, voltage-sensing S4 helices contain positively-charged lysine or arginine residues in repeated motifs. In its resting state, half of each S4 helix is in contact with the cell cytosol. Upon depolarization, the positively-charged residues on the S4 domains move toward the exoplasmic surface of the membrane. It is thought that the first 4 arginines account for the gating current, moving toward the extracellular solvent upon channel activation in response to membrane depolarization. The movement of 10–12 of these protein-bound positive charges triggers a conformational change that opens the channel. The exact mechanism by which this movement occurs is not currently agreed upon, however the canonical, transporter, paddle, and twisted models are examples of current theories.</p>
<p>Movement of the voltage-sensor triggers a conformational change of the gate of the conducting pathway, controlling the flow of ions through the channel.</p>
<p>The main functional part of the voltage-sensitive protein domain of these channels generally contains a region composed of S3b and S4 helices, known as the “paddle” due to its shape, which appears to be a conserved sequence, interchangeable across a wide variety of cells and species. A similar voltage sensor paddle has also been found in a family of voltage sensitive phosphatases in various species. Genetic engineering of the paddle region from a species of volcano-dwelling archaebacteria into rat brain potassium channels results in a fully functional ion channel, as long as the whole intact paddle is replaced. This “modularity” allows use of simple and inexpensive model systems to study the function of this region, its role in disease, and pharmaceutical control of its behavior rather than being limited to poorly characterized, expensive, and/or difficult to study preparations.</p>
<p>Although voltage-gated ion channels are typically activated by membrane depolarization, some channels, such as inward-rectifier potassium ion channels, are activated instead by hyperpolarization.</p>
<p>The gate is thought to be coupled to the voltage sensing regions of the channels and appears to contain a mechanical obstruction to ion flow. While the S6 domain has been agreed upon as the segment acting as this obstruction, its exact mechanism is unknown. Possible explanations include: the S6 segment makes a scissor-like movement allowing ions to flow through, the S6 segment breaks into two segments allowing of passing of ions through the channel, or the S6 channel serving as the gate itself. The mechanism by which the movement of the S4 segment affects that of S6 is still unknown, however it is theorized that there is a S4-S5 linker whose movement allows the opening of S6.</p>
<p>Inactivation of ion channels occurs within milliseconds after opening. Inactivation is thought to be mediated by an intracellular gate that controls the opening of the pore on the inside of the cell. This gate is modeled as a ball tethered to a flexible chain. During inactivation, the chain folds in on itself and the ball blocks the flow of ions through the channel. Fast inactivation is directly linked to the activation caused by intramembrane movements of the S4 segments, though the mechanism linking movement of S4 and the engagement of the inactivation gate is unknown.</p>
<p>Important representative families of voltage-gated ion channels are the:</p>
<ul>
<li>Voltage-gated sodium channels: This family contains at least 9 members and is largely responsible for action potential creation and propagation. The pore-forming α subunits are very large (up to 4,000 amino acids) and consist of four homologous repeat domains (I-IV) each comprising six transmembrane segments (S1-S6) for a total of 24 transmembrane segments. The members of this family also coassemble with auxiliary β subunits, each spanning the membrane once. Both α and β subunits are extensively glycosylated.</li>
<li>Voltage-gated calcium channels: This family consists of channels that are formed as a complex of several different subunits: α1, α2δ, β1-4, and γ. The α1 subunit forms the Ca<sup>2+</sup> selective ion conducting pore while the associated subunits have several functions including modulation of gating.These channels play an important role in both linking muscle excitation with contraction as well as neuronal excitation with transmitter release. The α subunits have an overall structural resemblance to those of the sodium channels and are equally large.</li>
<li>Cation channels of sperm: This small family of channels, normally referred to as Catsper channels, is related to the two-pore channels and distantly related to TRP channels.</li>
<li>Voltage-gated potassium channels (KV): This family contains almost 40 members, which are further divided into 12 subfamilies. These channels are known mainly for their role in repolarizing the cell membrane following action potentials. The α subunits have six transmembrane segments, homologous to a single domain of the sodium channels. Correspondingly, they assemble as tetramers to produce a functioning channel.</li>
<li>Some transient receptor potential channels: This group of channels, normally referred to simply as TRP channels, is named after their role in <em>Drosophila</em> phototransduction. This family, containing at least 28 members, is incredibly diverse in its method of activation. Some TRP channels seem to be constitutively open, while others are gated by voltage, intracellular Ca<sup>2+</sup>, pH, redox state, osmolarity, and mechanical stretch. These channels also vary according to the ion(s) they pass, some being selective for Ca<sup>2+</sup> while others are less selective, acting as cation channels. This family is subdivided into 6 subfamilies based on homology: classical (TRPC), vanilloid receptors (TRPV), melastatin (TRPM), polycystins (TRPP), mucolipins (TRPML), and ankyrin transmembrane protein 1 (TRPA).</li>
<li>Hyperpolarization-activated cyclic nucleotide-gated channels: The opening of these channels is due to hyperpolarization rather than the depolarization required for other cyclic nucleotide-gated channels. These channels are also sensitive to the cyclic nucleotides cAMP and cGMP, which alter the voltage sensitivity of the channel’s opening. These channels are permeable to the monovalent cations K<sup>+</sup> and Na<sup>+</sup>. There are 4 members of this family, all of which form tetramers of six-transmembrane α subunits. As these channels open under hyperpolarizing conditions, they function as pacemaking channels in the heart, particularly the SA node.</li>
<li>Voltage-gated proton channels: Voltage-gated proton channels open with depolarization, but in a strongly pH-sensitive manner. The result is that these channels open only when the electrochemical gradient is outward, such that their opening will only allow protons to leave cells. Their function thus appears to be acid extrusion from cells. Another important function occurs in phagocytes (e.g. eosinophils, neutrophils, macrophages) during the “respiratory burst.” When bacteria or other microbes are engulfed by phagocytes, the enzyme NADPH oxidase assembles in the membrane and begins to produce reactive oxygen species (ROS) that help kill bacteria. NADPH oxidase is electrogenic, moving electrons across the membrane, and proton channels open to allow proton flux to balance the electron movement electrically.</li>
</ul>

<div class="figure" style="text-align: center"><span id="fig:ionchannel"></span>
<img src="figures/potential/voltage_gated_ion_channels.png" alt="The structures (from left to right) of voltage-gated sodium, potassium, calcium and chloride channels. Top: side view; bottom: top view. Data from PDB 5EK0, PDB 2A79, PDB 3JBR and PDB 1KPL rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 25.20: The structures (from left to right) of voltage-gated sodium, potassium, calcium and chloride channels. Top: side view; bottom: top view. Data from <a href="https://www.rcsb.org/structure/5EK0">PDB 5EK0</a>, <a href="https://www.rcsb.org/structure/2A79">PDB 2A79</a>, <a href="https://www.rcsb.org/structure/3JBR">PDB 3JBR</a> and <a href="https://www.rcsb.org/structure/1KPL">PDB 1KPL</a> rendered with open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p><a href="https://en.wikipedia.org/wiki/Ligand-gated_ion_channel">Ligand-gated ion channels</a>, also commonly referred to as ionotropic receptors, are a group of transmembrane ion-channel proteins which open to allow ions such as Na<sup>+</sup>, K<sup>+</sup>, Ca<sup>2+</sup>, and Cl<sup>−</sup> to pass through the membrane in response to the binding of a chemical messenger (i.e. a ligand), such as a neurotransmitter.</p>
<p>When a presynaptic neuron is excited, it releases a neurotransmitter from vesicles into the synaptic cleft. The neurotransmitter then binds to receptors located on the postsynaptic neuron. If these receptors are ligand-gated ion channels, a resulting conformational change opens the ion channels, which leads to a flow of ions across the cell membrane. This, in turn, results in either a depolarization, for an excitatory receptor response, or a hyperpolarization, for an inhibitory response.</p>
<p>These receptor proteins are typically composed of at least two different domains: a transmembrane domain which includes the ion pore, and an extracellular domain which includes the ligand binding location (an allosteric binding site). This modularity has enabled a ‘divide and conquer’ approach to finding the structure of the proteins (crystallising each domain separately). The function of such receptors located at synapses is to convert the chemical signal of presynaptically released neurotransmitter directly and very quickly into a postsynaptic electrical signal. Many ligand-gated ion channels are additionally modulated by allosteric ligands, by channel blockers, ions, or the membrane potential. Ligand-gated ion channels are classified into three superfamilies which lack evolutionary relationship: cys-loop receptors, ionotropic glutamate receptors and ATP-gated channels.</p>
<p>The cys-loop receptors are named after a characteristic loop formed by a disulfide bond between two cysteine residues in the N terminal extracellular domain. They are part of a larger family of pentameric ligand-gated ion channels that usually lack this disulfide bond, hence the tentative name “Pro-loop receptors”. A binding site in the extracellular N-terminal ligand-binding domain gives them receptor specificity for (1) acetylcholine (AcCh), (2) serotonin, (3) glycine, (4) glutamate and (5) γ-aminobutyric acid (GABA) in vertebrates. The receptors are subdivided with respect to the type of ion that they conduct (anionic or cationic) and further into families defined by the endogenous ligand. They are usually pentameric with each subunit containing 4 transmembrane helices constituting the transmembrane domain, and a beta sheet sandwich type, extracellular, N terminal, ligand binding domain. Some also contain an intracellular domain like shown in the image.</p>
<p>The prototypic ligand-gated ion channel is the nicotinic acetylcholine receptor. It consists of a pentamer of protein subunits (typically ααβγδ), with two binding sites for acetylcholine (one at the interface of each alpha subunit). When the acetylcholine binds it alters the receptor’s configuration (twists the T2 helices which moves the leucine residues, which block the pore, out of the channel pathway) and causes the constriction in the pore of approximately 3 angstroms to widen to approximately 8 angstroms so that ions can pass through. This pore allows Na<sup>+</sup> ions to flow down their electrochemical gradient into the cell. With a sufficient number of channels opening at once, the inward flow of positive charges carried by Na<sup>+</sup> ions depolarizes the postsynaptic membrane sufficiently to initiate an action potential.</p>
<p>While single-cell organisms like bacteria would have little apparent need for the transmission of an action potential, a bacterial homologue to an LIC has been identified, hypothesized to act nonetheless as a chemoreceptor. This prokaryotic nAChR variant is known as the GLIC receptor, after the species in which it was identified; Gloeobacter Ligand-gated Ion C channel.</p>
<p>The ionotropic glutamate receptors bind the neurotransmitter glutamate. They form tetramers with each subunit consisting of an extracellular amino terminal domain (ATD, which is involved tetramer assembly), an extracellular ligand binding domain (LBD, which binds glutamate), and a transmembrane domain (TMD, which forms the ion channel). The transmembrane domain of each subunit contains three transmembrane helices as well as a half membrane helix with a reentrant loop. The structure of the protein starts with the ATD at the N terminus followed by the first half of the LBD which is interrupted by helices 1,2 and 3 of the TMD before continuing with the final half of the LBD and then finishing with helix 4 of the TMD at the C terminus. This means there are three links between the TMD and the extracellular domains. Each subunit of the tetramer has a binding site for glutamate formed by the two LBD sections forming a clamshell like shape. Only two of these sites in the tetramer need to be occupied to open the ion channel. The pore is mainly formed by the half helix 2 in a way which resembles an inverted potassium channel.</p>

<div class="figure" style="text-align: center"><span id="fig:lgic"></span>
<img src="figures/potential/ligand_gated_ion_channels.png" alt="Structures of ligand-gated ion channels. Top row: The nicotinic acetylcholine receptor (left) and the serotonin 5-HT3 receptor. Middle row: The GABAA receptor (left) and glycine receptor (rigt). Bottom row: The AMPA receptor (left) and NMDA receptor (reight). Data from PDB 2BG9, PDB 6HIO, PDB 6D6U, PDB 3JAD, PDB 5IDE, and PDB 4PE5, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 25.21: Structures of ligand-gated ion channels. Top row: The nicotinic acetylcholine receptor (left) and the serotonin 5-HT3 receptor. Middle row: The GABA<sub>A</sub> receptor (left) and glycine receptor (rigt). Bottom row: The AMPA receptor (left) and NMDA receptor (reight). Data from <a href="https://www.rcsb.org/structure/2BG9">PDB 2BG9</a>, <a href="https://www.rcsb.org/structure/6HIO">PDB 6HIO</a>, <a href="https://www.rcsb.org/structure/6D6U">PDB 6D6U</a>, <a href="https://www.rcsb.org/structure/3JAD">PDB 3JAD</a>, <a href="https://www.rcsb.org/structure/5IDE">PDB 5IDE</a>, and <a href="https://www.rcsb.org/structure/4PE5">PDB 4PE5</a>, rendered with open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p>Also called G protein-coupled receptor, seven-transmembrane domain receptor, 7 TM receptor, constitute a large protein family of receptors that sense molecules outside the cell and activate inside signal transduction pathways and, ultimately, cellular responses. They pass through the cell membrane 7 times. G-protein-Linked receptors are a huge family that have hundreds of members identified. Ion-channel-linked receptors (e.g. GABAB, NMDA, etc.) are only a part of them.</p>
</div>
<div id="the-reversal-potential" class="section level2">
<h2><span class="header-section-number">25.16</span> The Reversal Potential</h2>
<p>The reversal potential (or equilibrium potential) of an ion is the value of transmembrane voltage at which diffusive and electrical forces counterbalance, so that there is no net ion flow across the membrane. This means that the transmembrane voltage exactly opposes the force of diffusion of the ion, such that the net current of the ion across the membrane is zero and unchanging. The reversal potential is important because it corresponds to the voltage that acts on channels permeable to that ion—in other words, it gives the voltage that the ion concentration gradient generates when it acts as a battery.</p>
<p>The equilibrium potential of a particular ion is usually designated by the notation E<sub>ion</sub>.The equilibrium potential for any ion can be calculated using the Nernst equation. For example, reversal potential for potassium ions will be as follows:</p>
<p><span class="math display">\[  E_{eq,K^+} = \frac{RT}{zF} \ln \frac{[K^+]_{o}}{[K^+]_{i}} \]</span></p>
<p>where</p>
<ul>
<li>E<sub>eq,K<sup>+</sup></sub> is the equilibrium potential for potassium, measured in volts</li>
<li>R is the universal gas constant, equal to 8.314 Joule·K<sup>−1</sup>·mol<sup>−1</sup></li>
<li>T is the absolute temperature, measured in Kelvin</li>
<li>z is the number of elementary charges of the ion in question involved in the reaction</li>
<li>F is the Faraday constant, equal to 96,485 Coulomb·mol<sup>−1</sup> or J·V<sup>−1</sup>·mol<sup>−1</sup></li>
<li>[K<sup>+</sup>]<sub>o</sub> is the extracellular concentration of potassium, measured in mol·m<sup>−3</sup> or mmol·l<sup>−1</sup></li>
<li>[K<sup>+</sup>]<sub>i</sub> is the intracellular concentration of potassium</li>
</ul>
<p>Even if two different ions have the same charge (i.e., K<sup>+</sup> and Na<sup>+</sup>), they can still have very different equilibrium potentials, provided their outside and/or inside concentrations differ. Take, for example, the equilibrium potentials of potassium and sodium in neurons. The potassium equilibrium potential E<sub>K</sub> is −84 mV with 5 mM potassium outside and 140 mM inside. On the other hand, the sodium equilibrium potential, E<sub>Na</sub>, is approximately +66 mV with approximately 12 mM sodium inside and 140 mM outside.</p>
<p>A neuron’s resting membrane potential actually changes during the development of an organism. In order for a neuron to eventually adopt its full adult function, its potential must be tightly regulated during development. As an organism progresses through development the resting membrane potential becomes more negative. Glial cells are also differentiating and proliferating as development progresses in the brain. The addition of these glial cells increases the organism’s ability to regulate extracellular potassium. The drop in extracellular potassium can lead to a decrease in membrane potential of 35 mV.</p>
<p>Cell excitability is the change in membrane potential that is necessary for cellular responses in various tissues. Cell excitability is a property that is induced during early embriogenesis. Excitability of a cell has also been defined as the ease with which a response may be triggered. The resting potential forms the basis of cell excitability and these processes are fundamental for the generation of graded and action potentials.</p>
<p>The most important regulators of cell excitability are the extracellular calcium concentration and the calcium-sensing receptor. Calcium is also the most important second messenger in excitable cell signaling. Other important proteins that regulate cell excitability are voltage-gated ion channels, ion transporters, membrane receptors and hyperpolarization-activated cyclic-nucleotide-gated channels. For example, potassium channels are important regulators of excitability in neurons, cardiac myocytes and many other excitable cells like astrocytes. Activation of synaptic receptors initiates long-lasting changes in neuronal excitability.</p>
<p>Many cell types are considered to have an excitable membrane. Excitable cells are neurons, cardiac myocytes, skeletal myocytes, smooth muscle cells, many types of endothelial cells (e.g. beta cells), glial cells (e.g. astrocytes), mechanoreceptor cells (e.g. hair cells and Merkel cells), chemoreceptor cells (e.g. glomus cells, taste receptors), some plant cells and possibly immune cells. Astrocytes display a form of non-electrical excitability based on intracellular calcium variations related to the expression of several receptors through which they can detect the synaptic signal. In neurons, there are different membrane properties in some portions of the cell, for example, dendritic excitability endows neurons with the capacity for coincidence detection of spatially separated inputs.</p>
</div>
<div id="the-resting-potential" class="section level2">
<h2><span class="header-section-number">25.17</span> The Resting Potential</h2>
<p>When the membrane potential of a cell goes for a long period of time without changing significantly, it is referred to as a resting potential or resting voltage. This term is used for the membrane potential of non-excitable cells, but also for the membrane potential of excitable cells in the absence of excitation. In excitable cells, the other possible states are graded membrane potentials (of variable amplitude), and action potentials, which are large, all-or-nothing rises in membrane potential that usually follow a fixed time course. Excitable cells include neurons, muscle cells, and some secretory cells in glands. Even in other types of cells, however, the membrane voltage can undergo changes in response to environmental or intracellular stimuli. For example, depolarization of the plasma membrane appears to be an important step in programmed cell death.</p>
<p>The interactions that generate the resting potential are modeled by the Goldman equation. This is similar in form to the Nernst equation shown above, in that it is based on the charges of the ions in question, as well as the difference between their inside and outside concentrations. However, it also takes into consideration the relative permeability of the plasma membrane to each ion in question.</p>
<p><span class="math display">\[ E_{m} = \frac{RT}{F} \ln{ \left( \frac{ P_{\mathrm{K}}[\mathrm{K}^{+}]_\mathrm{out} + P_{\mathrm{Na}}[\mathrm{Na}^{+}]_\mathrm{out} + P_{\mathrm{Cl}}[\mathrm{Cl}^{-}]_\mathrm{in}}{ P_{\mathrm{K}}[\mathrm{K}^{+}]_\mathrm{in} + P_{\mathrm{Na}}[\mathrm{Na}^{+}]_\mathrm{in} + P_{\mathrm{Cl}}[\mathrm{Cl}^{-}]_\mathrm{out}} \right) } \]</span></p>
<p>The three ions that appear in this equation are potassium (K<sup>+</sup>), sodium (Na<sup>+</sup>), and chloride (Cl<sup>−</sup>). Calcium is omitted, but can be added to deal with situations in which it plays a significant role. Being an anion, the chloride terms are treated differently from the cation terms; the intracellular concentration is in the numerator, and the extracellular concentration in the denominator, which is reversed from the cation terms. <em>P<sub>i</sub></em> stands for the relative permeability of the ion type <em>i</em>.</p>
<p>In essence, the Goldman formula expresses the membrane potential as a weighted average of the reversal potentials for the individual ion types, weighted by permeability. (Although the membrane potential changes about 100 mV during an action potential, the concentrations of ions inside and outside the cell do not change significantly. They remain close to their respective concentrations when the membrane is at resting potential.) In most animal cells, the permeability to potassium is much higher in the resting state than the permeability to sodium. As a consequence, the resting potential is usually close to the potassium reversal potential. The permeability to chloride can be high enough to be significant, but, unlike the other ions, chloride is not actively pumped, and therefore equilibrates at a reversal potential very close to the resting potential determined by the other ions.</p>
<p>Values of resting membrane potential in most animal cells usually vary between the potassium reversal potential (usually around -80 mV) and around -40 mV. The resting potential in excitable cells (capable of producing action potentials) is usually near -60 mV—more depolarized voltages would lead to spontaneous generation of action potentials. Immature or undifferentiated cells show highly variable values of resting voltage, usually significantly more positive than in differentiated cells. In such cells, the resting potential value correlates with the degree of differentiation: undifferentiated cells in some cases may not show any transmembrane voltage difference at all.</p>
<p>Maintenance of the resting potential can be metabolically costly for a cell because of its requirement for active pumping of ions to counteract losses due to leakage channels. The cost is highest when the cell function requires an especially depolarized value of membrane voltage. For example, the resting potential in daylight-adapted blowfly (<em>Calliphora vicina</em>) photoreceptors can be as high as -30 mV. This elevated membrane potential allows the cells to respond very rapidly to visual inputs; the cost is that maintenance of the resting potential may consume more than 20% of overall cellular ATP.</p>
<p>On the other hand, the high resting potential in undifferentiated cells can be a metabolic advantage. This apparent paradox is resolved by examination of the origin of that resting potential. Little-differentiated cells are characterized by extremely high input resistance, which implies that few leakage channels are present at this stage of cell life. As an apparent result, potassium permeability becomes similar to that for sodium ions, which places resting potential in-between the reversal potentials for sodium and potassium as discussed above. The reduced leakage currents also mean there is little need for active pumping in order to compensate, therefore low metabolic cost.</p>
</div>
<div id="the-action-potential" class="section level2">
<h2><span class="header-section-number">25.18</span> The Action Potential</h2>
<p>An action potential occurs when the membrane potential of a specific cell location rapidly rises and falls: this depolarisation then causes adjacent locations to similarly depolarise. Action potentials occur in several types of animal cells, called excitable cells, which include neurons, muscle cells, endocrine cells, glomus cells (peripheral chemoreceptor cells mainly located in the carotid and aortic bodies), and in some plant cells.</p>
<p>In neurons, action potentials play a central role in cell-to-cell communication by providing for—or with regard to saltatory conduction, assisting—the propagation of signals along the neuron’s axon toward synaptic boutons situated at the ends of an axon; these signals can then connect with other neurons at synapses, or to motor cells or glands. In other types of cells, their main function is to activate intracellular processes. In muscle cells, for example, an action potential is the first step in the chain of events leading to contraction. In beta cells of the pancreas, they provoke release of insulin. Action potentials in neurons are also known as “nerve impulses” or “spikes”, and the temporal sequence of action potentials generated by a neuron is called its “spike train”. A neuron that emits an action potential, or nerve impulse, is often said to “fire”.</p>
<p>Action potentials are generated by special types of voltage-gated ion channels embedded in a cell’s plasma membrane. These channels are shut when the membrane potential is near the (negative) resting potential of the cell, but they rapidly begin to open if the membrane potential increases to a precisely defined threshold voltage, depolarising the transmembrane potential. When the channels open, they allow an inward flow of sodium ions, which changes the electrochemical gradient, which in turn produces a further rise in the membrane potential. This then causes more channels to open, producing a greater electric current across the cell membrane and so on. The process proceeds explosively until all of the available ion channels are open, resulting in a large upswing in the membrane potential. The rapid influx of sodium ions causes the polarity of the plasma membrane to reverse, and the ion channels then rapidly inactivate. As the sodium channels close, sodium ions can no longer enter the neuron, and they are then actively transported back out of the plasma membrane. Potassium channels are then activated, and there is an outward current of potassium ions, returning the electrochemical gradient to the resting state. After an action potential has occurred, there is a transient negative shift, called the afterhyperpolarization.</p>
<p>In animal cells, there are two primary types of action potentials. One type is generated by voltage-gated sodium channels, the other by voltage-gated calcium channels. Sodium-based action potentials usually last for under one millisecond, but calcium-based action potentials may last for 100 milliseconds or longer. In some types of neurons, slow calcium spikes provide the driving force for a long burst of rapidly emitted sodium spikes. In cardiac muscle cells, on the other hand, an initial fast sodium spike provides a “primer” to provoke the rapid onset of a calcium spike, which then produces muscle contraction</p>

<div class="figure" style="text-align: center"><span id="fig:actionpotential"></span>
<img src="figures/potential/ActionPotential.svg" alt="Ion movement during an action potential. Key: a) Sodium (Na+) ion. b) Potassium (K+) ion. c) Sodium channel. d) Potassium channel. e) Sodium-potassium pump. In the stages of an action potential, the permeability of the membrane of the neuron changes. At the resting state (1), sodium and potassium ions have limited ability to pass through the membrane, and the neuron has a net negative charge inside. Once the action potential is triggered, the depolarization (2) of the neuron activates sodium channels, allowing sodium ions to pass through the cell membrane into the cell, resulting in a net positive charge in the neuron relative to the extracellular fluid. After the action potential peak is reached, the neuron begins repolarization (3), where the sodium channels close and potassium channels open, allowing potassium ions to cross the membrane into the extracellular fluid, returning the membrane potential to a negative value. Finally, there is a refractory period (4), during which the voltage-dependent ion channels are inactivated while the Na+ and K+ ions return to their resting state distributions across the membrane (1), and the neuron is ready to repeat the process for the next action potential." width="70%" />
<p class="caption">
Figure 25.22: <a href="https://commons.wikimedia.org/wiki/File:Membrane_Permeability_of_a_Neuron_During_an_Action_Potential.svg">Ion movement during an action potential.</a> Key: a) Sodium (Na<sup>+</sup>) ion. b) Potassium (K<sup>+</sup>) ion. c) Sodium channel. d) Potassium channel. e) Sodium-potassium pump. In the stages of an action potential, the permeability of the membrane of the neuron changes. At the resting state (1), sodium and potassium ions have limited ability to pass through the membrane, and the neuron has a net negative charge inside. Once the action potential is triggered, the depolarization (2) of the neuron activates sodium channels, allowing sodium ions to pass through the cell membrane into the cell, resulting in a net positive charge in the neuron relative to the extracellular fluid. After the action potential peak is reached, the neuron begins repolarization (3), where the sodium channels close and potassium channels open, allowing potassium ions to cross the membrane into the extracellular fluid, returning the membrane potential to a negative value. Finally, there is a refractory period (4), during which the voltage-dependent ion channels are inactivated while the Na<sup>+</sup> and K<sup>+</sup> ions return to their resting state distributions across the membrane (1), and the neuron is ready to repeat the process for the next action potential.
</p>
</div>
</div>
<div id="graded-potentials" class="section level2">
<h2><span class="header-section-number">25.19</span> Graded Potentials</h2>
<p>As explained above, the potential at any point in a cell’s membrane is determined by the ion concentration differences between the intracellular and extracellular areas, and by the permeability of the membrane to each type of ion. The ion concentrations do not normally change very quickly (with the exception of Ca<sup>2+</sup>, where the baseline intracellular concentration is so low that even a small influx may increase it by orders of magnitude), but the permeabilities of the ions can change in a fraction of a millisecond, as a result of activation of ligand-gated ion channels. The change in membrane potential can be either large or small, depending on how many ion channels are activated and what type they are, and can be either long or short, depending on the lengths of time that the channels remain open. Changes of this type are referred to as graded potentials, in contrast to action potentials, which have a fixed amplitude and time course.</p>
<p>As can be derived from the Goldman equation shown above, the effect of increasing the permeability of a membrane to a particular type of ion shifts the membrane potential toward the reversal potential for that ion. Thus, opening Na<sup>+</sup> channels shifts the membrane potential toward the Na<sup>+</sup> reversal potential, which is usually around +100 mV. Likewise, opening K<sup>+</sup> channels shifts the membrane potential toward about –90 mV, and opening Cl<sup>−</sup> channels shifts it toward about –70 mV (resting potential of most membranes). Thus, Na<sup>+</sup> channels shift the membrane potential in a positive direction, K<sup>+</sup> channels shift it in a negative direction (except when the membrane is hyperpolarized to a value more negative than the K<sup>+</sup> reversal potential), and Cl<sup>−</sup> channels tend to shift it towards the resting potential.</p>
<p>Graded membrane potentials are particularly important in neurons, where they are produced by synapses—a temporary change in membrane potential produced by activation of a synapse by a single graded or action potential is called a postsynaptic potential. Neurotransmitters that act to open Na<sup>+</sup> channels typically cause the membrane potential to become more positive, while neurotransmitters that activate K<sup>+</sup> channels typically cause it to become more negative; those that inhibit these channels tend to have the opposite effect.</p>
<p>Whether a postsynaptic potential is considered excitatory or inhibitory depends on the reversal potential for the ions of that current, and the threshold for the cell to fire an action potential (around –50mV). A postsynaptic current with a reversal potential above threshold, such as a typical Na<sup>+</sup> current, is considered excitatory. A current with a reversal potential below threshold, such as a typical K<sup>+</sup> current, is considered inhibitory. A current with a reversal potential above the resting potential, but below threshold, will not by itself elicit action potentials, but will produce subthreshold membrane potential oscillations. Thus, neurotransmitters that act to open Na<sup>+</sup> channels produce excitatory postsynaptic potentials, or EPSPs, whereas neurotransmitters that act to open K<sup>+</sup> or Cl<sup>−</sup> channels typically produce inhibitory postsynaptic potentials, or IPSPs. When multiple types of channels are open within the same time period, their postsynaptic potentials summate (are added together).</p>
<p>From the viewpoint of biophysics, the resting membrane potential is merely the membrane potential that results from the membrane permeabilities that predominate when the cell is resting. The Goldman equation of weighted averages always applies, but the following approach may be more easily visualized. At any given moment, there are two factors for an ion that determine how much influence that ion will have over the membrane potential of a cell:</p>
<ol style="list-style-type: decimal">
<li>That ion’s driving force</li>
<li>That ion’s permeability</li>
</ol>
<p>If the driving force is high, then the ion is being “pushed” across the membrane. If the permeability is high, it will be easier for the ion to diffuse across the membrane.</p>
<ul>
<li>Driving force is the net electrical force available to move that ion across the membrane. It is calculated as the difference between the voltage that the ion “wants” to be at (its equilibrium potential) and the actual membrane potential (E<sub>m</sub>). So, in formal terms, the driving force for an ion = E<sub>m</sub> - E<sub>ion</sub></li>
<li>For example, at our earlier calculated resting potential of −73 mV, the driving force on potassium is 7 mV : (−73 mV) − (−80 mV) = 7 mV. The driving force on sodium would be (−73 mV) − (60 mV) = −133 mV.</li>
<li>Permeability is a measure of how easily an ion can cross the membrane. It is normally measured as the (electrical) conductance and the unit, siemens (S), corresponds to 1 C·s<sup>−1</sup>·V<sup>−1</sup>, that is one coulomb per second per volt of potential.</li>
</ul>
<p>So, in a resting membrane, while the driving force for potassium is low, its permeability is very high. Sodium has a huge driving force but almost no resting permeability. In this case, potassium carries about 20 times more current than sodium, and thus has 20 times more influence over E<sub>m</sub> than does sodium.</p>
<p>However, consider another case—the peak of the action potential. Here, permeability to Na is high and K permeability is relatively low. Thus, the membrane moves to near E<sub>Na</sub> and far from E<sub>K</sub>.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="the-excretory-system.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="animal-locomotion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/biology-text/edit/master/nervous_system.rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
